<!doctype html><html lang=en dir=ltr><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="Skip to main content Advertisement BMC Part of Springer Nature
Search Explore journals Get published About BMC Login Journal of Animal Science and Technology logo Journal of Animal Science and Technology Articles Review Open Access Published: 15 May 2014 Hanwoo cattle: origin, domestication, breeding strategies and genomic selection
Seung-Hwan Lee1†, Byoung-Ho Park2†, Aditi Sharma1†, Chang-Gwon Dang1, Seung-Soo Lee2, Tae-Jeong Choi2, Yeon-Ho Choy2, Hyeong-Cheol Kim1, Ki-Jun Jeon1, Si-Dong Kim2, Seong-Heum Yeon1, Soo-Bong Park2 & Hee-Seol Kang1 Journal of Animal Science and Technology volume 56, Article number: 2 (2014) Cite this article"><meta name=theme-color content="#FFFFFF"><meta name=color-scheme content="light dark"><meta property="og:title" content="Hanwoo cattle: origin, domestication, breeding strategies and genomic selection"><meta property="og:description" content="Skip to main content Advertisement BMC Part of Springer Nature
Search Explore journals Get published About BMC Login Journal of Animal Science and Technology logo Journal of Animal Science and Technology Articles Review Open Access Published: 15 May 2014 Hanwoo cattle: origin, domestication, breeding strategies and genomic selection
Seung-Hwan Lee1†, Byoung-Ho Park2†, Aditi Sharma1†, Chang-Gwon Dang1, Seung-Soo Lee2, Tae-Jeong Choi2, Yeon-Ho Choy2, Hyeong-Cheol Kim1, Ki-Jun Jeon1, Si-Dong Kim2, Seong-Heum Yeon1, Soo-Bong Park2 & Hee-Seol Kang1 Journal of Animal Science and Technology volume 56, Article number: 2 (2014) Cite this article"><meta property="og:type" content="article"><meta property="og:url" content="http://localhost:1313/docs/hanwoo-cattle/"><meta property="article:section" content="docs"><title>Hanwoo cattle: origin, domestication, breeding strategies and genomic selection | Derivatives</title><link rel=manifest href=/manifest.json><link rel=icon href=/favicon.png type=image/x-icon><link rel=stylesheet href=/book.min.1a6fac2df464e68d664cb846d822c678cd622692d6fae24ee18901a08bfc8eb6.css integrity="sha256-Gm+sLfRk5o1mTLhG2CLGeM1iJpLW+uJO4YkBoIv8jrY=" crossorigin=anonymous><script defer src=/flexsearch.min.js></script>
<script defer src=/en.search.min.a1bf0aa8629457be4b1df9c86949659a2d7e6e5b8abf70b627dbc156a50e8b11.js integrity="sha256-ob8KqGKUV75LHfnIaUllmi1+bluKv3C2J9vBVqUOixE=" crossorigin=anonymous></script></head><body dir=ltr><input type=checkbox class="hidden toggle" id=menu-control>
<input type=checkbox class="hidden toggle" id=toc-control><main class="container flex"><aside class=book-menu><div class=book-menu-content><nav><h2 class=book-brand><a class="flex align-center" href=/><span>Derivatives</span></a></h2><div class=book-search><input type=text id=book-search-input placeholder=Search aria-label=Search maxlength=64 data-hotkeys=s/><div class="book-search-spinner hidden"></div><ul id=book-search-results></ul></div><ul><li><a href=http://localhost:1313/docs/readme/>README</a></li><li><a href=http://localhost:1313/docs/architecture-human-orc/>Architecture of the Human Origin Recognition Complex*</a></li><li><a href=http://localhost:1313/docs/amyloid-%CE%B2/>Origins of amyloid-β</a></li><li><a href=http://localhost:1313/docs/arsenic/>Effects of arsenic and heavy metals on metabolic pathways in cells of human origin: Similarities and differences</a></li><li><a href=http://localhost:1313/docs/back-to-future/>Back to the future: origins and directions of the “Agile Manifesto” – views of the originators</a></li><li><a href=http://localhost:1313/docs/biopolymers/>Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers</a></li><li><input type=checkbox id=section-4e626a39f813bdfb0b0f11621a8d97e6 class=toggle>
<label for=section-4e626a39f813bdfb0b0f11621a8d97e6 class="flex justify-between"><a href=http://localhost:1313/docs/cc-by/>CC-BY</a></label><ul><li><a href=http://localhost:1313/docs/cc-by/cc-by-2-summary/>Creative Commons License Deed Attribution 2.0 Generic (CC BY 2.0)</a></li><li><a href=http://localhost:1313/docs/cc-by/cc-by-2/>Creative Commons Legal Code Attribution 2.0</a></li><li><a href=http://localhost:1313/docs/cc-by/cc-by-3-summary/>Creative Commons License Deed Attribution 3.0 Unported (CC BY 3.0)</a></li><li><a href=http://localhost:1313/docs/cc-by/cc-by-3/>Creative Commons Attribution 3.0 Unported</a></li><li><a href=http://localhost:1313/docs/cc-by/cc-by-4-summary/>Creative Commons License Deed Attribution 4.0 International (CC BY 4.0)</a></li><li><a href=http://localhost:1313/docs/cc-by/cc-by-4-de/>Creative Commons Konzessionsurkunde Namensnennung 4.0 International (CC BY 4.0)</a></li><li><a href=http://localhost:1313/docs/cc-by/cc-by-4/>Creative Commons Legal Code Attribution 4.0 International</a></li></ul></li><li><a href=http://localhost:1313/docs/charon-smooth-plains/>The nature and origin of Charon's smooth plains</a></li><li><a href=http://localhost:1313/docs/chinese-word-cell/>Origin of the Chinese word for “cell”: an unusual but wonderful idea of a mathematician</a></li><li><a href=http://localhost:1313/docs/city-origins/>City origins</a></li><li><a href=http://localhost:1313/docs/counterfeit-versus-original/>Counterfeit versus original patronage: Do emotional brand attachment, brand involvement, and past experience matter?</a></li><li><a href=http://localhost:1313/docs/creatine/>ON THE ORIGIN OF CREATINE</a></li><li><a href=http://localhost:1313/docs/detecting-moisture/>Detecting the origins of moisture over Southeast China: Seasonal variation and heavy Rainfall</a></li><li><a href=http://localhost:1313/docs/differentiation-spirits/>Differentiation Between Spirits According to Their Botanical Origin</a></li><li><a href=http://localhost:1313/docs/food-taboos/>Food taboos: their origins and purposes</a></li><li><a href=http://localhost:1313/docs/ganges/>Ganges: special at its origin</a></li><li><a href=http://localhost:1313/docs/given-that-detailed/>Given that the detailed original criteria for deliberate practice have not changed, could the understanding of this complex concept have improved over time? A response to Macnamara and Hambrick (2020)</a></li><li><a href=http://localhost:1313/docs/hanwoo-cattle/ class=active>Hanwoo cattle: origin, domestication, breeding strategies and genomic selection</a></li><li><a href=http://localhost:1313/docs/hoarding/>The developmental origins of hoarding disorder in adolescence: a longitudinal clinical interview study following an epidemiological survey</a></li><li><a href=http://localhost:1313/docs/how-to-write/>How to write an original article for the *Journal of Orthopaedics and Traumatology*</a></li><li><a href=http://localhost:1313/docs/in-out-qinghai-tibet-plateau/>“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds</a></li><li><a href=http://localhost:1313/docs/interview-patrick-o-brown/>An interview with Patrick O Brown on the origins and future of open access</a></li><li><a href=http://localhost:1313/docs/manipulated-photos/>Can people identify original and manipulated photos of real-world scenes?</a></li><li><a href=http://localhost:1313/docs/neighborhood-spam/>Neighborhoods and bands: an analysis of the origins of spam</a></li><li><input type=checkbox id=section-18bd70f4fad61eb2742d114842aa84cf class=toggle>
<label for=section-18bd70f4fad61eb2742d114842aa84cf class="flex justify-between"><a href=http://localhost:1313/docs/letters/>Letters</a></label><ul><li><a href=http://localhost:1313/docs/letters/non-neanderthal/>Non-Neanderthal Origin of the HLA-DPB1*0401</a></li><li><a href=http://localhost:1313/docs/letters/reply-to-ding/>Reply to Ding et al.: Non-Neanderthal Origin of the HLA-DPB1*0401</a></li></ul></li><li><a href=http://localhost:1313/docs/origin-of-the-ectopic-beat/>What is the origin of the ectopic beat?</a></li><li><a href=http://localhost:1313/docs/origins/>Origins</a></li><li><a href=http://localhost:1313/docs/patient-safety/>The 5th anniversary of “Patient Safety in Surgery” – from the Journal’s origin to its future vision</a></li><li><a href=http://localhost:1313/docs/rendang/>Tracing the origins of rendang and its development</a></li><li><a href=http://localhost:1313/docs/soft-robot/>Assessing the Impact of an Original Soft Communicative Robot in a Nursing Home in Japan: Will Softness or Conversations Bring more Smiles to Older People?</a></li><li><a href=http://localhost:1313/docs/softwaretest-originaldaten/>Softwaretest mit Originaldaten Eine Analyse aus Sicht des Datenschutzes</a></li><li><a href=http://localhost:1313/docs/spectacular-submarine/>Origin of spectacular fields of submarine sediment waves around volcanic islands</a></li><li><a href=http://localhost:1313/docs/urban-conservation/>Urban Conservation Through Chinese Local Original Architecture: The Preservation and Renovation of Zhongshan Road in Hangzhou</a></li><li><a href=http://localhost:1313/docs/vertebrate-gills/>The Origin of Vertebrate Gills</a></li><li><a href=http://localhost:1313/docs/volcanic-origin-1741/>Volcanic origin of the 1741 Oshima-Oshima tsunami in the Japan Sea</a></li><li><a href=http://localhost:1313/docs/what-is-morita-therapy/>What is Morita Therapy? The Nature, Origins, and Cross-Cultural Application of a Unique Japanese Psychotherapy</a></li><li><a href=http://localhost:1313/docs/wildcat-reappearance/>Revealing the origin of wildcat reappearance after presumed long-term absence</a></li></ul></nav><script>(function(){var e=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(){localStorage.setItem("menu.scrollTop",e.scrollTop)}),e.scrollTop=localStorage.getItem("menu.scrollTop")})()</script></div></aside><div class=book-page><header class=book-header><div class="flex align-center justify-between"><label for=menu-control><img src=/svg/menu.svg class=book-icon alt=Menu></label>
<strong>Hanwoo cattle: origin, domestication, breeding strategies and genomic selection</strong>
<label for=toc-control><img src=/svg/toc.svg class=book-icon alt="Table of Contents"></label></div><aside class="hidden clearfix"><nav id=TableOfContents><ul><li><a href=#abstract>Abstract</a></li><li><a href=#introduction>Introduction</a></li><li><a href=#origin-and-domestication>Origin and Domestication</a></li><li><a href=#population-genetic-parameters-in-hanwoo>Population genetic parameters in Hanwoo</a></li><li><a href=#hanwoo-breeding-program>Hanwoo breeding program</a></li><li><a href=#genetic-architecture-for-carcass-trait-in-hanwoo>Genetic architecture for carcass trait in Hanwoo</a></li><li><a href=#genomic-strategy-for-breeding-scheme>Genomic strategy for breeding scheme</a></li><li><a href=#strategies-for-genomic-selection-program-in-hanwoo>Strategies for genomic selection program in Hanwoo</a></li><li><a href=#conclusion>Conclusion</a></li><li><a href=#references>References</a></li><li><a href=#acknowledgements>Acknowledgements</a></li><li><a href=#author-information>Author information</a><ul><li><a href=#author-notes>Author notes</a></li><li><a href=#affiliations>Affiliations</a></li><li><a href=#corresponding-author>Corresponding author</a></li></ul></li><li><a href=#additional-information>Additional information</a><ul><li><a href=#competing-interests>Competing interests</a></li><li><a href=#authors-contributions>Authors’ contributions</a></li></ul></li><li><a href=#rights-and-permissions>Rights and permissions</a><ul><li><a href=#open-access>Open Access</a></li></ul></li><li><a href=#reprints-and-permissions>Reprints and Permissions</a><ul><li><a href=#about-this-article>About this article</a></li><li><a href=#share-this-article>Share this article</a></li><li><a href=#get-shareable-link>Get shareable link</a></li></ul></li><li><a href=#keywords>Keywords</a></li><li><a href=#contact-us>Contact us</a><ul><li><a href=#submission-enquiries>Submission enquiries</a></li><li><a href=#contact-us-1>Contact Us</a></li></ul></li><li><a href=#submit-your-next-manuscript-to-biomed-central-and-take-full-advantage-of>Submit your next manuscript to BioMed Central and take full advantage of</a></li></ul></nav></aside></header><article class=markdown><h1><a href=/docs/hanwoo-cattle/>Hanwoo cattle: origin, domestication, breeding strategies and genomic selection</a></h1><h2>Seung-Hwan Lee, Byoung-Ho Park, Aditi Sharma, Chang-Gwon Dang, Seung-Soo Lee, Tae-Jeong Choi, Yeon-Ho Choy, Hyeong-Cheol Kim, Ki-Jun Jeon, Si-Dong Kim, Seong-Heum Yeon, Soo-Bong Park & Hee-Seol Kang</h2><p>Skip to main content
Advertisement
BMC
Part of Springer Nature</p><p>Search
Explore journals
Get published
About BMC
Login
Journal of Animal Science and Technology logo
Journal of Animal Science and Technology
Articles
Review
Open Access
Published: 15 May 2014
Hanwoo cattle: origin, domestication, breeding strategies and genomic selection</p><p>Seung-Hwan Lee<sup id=fnref:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>†, Byoung-Ho Park<sup id=fnref:2><a href=#fn:2 class=footnote-ref role=doc-noteref>2</a></sup>†, Aditi Sharma<sup id=fnref1:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>†, Chang-Gwon Dang<sup id=fnref2:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>, Seung-Soo Lee<sup id=fnref1:2><a href=#fn:2 class=footnote-ref role=doc-noteref>2</a></sup>, Tae-Jeong Choi<sup id=fnref2:2><a href=#fn:2 class=footnote-ref role=doc-noteref>2</a></sup>, Yeon-Ho Choy<sup id=fnref3:2><a href=#fn:2 class=footnote-ref role=doc-noteref>2</a></sup>, Hyeong-Cheol Kim<sup id=fnref3:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>, Ki-Jun Jeon<sup id=fnref4:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>, Si-Dong Kim<sup id=fnref4:2><a href=#fn:2 class=footnote-ref role=doc-noteref>2</a></sup>, Seong-Heum Yeon<sup id=fnref5:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>, Soo-Bong Park<sup id=fnref5:2><a href=#fn:2 class=footnote-ref role=doc-noteref>2</a></sup> & Hee-Seol Kang<sup id=fnref6:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>
Journal of Animal Science and Technology volume 56, Article number: 2 (2014) Cite this article</p><p>* Correspondence: <a href=mailto:yeonsu58@korea.kr>yeonsu58@korea.kr</a></p><p>† Equal contributors</p><p>Full list of author information is available at the end of the article</p><p>© 2014 Lee et al.; licensee BioMed Central Ltd. This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href=http://creativecommons.org/licenses/by/4.0%29>http://creativecommons.org/licenses/by/4.0)</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver (<a href=http://creativecommons.org/publicdomain/zero/1.0/>http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated.</p><p>13k Accesses
53 Citations
3 Altmetric
Metrics details</p><div class=abstract><h2 id=abstract>Abstract
<a class=anchor href=#abstract>#</a></h2><p>Hanwoo (Korean cattle) is the native, taurine type of cattle breed of Korea and its history as a draft animal dates back to 5000 Years. In earlier times Hanwoo was used extensively for farming, transportation. Over the period of time, Hanwoo has changed to be meat type cattle. Full-scale production of Hanwoo as meat-type cattle has occurred since 1960s with the rapid growth of the Korean economy. Hanwoo is one of the most economically important species in Korea as it is a significant source of nutrition to the Korean people. Hanwoo beef is the most cherished food of Korea. One of the main goals of researchers is to increase the meat quality, quantity and taste of the beef. In this review we describe the origin, domestication of Hanwoo cattle and breeding program initiated from 1980’s. Moreover the advent of technological advancement had provided us a platform to perform genome wide selection on economic traits and its implementation into traditional breeding programs.</p></div><p><strong>Keywords:</strong> Hanwoo (Korean cattle), Origin, Domestication and breeding program</p><h2 id=introduction>Introduction
<a class=anchor href=#introduction>#</a></h2><p>Hanwoo (Korean cattle) is the native, taurine type, small sized cattle breed of Korea and its history as a draft animal dates back to 5000 Years. In earlier times Hanwoo was used extensively for farming, transportation and for religious sacrifice. Over the period of time Hanwoo cattle has evolved to be meat type cattle. Full-scale production of Hanwoo as meat-type cattle has occurred since 1960s with the rapid growth of the Korean economy. Despite its high price, beef from Hanwoo cattle is very popular among both Koreans and foreigners. Hanwoo beef is known for its marbled fat, tenderness, juiciness and characteristic flavor. There are four breeds of Hanwoo in Korea viz. brown Hanwoo, brindle Hanwoo, black Hanwoo and jeju black (Figure 1). Among all these colour variants brown is the most common one (Figure 1). National Institute of Animal Science along with the small stakeholders has taken an initiative for a better management and improvement of Hanwoo cattle. Hanwoo Experiment Station (A Research Station of National Institute of Animal Science) in Pyeongchang-gun, Gangwon-do, Korea is dedicated exclusively for research on Hanwoo cattle. Along with Hanwoo experiment station there are other institutes and universities as well which are working towards the betterment of this native cattle breed.</p><p><img src=/img/hanwoo-cattle_fig1.jpg alt="Pictures of four Hanwoo cattle. (A) Brown Hanwoo, (B) Brindle Hanwoo, (C) Black Hanwoo and (D) Jeju black Hanwoo."></p><figcaption><p><strong>Figure 1</strong>
figure1
Pictures of four Hanwoo cattle. <strong>(A)</strong> Brown Hanwoo, <strong>(B)</strong> Brindle Hanwoo, <strong>(C)</strong> Black Hanwoo and <strong>(D)</strong> Jeju black Hanwoo.</p></figcaption><p>Full size image</p><p>Korea along with Hanwoo has a market for imported beef from Australia, USA, New Zealand, Canada and Mexico as well (KMTA, [1]). When Hanwoo beef was compared with the Australian Angus, it showed characteristic compositional and quality differences that should result from genetic and environmental differences between them [2]. Hanwoo is lower in cholesterol compared to other beef. It also has a higher omega 3 fatty acid count which makes it healthier than beef from other cattle. Taste-wise, it’s very soft, juicy, and delicious. The marbling in Hanwoo steak is excellent as well and it has the right balance of meat and fat. Korean consumers decided their overall acceptability of Hanwoo beef are as follows: weights of tenderness 55%, juiciness 18%, and flavor-likeness 27% [3]. No wonder Korean consumers, despite the high price prefer Hanwoo beef to the imported one. In order to produce high quality Hanwoo beef along with specially designed breeding programs special care and feed strategies are deployed.</p><p>The major goal of researchers is to increase both the quality (Marbling, tenderness and flavor) and the quantity (Carcass weight) of the meat to benefit the Hanwoo beef industry. Therefore, the current selection index used in Korea Proven Bulls (KPN) program is based on 1) carcass weight on marbling score 2) the comparison on carcass weight. In order to support the beef industry along with the conventional selection program, the genomic selection is also being incorporated in selection and breeding programs. In the present review we discussed the origin, domestication, breeding strategies and genomic selection in Hanwoo cattle.</p><h2 id=origin-and-domestication>Origin and Domestication
<a class=anchor href=#origin-and-domestication>#</a></h2><p>Origin and domestication of not just Hanwoo but all north-east asian cattle is a topic of discussion among researchers. Researchers are of divided view when it comes to ancestry and domestication of Hanwoo. There are some studies on the origin and ancestry of this cattle breed. Han S. W., [4] suggested Hanwoo to be a cross between zebu and taurine cattle, which migrated to Korea from Mongolia via north china while Lee & Pollack [5] proposed Hanwoo to have originated as a hybrid from auroch and zebu cattle. Yoon et al. [6] suggested an independent domestication event for Hanwoo. Mannen et al. [7] also suggested the independent mitochondrial origin in Asian cattle. Their study included Mongolian, Korean and Japanese cattle. In a recent study McTavish et al. [8] suggested that the asian cattle (Hanwoo and Japanese Black) to be of hybrid taurine–indicine origin.We carried out cattle diversity analysis which showed a clear differentiation of asian taurine from western taurine cattle (Figure 2B-2C). Based on 50 K SNP chip data we computed a neighbor joining phylogenetic tree where we observed a clear partitioning for African taurine, European taurine, Asian taurine and Zebu cattle. The tree clearly clustered Korean cattle breeds on a separate node than the European taurine cattle breeds. This might suggest a separate domestication event for north-east asian cattle. Also, mitochondria based studies have identified a haplotype T4 that is observed in north east asian cattle breeds but not in other breeds (Mannen et al., [7]). The PCA plot also clearly separated Hanwoo and Wagyu (Japanese cattle) from Hereford and angus cattle thus supporting the inference from neighbor joining tree.</p><p><img src=/img/hanwoo-cattle_fig2.jpg alt="Comparison of Hanwoo compared to other taurine cattle."></p><figcaption><p><strong>Figure 2</strong>
figure2
<strong>Comparison of Hanwoo compared to other taurine cattle. (A)</strong> Model-based population assignment for individuals based on the 50K SNP panel using STRUCTURE (K=3), YBH: Yanbian, CHB: Korean brindle, CS: Chosun, JBB: Jeju black, Ag: Angus, BS: Brown swiss, LM: Limousine, HF: Hereford, H: Holstein, BR: Brahman, HW: Hanwoo <strong>(B)</strong> A neighbor joining tree based on Pairwise Fst values, <strong>(C)</strong> Principal component analysis capturing a clear difference of north-east asian cattle.</p></figcaption><p>Full size image</p><h2 id=population-genetic-parameters-in-hanwoo>Population genetic parameters in Hanwoo
<a class=anchor href=#population-genetic-parameters-in-hanwoo>#</a></h2><p>Linkage disequilibrium (LD) and effective population size (Ne) are important genetic parameters to understand population structure for optimal breeding program design. Lee et al., [9] performed a LD analysis and studied Ne for the entire Korean Hanwoo cattle genome, which was the first LD map and effective population size estimate ever calculated for this breed. A panel of 4,525 markers was used in the final LD analysis. The pairwise r2 statistic of SNPs up to 50 Mb apart across the genome was estimated. A mean value of r2 = 0.23 was observed in pairwise distances of &lt;25 kb and dropped to 0.1 at 40 to 60 kb, which is similar to the average intermarker distance used in this study. The proportion of SNPs in useful LD (r2 ≥ 0.25) was 20% for the distance of 10 and 20 kb between SNPs. Analyses of past effective population size estimates based on direct estimates of recombination rates from SNP data demonstrated a decline in effective population size to Ne = 98.1 up to three generations ago (Figure 3). In this population genetic parameter (LD and Ne) study, the Ne in Hanwoo was larger overall than in North American Holstein cattle. In particular, the Ne of Hanwoo dramatically decreased 25 generations ago, this is when official Hanwoo breeding programs were initiated. Effect of selection pressure on Ne in the Hanwoo population was evident from this study.</p><p><img src=/img/hanwoo-cattle_fig3.jpg alt="Effective population size from chromosome of the past Hanwoo (Korean cattle) population."></p><figcaption><p><strong>Figure 3</strong>
figure3
<strong>Effective population size from chromosome of the past Hanwoo (Korean cattle) population.</strong> Effective population size was estimated in each autosomal chromosome. The generation in the past population was calculated as 1/2c. Ne is plotted against the past generation up to 500 generation. (Extracted from [10]).
Full size image</p><h2 id=hanwoo-breeding-program>Hanwoo breeding program
<a class=anchor href=#hanwoo-breeding-program>#</a></h2><p>Marbling score (MS), carcass weight (CWT), eye muscle area (EMA) and back fat thickness (BF) are the traits to impact Korean beef industry the most. Therefore the selection index for Korea Proven Bulls uses eye muscle area on marbling score and their comparison on CWT.</p><p>The first genetic breeding program called “Hanwoo-Gaeryang-Danji (HGD)” was initiated by Ministry of Agriculture and Forestry (MAF) in 1979. However, due to lack of awareness in the farmers to record the data and scarcity of the trained emulators, the program didn’t meet its objective. In 1999, a new program called “Hanwoo-Gaeryang-Nongga (HGN)” was introduced. Whereas HGD focused on a particular province HGN concentrated on individual farms. The cows from HGN program are used as major breeding stock for “Hanwoo Performance and Progeny Test (HPPT)” program (MAF, [11]). Prior to the HPPT program, bulls to be used for artificial insemination were selected solely on the basis of phenotypes. It was in 1987 that the first proven bull was produced using HPPT program. As shown in Figure 4, the HPPT program was a two stage selection program. The first stage was a performance test for the young bulls and the second stage was progeny test of selected young bulls. Young bull calves of current proven bulls were harvested at the age of 6 months from HGN belonging to HGD based on their phenotypic values and underwent a performance test up to age of 12 months. At this stage young bulls were selected based on a combination of their breeding values of weight at 12 months and average daily gain estimated from its performance test records. In the progeny test, cows from the HGN program were inseminated with the semen from young bulls and the male calves were harvested from farms and raised for the station progeny test. These steers were raised in a group until slaughter at 24 months. Carcass data comprised EMA, BF, MAR and CWT. Based on the progeny test results 20 bulls are selected for the AI program. Selection is based on a selection index which used weighted breeding values for BF, MAR and EMA.</p><p><img src=/img/hanwoo-cattle_fig4.jpg alt="Pictorial representation of Hanwoo performances and progeny test (HPPT) program."></p><figcaption><p><strong>Figure 4</strong>
figure4
<strong>Pictorial representation of Hanwoo performances and progeny test (HPPT) program.</strong></p></figcaption><p>Full size image
Park et al. [12] carried out a study to determine the trend of improvement as well as the estimation of genetic parameters of the traits being used for seedstock selection based on the data collected from the past. In his study he concluded that the performance and progeny test based selection system of Korea is good enough to accommodate circumstances where fewer sires are used on many more cows. Although progeny tests take longer and cost more, they seem to be appropriate under the circumstances of the domestic market with its higher requirement for better meat quality. The study suggested accumulative data collection, genetic evaluation model development, revision of selection indices, as well as cooperation among farms, associations, National Agricultural Cooperative Federation, universities, research institutes and government agencies must be applied to the Hanwoo selection program. According to the Park’s study, the current progeny tested evaluation runs every 6 months to select KPN in Korea. The breeding program has achieved a substantial genetic improvement for CWT and EMA. However there has been just a slightly negative genetic response for MS with an -0.036 of genetic response per year [12].</p><p>The heritability and genetic correlation are important genetic parameters in breeding program. In Hanwoo population, heritability and genetic correlation were estimated using carcass data from National Hanwoo progeny tested population (Figure 5A). A negative genetic correlation was observed between BF and EMA (-0.2) and MS and BF (-0.02) while positive correlation was observed in all other traits. A high positive correlation was observed between EMA and CWT (0.45) indicating that a greater EMA is associated with a higher production of CWT. The value of correlation between EMA and CWT in our study is more than that reported by and Koch et al. [13] and less as reported by Baik et al. [14] and Cundiff et al. [15]. High estimated heritabilities for BF (0.5), EMA (0.41), MS (0.4) and moderate for CWT (0.33) were observed. This indicates that in Hanwoo cattle a considerable genetic variability still exits which may be used for the improvement of carcass traits. Estimation of heritability, from genome-wide SNPs has recently attracted interest and offers several advantages over traditional pedigree-based methods. A comparison of pedigree and genotype based heritability for carcass traits was thus drawn (Figure 5B). The amount of variation that could be accounted for by SNP genotypes was concordant with pedigree-based heritabilities and varied from very low for CWT to medium-high for BF.</p><p><img src=/img/hanwoo-cattle_fig5.jpg alt="Heritability and genetic correlation for carcass trait in Hanwoo."></p><figcaption><p><strong>Figure 5</strong>
figure5
<strong>Heritability and genetic correlation for carcass trait in Hanwoo. Data are taken from a report for Hanwoo genetic improvement meeting (2013). (A)</strong> Along the diagonal (From upper left) are heritability values, upper triangle shows variance for genetic correlation and lower triangle shows residual variance. <strong>(B)</strong> Estimate of heritability from a pedigree and genotype for carcass traits in Hanwoo.</p></figcaption><p>Full size image
We know conventional breeding methods rely on physical characteristics or phenotypes to calculate the breeding values (BV) of animals. The traditional methods are often not so accurate, inefficient and time consuming as most traits like meat quality are hard to measure and evident only when animal reaches maturity leading to a delay in verifying breeding results. With the advent of genome wide SNP panels we can overcome the drawbacks of the conventional breeding methods. Genome wide SNP panels will allow us to accurately and cost & time effectively determine the genomic estimated breeding values (GEBV) even in young animals and thus select animals at young age.</p><h2 id=genetic-architecture-for-carcass-trait-in-hanwoo>Genetic architecture for carcass trait in Hanwoo
<a class=anchor href=#genetic-architecture-for-carcass-trait-in-hanwoo>#</a></h2><p>Korean beef industry lay emphasis on meat yield and quality traits like carcass weight, marbling, intramuscular fat, eye muscle area. To identify the significant associations of SNPs with these traits can directly affect the selection of animals for a breeding program.</p><p>Genome-wide association study (GWAS) is a process for inspection and screening of detectable common genetic variants (single-nucleotide polymorphisms) in individuals to identify the variant(s) associated with the trait under study. There are several GWAS studies that are successfully conducted in livestock species. A recent study on Hanwoo cattle conducted by lee et al. (2013) identified 6 highly significant SNPs on chromosome 14 to be associated with carcass weight in Hanwoo cattle (Figure 6). This genome-wide association study was conducted to identify major loci that are significantly associated with carcass weight, and their effects, so as to provide an insight into the genetic architecture of carcass weight in Hanwoo. This genome-wide association study identified a major chromosome region ranging from 23 Mb to 25 Mb on chromosome 14 as being associated with carcass weight in Hanwoo. The most significant SNP detected was <em>BTB-01280026</em> (<em>P = 4.02 × 10-11</em>), located in the 25 Mb region on Bos taurus autosome 14 (BTA14). The most significant SNPs accounted for 6.73% to 10.55% of additive genetic variance, which is quite a large proportion of the total additive genetic variance. The most significant SNP (<em>BTB-01280026; P = 4.02 × 10-11</em>) had 16.96 kg of allele substitution effect, and the second most significant SNP (<em>Hapmap27934-BTC-065223; P = 4.04 × 10-11</em>) had 18.06 kg of effect on carcass weight, which correspond to 44% and 47%, respectively, of the phenotypic standard deviation for carcass weight in Hanwoo cattle. Our results demonstrated that carcass weight was affected by a major Quantitative Trait Locus (QTL) with a large effect and by many SNPs with small effects that are normally distributed. As evident from manhattan plots of GWAS for four carcass traits shown in Figure 6 a highly significant association was observed for carcass weight on BTA14 while for other three traits there was no such association observed which shows the effect of many genes on a trait across all chromosomes.</p><p><img src=/img/hanwoo-cattle_fig6.jpg alt="Association of SNPs with the carcass traits in the Hanwoo shown as a Manhattan plot."></p><figcaption><p><strong>Figure 6</strong>
figure6
<strong>Association of SNPs with the carcass traits in the Hanwoo shown as a Manhattan plot.</strong></p></figcaption><p>Full size image
In order to determine if polygenic inheritance is a general phenomenon for carcass traits in Hanwoo population, we directly estimated the proportion of phenotypic variance explained by the common SNPs on each chromosomes. Proportion of variance attibuted to each chromosome averaged across four carcass traits against chromosome length (Figure 7) was calculated using genotype relationship matrix (GRM) on each chromosomes which supported the results shown in manhattan plots of GWAS. While for all other traits we can see most of the chromosomes having marginal effect but for CWT where Chromosome 14 is clearly showing a high effect on the trait. By calculating the proportion of the genome represented by each chromosome (not including the length of sex chromosomes), we tested for a correlation between the variance explained by each chromosome relative to its size. It was observed that several of the smaller chromosomes contributed less to the overall variance than several of the larger chromosomes, however trend was not significant.</p><p><img src=/img/hanwoo-cattle_fig7.jpg alt="Proportion of variance attibuted to each chromosome averaged across four carcass traits against chromosome length."></p><figcaption><p><strong>Figure 7</strong>
figure7
<strong>Proportion of variance attibuted to each chromosome averaged across four carcass traits against chromosome length.</strong></p></figcaption><p>Full size image
Also there are reports on GWAS studies on other traits of economical importance such as MS, IMF, EMA and sensory panel. Yi et al. [16] carried out a GWAS study to identify QTL for growth and carcass quality traits using high-density SNP panels. The data set comprised of 61 sires, their 486 steers, and the 54,001 SNP markers on 29 bovine autosomal chromosomes. Traits to be analyzed in this study were six growth and carcass quality traits including weaning weight (WWT), 365-d yearling weight (YWT), CWT after slaughter, BF, EMA, and MS. A total of 16(0), 18(4), 20(13), 11(23), 10(13) and 19(1) SNPs were detected at the 5% chromosome (genome)-wise level for the traits, WWT, YWT, CWT, BF, EMA and MS, respectively. Among the 148 SNPs, 91 SNPs had dominance effects, suggesting that dominance inheritance mode be considered in genetic improvement for growth and caracass quality in Hanwoo. Thirty five QTL regions on 17 Bos taurus chromosomes (i.e. BTA 3, 4, 5, 6, 7, 11, 12, 13, 14, 15, 16, 17, 18, 20, 23, 26, and 28) were detected. Strong evidence for the QTL influencing CWT were detected on BTA14. Also, the QTL for WWT, YWT, BF, and EMA were detected on BTA20. The GWAS studies could thus greatly help in developing the selection strategies for breeding programs.</p><h2 id=genomic-strategy-for-breeding-scheme>Genomic strategy for breeding scheme
<a class=anchor href=#genomic-strategy-for-breeding-scheme>#</a></h2><p>Genomic selection (GS) employs selection of an individual based on the genomic breeding value assessed through evaluating all the genetic markers located throughout the genome of that individual. Thus, the underlying principle of GS is to exploit the linkage disequilibrium of the QTL with one or more genetic marker(s) [17]. Molecular markers can be used to predict GBV of breeding animals by exploiting population-wide linkage disequilibrium between QTL and genetic markers spanned over the genome. The key–factor behind the success of genomic selection is to utilize the next generation sequencing approaches and the associated bioinformatics tools to identify the SNPs. Simulation studies in some domesticated species like, beef cattle, swine and chicken (Meuwissen et al., [18]) have suggested that the breeding values can be predicted with high accuracy using genetic markers alone but its validation is required especially in samples of the population different from that in which the effect of the markers was estimated. In genomic selection, the estimation of genomic breeding values is predicted to sum up all loci that are estimated based on phenotypes and genotypes in training population. This is particularly useful for traits that are very difficult to measure, such as marbling. The accuracy of these genomic predictions depends on the genetic architecture of the complex traits. For example, number of loci affecting the trait and distribution of their effects. In case of Hanwoo, MAS (Marker Associated Selection) and GS will provide a way to predict phenotype of marbling score as a preselection method that can be used in performance test [19].</p><h2 id=strategies-for-genomic-selection-program-in-hanwoo>Strategies for genomic selection program in Hanwoo
<a class=anchor href=#strategies-for-genomic-selection-program-in-hanwoo>#</a></h2><p>Application of genomic selection in Hanwoo preselection process such as performance test and then in progeny test in the breeding program might raise the genetic gain for marbling. A comparison of accuracy of GEBV and traditional EBV is tabulated in Table 1. In order to estimate the genomic breeding value, well organized reference population (Training data set) is to be build. The two most important features of reference population are 1) to representation of the entire Hanwoo population (all the variants) and the sample size. For the same, current progeny tested steers, selected proven bulls and candidate bulls from the progeny test program will be a reasonable population. Reference population must contain minimum of 5,000 animals. A genetic gain of 30% can be observed if we have a large reference population.</p><p><strong>Table 1 Accuracy of GEBV and traditional EBV estimated by 50 K SNP chip for Hanwoo cattle</strong></p><table><thead><tr><th>Traits</th><th>BLUP</th><th>GBLUP</th><th>Difference</th></tr></thead><tbody><tr><td>Eye muscle area (cm2)</td><td>0.11(0.08)</td><td>0.29(0.07)</td><td>0.18</td></tr><tr><td>Back fat thickness (mm)</td><td>0.11(0.08)</td><td>0.30(0.11)</td><td>0.19</td></tr><tr><td>Marbling score (1 ~ 9)</td><td>0.11(0.08)</td><td>0.27(0.12)</td><td>0.16</td></tr></tbody></table><p>Full size table</p><h2 id=conclusion>Conclusion
<a class=anchor href=#conclusion>#</a></h2><p>Recent advances in molecular biotechnology facilitate not only detection of genes that contribute to genetic variation of quantitative traits but also incorporation of genomic information into a conventional animal breeding program. The incorporated molecular information into GEBV may achieve an improvement of EBV and selection accuracy in cattle populations. In conclusion, we suggest incorporating molecular information into the conventional breeding programs to achieve better results in comparatively shorter time. Including molecular information will be a step further to achieve higher standards of Hanwoo beef.</p><h2 id=references>References
<a class=anchor href=#references>#</a></h2><ol><li>Korea Meat Trade Association (KMTA): <em>Information and Data of Agricultural Statistics Of Korea</em>. 2011, Accessed on January 12, 2011. <a href="http://kmta.or.kr/html/sub6-1.html?scode=6">http://kmta.or.kr/html/sub6-1.html?scode=6</a>,
Google Scholar</li><li>Jo C, Cho SH, Chang J, Nam KC: Keys to production and processing of Hanwoo beef: a perspective of tradition and science. <em>Anim Front</em>. 2012, 2 (4): 32-38. 10.2527/af.2012-0060.
Article
Google Scholar</li><li>Cho SH, Kim J, Park BY, Seong PN, Kang GH, Kim JH, Jung SG, Im SK, Kim DH: Assessment of meat quality properties and development of a palatability prediction model for Korean Hanwoo steer beef. <em>Meat Sci</em>. 2010, 86: 236-242.
CAS
Article
PubMed
Google Scholar</li><li>Han SW: <em>The Breed of Cattles</em>. 1996, 148-160. Sun-Jin publishing
Google Scholar</li><li>Lee C, Pollak EJ: Genetic antagonism between body weight and milk production in beef cattle. <em>J Anim Sci</em>. 2002, 80: 316-321.
CAS
PubMed
Google Scholar</li><li>Yoon DH, Park EW, Lee SH, Lee HK, Oh SJ, Cheong IC, Hong KC: Assessment of genetic diversity and relationships between Korean cattle and other cattle breeds by microsatellite loci. <em>J Anim Sci Technol (Kor)</em>. 2005, 47 (3): 341-354.
CAS
Article
Google Scholar</li><li>Mannen H, Kohno M, Nagata Y, Tsuji S, Bradley DG, Yeo JS, Nyamsamba D, Zagdsuren Y, Yokohama M, Nomura K, Amano T: Independent mitochondrial origin and historical genetic differentiation in North Eastern Asian cattle. <em>Mol Phylogenet Evol</em>. 2004, 32: 539-544.
CAS
Article
PubMed
Google Scholar</li><li>McTavish EJ, Decker JE, Schnabel RD, Taylor JF, Hillis DM: <em>New World Cattle Show Ancestry From Multiple Independent Domestication Events</em>. 2013, PNAS Early Edition
Google Scholar</li><li>Lee SH, Cho YM, Lim D, Kim HC, Choi BH, Park HS, Kim OH, Kim S, Kim TH, Yoon D, Hong SK: Linkage disequilibrium and effective population size in Hanwoo Korean Cattle. <em>Asian Australas J Anim Sci</em>. 2011, 24 (12): 1660-1665. 10.5713/ajas.2011.11165.
Article
Google Scholar</li><li>Lee SH, Julius VW, Lee SH, Lim DJ, Park EW, Gondro C, Yoon D, Oh SJ, Kim OH, Gibson J, Thompson J: Genome wide QTL mapping to identify candidate genes for carcass traits in Hanwoo (Korean Cattle). <em>Genes Genom</em>. 2012, 34 (1): 43-49. 10.1007/s13258-011-0081-6.
Article
Google Scholar</li><li>Ministry of Agriculture and Fishery (MAF): <em>Code of Practice For Development Programs In Agriculture And Forestry</em>. 1999, 40
Google Scholar</li><li>Park B, Choi T, Kim S, Oh S-H: National genetic evaluation (system) of Hanwoo (Korean Native Cattle). <em>Asian-Aust J Anim Sci</em>. 2013, 26 (2): 151-156. 10.5713/ajas.2012.12439.
CAS
Article
Google Scholar</li><li>Koch RM, Cundiff LV, Gregory EK: Heritabilitied and genetic, environmental and phenotypic correlations of carcass traits in a population of diverse biological types and their implications in selection programs. <em>J Anim Sci</em>. 1982, 55: 1319-1324.
Google Scholar</li><li>Baik DH, Hoque1 MA, Choe HS: Estimation of genetic and environmental parameters of carcass traits in hanwoo (korean native cattle) populations. <em>Asian-Aust J Anim Sci</em>. 2002, 15 (11): 1523-1526.
Article
Google Scholar</li><li>Cundiff LV, Gregory EK, Koch RM, Dickerson GE: Genetic relationships among growth and carcass traits of beef cattle. <em>J Anim Sci</em>. 1971, 33: 550-555.
CAS
PubMed
Google Scholar</li><li>Yi L: <em>Genome-Wide Association Study to Identify Qtl For Growth And Carcass Quality Traits In Korean Native Cattle, Hanwoo</em>. 2012
Google Scholar</li><li>Goddard ME, Hayes BJ: Mapping genes for complex traits in domestic animals and their use in breeding programmes. <em>Nat Rev Genet</em>. 2009, 10: 381-391.
CAS
Article
PubMed
Google Scholar</li><li>Meuwissen T, Hayes B, Goddard M: Prediction of total genetic value using genome-wide dense marker maps. <em>Genetics</em>. 2001, 157: 1819-1829.
CAS
PubMed
PubMed Central
Google Scholar</li><li>Lee SH, van der Werf JHJ, Hayes BJ, Goddard ME, Visscher PM: Predicting unobserved phenotypes for complex traits from whole-genome SNP data. <em>PLoS Genet</em>. 2008, 4: e1000231-
Article
PubMed
PubMed Central
Google Scholar</li></ol><p>Download references</p><h2 id=acknowledgements>Acknowledgements
<a class=anchor href=#acknowledgements>#</a></h2><p>This study was supported by awards from the AGENDA project (Grant no. PJ907008062013) and Molecular Breeding program (PJ0081882013) of Next Generation BIOGREEN21 project in the National Institute of Animal Science, RDA.</p><h2 id=author-information>Author information
<a class=anchor href=#author-information>#</a></h2><h3 id=author-notes>Author notes
<a class=anchor href=#author-notes>#</a></h3><h3 id=affiliations>Affiliations
<a class=anchor href=#affiliations>#</a></h3><p>RDA, Hanwoo Experiment Station, National Institute of Animal Science, Pyeong-Chang, 232-950, Korea
Seung-Hwan Lee, Aditi Sharma, Chang-Gwon Dang, Hyeong-Cheol Kim, Ki-Jun Jeon, Seong-Heum Yeon & Hee-Seol Kang</p><p>Animal Genetic and Breeding Division, National Institute of Animal Science, Cheon-An, Korea
Byoung-Ho Park, Seung-Soo Lee, Tae-Jeong Choi, Yeon-Ho Choy, Si-Dong Kim & Soo-Bong Park</p><h3 id=corresponding-author>Corresponding author
<a class=anchor href=#corresponding-author>#</a></h3><p>Correspondence to Seong-Heum Yeon.</p><h2 id=additional-information>Additional information
<a class=anchor href=#additional-information>#</a></h2><h3 id=competing-interests>Competing interests
<a class=anchor href=#competing-interests>#</a></h3><p>The authors declare that they have no competing interests.</p><h3 id=authors-contributions>Authors’ contributions
<a class=anchor href=#authors-contributions>#</a></h3><p>Conceived and designed the experiments: SHL BHP AS CGD SSL. Performed the experiments: TJC YHC HCK KJJ SDK. Analyzed the data: SHL BHP AS CGD TJC. Contributed reagents/materials/analysis tools: SHL SHY SBP HSK. Wrote the paper: SHL BHP AS CGD. Read and commented on the earlier drafts of this manuscript: SHL AS SHY SBP HSK. All authors read and approved the final manuscript.</p><p>Seung-Hwan Lee, Byoung-Ho Park, Aditi Sharma contributed equally to this work.</p><p>Authors’ original submitted files for images</p><p>Below are the links to the authors’ original submitted files for images.</p><p>Authors’ original file for figure 1
Authors’ original file for figure 2
Authors’ original file for figure 3
Authors’ original file for figure 4
Authors’ original file for figure 5
Authors’ original file for figure 6
Authors’ original file for figure 7</p><p>Lee et al. Journal of Animal Science and Technology 2014, 56:2 <a href=http://www.janimscitechnol.com/content/56/1/2>http://www.janimscitechnol.com/content/56/1/2</a></p><h2 id=rights-and-permissions>Rights and permissions
<a class=anchor href=#rights-and-permissions>#</a></h2><h3 id=open-access>Open Access
<a class=anchor href=#open-access>#</a></h3><p>This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made.</p><p>The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder.</p><p>To view a copy of this licence, visit <a href=https://creativecommons.org/licenses/by/4.0/>https://creativecommons.org/licenses/by/4.0/</a>.</p><p>The Creative Commons Public Domain Dedication waiver (<a href=https://creativecommons.org/publicdomain/zero/1.0/>https://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</p><h2 id=reprints-and-permissions>Reprints and Permissions
<a class=anchor href=#reprints-and-permissions>#</a></h2><h3 id=about-this-article>About this article
<a class=anchor href=#about-this-article>#</a></h3><p>Verify currency and authenticity via CrossMark
Cite this article
Lee, SH., Park, BH., Sharma, A. et al. Hanwoo cattle: origin, domestication, breeding strategies and genomic selection. J Anim Sci Technol 56, 2 (2014). <a href=https://doi.org/10.1186/2055-0391-56-2>https://doi.org/10.1186/2055-0391-56-2</a></p><p>Download citation</p><p>Received
07 April 2014</p><p>Accepted
07 April 2014</p><p>Published
15 May 2014</p><p>DOI
<a href=https://doi.org/10.1186/2055-0391-56-2>https://doi.org/10.1186/2055-0391-56-2</a></p><h3 id=share-this-article>Share this article
<a class=anchor href=#share-this-article>#</a></h3><p>Anyone you share the following link with will be able to read this content:</p><h3 id=get-shareable-link>Get shareable link
<a class=anchor href=#get-shareable-link>#</a></h3><p>Provided by the Springer Nature SharedIt content-sharing initiative</p><h2 id=keywords>Keywords
<a class=anchor href=#keywords>#</a></h2><p>Hanwoo (Korean cattle)
Origin
Domestication and breeding program</p><p>Download PDF
Sections
Figures
References
Abstract
Introduction
Conclusion
References
Acknowledgements
Author information
Additional information
Authors’ original submitted files for images
Rights and permissions
About this article
Advertisement</p><p>Journal of Animal Science and Technology
ISSN: 2055-0391</p><h2 id=contact-us>Contact us
<a class=anchor href=#contact-us>#</a></h2><h3 id=submission-enquiries>Submission enquiries
<a class=anchor href=#submission-enquiries>#</a></h3><p>Access here and click</p><h3 id=contact-us-1>Contact Us
<a class=anchor href=#contact-us-1>#</a></h3><p>General enquiries: <a href=mailto:info@biomedcentral.com>info@biomedcentral.com</a>
Read more on our blogs
Receive BMC newsletters
Manage article alerts
Language editing for authors
Scientific editing for authors
Policies
Accessibility
Press center
Support and Contact
Leave feedback
Careers
Follow BMC</p><p>BMC Twitter page
BMC Facebook page
BMC Weibo page
By using this website, you agree to our Terms and Conditions, California Privacy Statement, Privacy statement and Cookies policy. Manage cookies/Do not sell my data we use in the preference centre.</p><p>Springer Nature
© 2021 BioMed Central Ltd unless otherwise stated. Part of Springer Nature.</p><h2 id=submit-your-next-manuscript-to-biomed-central-and-take-full-advantage-of>Submit your next manuscript to BioMed Central and take full advantage of
<a class=anchor href=#submit-your-next-manuscript-to-biomed-central-and-take-full-advantage-of>#</a></h2><ul><li>Convenient online submission</li><li>Thorough peer review</li><li>No space constraints or color figure charges</li><li>Immediate publication on acceptance</li><li>Inclusion in PubMed, CAS, Scopus and Google Scholar</li><li>Research which is freely available for redistribution</li></ul><p>Submit your manuscript at <a href=https://www.biomedcentral.com/submit>www.biomedcentral.com/submit</a></p><div class=footnotes role=doc-endnotes><hr><ol><li id=fn:1><p>Hanwoo Experiment Station, National Institute of Animal Science, RDA, Pyeong-Chang 232-950, Korea&#160;<a href=#fnref:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref1:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref2:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref3:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref4:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref5:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref6:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:2><p>Animal Genetic and Breeding Division,
National Institute of Animal Science, Cheon-An, Korea.&#160;<a href=#fnref:2 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref1:2 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref2:2 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref3:2 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref4:2 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref5:2 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li></ol></div></article><footer class=book-footer><div class="flex flex-wrap justify-between"></div><script>(function(){function e(e){const t=window.getSelection(),n=document.createRange();n.selectNodeContents(e),t.removeAllRanges(),t.addRange(n)}document.querySelectorAll("pre code").forEach(t=>{t.addEventListener("click",function(){e(t.parentElement),navigator.clipboard&&navigator.clipboard.writeText(t.parentElement.textContent)})})})()</script></footer><div class=book-comments></div><label for=menu-control class="hidden book-menu-overlay"></label></div><aside class=book-toc><div class=book-toc-content><nav id=TableOfContents><ul><li><a href=#abstract>Abstract</a></li><li><a href=#introduction>Introduction</a></li><li><a href=#origin-and-domestication>Origin and Domestication</a></li><li><a href=#population-genetic-parameters-in-hanwoo>Population genetic parameters in Hanwoo</a></li><li><a href=#hanwoo-breeding-program>Hanwoo breeding program</a></li><li><a href=#genetic-architecture-for-carcass-trait-in-hanwoo>Genetic architecture for carcass trait in Hanwoo</a></li><li><a href=#genomic-strategy-for-breeding-scheme>Genomic strategy for breeding scheme</a></li><li><a href=#strategies-for-genomic-selection-program-in-hanwoo>Strategies for genomic selection program in Hanwoo</a></li><li><a href=#conclusion>Conclusion</a></li><li><a href=#references>References</a></li><li><a href=#acknowledgements>Acknowledgements</a></li><li><a href=#author-information>Author information</a><ul><li><a href=#author-notes>Author notes</a></li><li><a href=#affiliations>Affiliations</a></li><li><a href=#corresponding-author>Corresponding author</a></li></ul></li><li><a href=#additional-information>Additional information</a><ul><li><a href=#competing-interests>Competing interests</a></li><li><a href=#authors-contributions>Authors’ contributions</a></li></ul></li><li><a href=#rights-and-permissions>Rights and permissions</a><ul><li><a href=#open-access>Open Access</a></li></ul></li><li><a href=#reprints-and-permissions>Reprints and Permissions</a><ul><li><a href=#about-this-article>About this article</a></li><li><a href=#share-this-article>Share this article</a></li><li><a href=#get-shareable-link>Get shareable link</a></li></ul></li><li><a href=#keywords>Keywords</a></li><li><a href=#contact-us>Contact us</a><ul><li><a href=#submission-enquiries>Submission enquiries</a></li><li><a href=#contact-us-1>Contact Us</a></li></ul></li><li><a href=#submit-your-next-manuscript-to-biomed-central-and-take-full-advantage-of>Submit your next manuscript to BioMed Central and take full advantage of</a></li></ul></nav></div></aside></main></body></html>