<!doctype html><html lang=en dir=ltr><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="Skip to main content Advertisement SpringerLink
Search Log in Original Article Open Access Published: 19 November 2018 Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers
Calvin M. Runnels, Kathryn A. Lanier, Justin Krish Williams, Jessica C. Bowman, Anton S. Petrov, Nicholas V. Hud & Loren Dean Williams Journal of Molecular Evolution volume 86, pages 598–610 (2018)Cite this article
3227 Accesses 20 Citations 16 Altmetric Metrics details
Abstract # Life as we know it requires three basic types of polymers: polypeptide, polynucleotide, and polysaccharide."><meta name=theme-color content="#FFFFFF"><meta name=color-scheme content="light dark"><meta property="og:title" content="Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers"><meta property="og:description" content="Skip to main content Advertisement SpringerLink
Search Log in Original Article Open Access Published: 19 November 2018 Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers
Calvin M. Runnels, Kathryn A. Lanier, Justin Krish Williams, Jessica C. Bowman, Anton S. Petrov, Nicholas V. Hud & Loren Dean Williams Journal of Molecular Evolution volume 86, pages 598–610 (2018)Cite this article
3227 Accesses 20 Citations 16 Altmetric Metrics details
Abstract # Life as we know it requires three basic types of polymers: polypeptide, polynucleotide, and polysaccharide."><meta property="og:type" content="article"><meta property="og:url" content="https://csmeyns.github.io/derivatives/docs/biopolymers/"><meta property="article:section" content="docs"><title>Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers | Derivatives</title><link rel=manifest href=/derivatives/manifest.json><link rel=icon href=/derivatives/favicon.png type=image/x-icon><link rel=stylesheet href=/derivatives/book.min.1a6fac2df464e68d664cb846d822c678cd622692d6fae24ee18901a08bfc8eb6.css integrity="sha256-Gm+sLfRk5o1mTLhG2CLGeM1iJpLW+uJO4YkBoIv8jrY=" crossorigin=anonymous><script defer src=/derivatives/flexsearch.min.js></script>
<script defer src=/derivatives/en.search.min.a67a3095c8f0653224f7610bdabe6a4ca5e39404c4ba9bcdfd70d6644501f4f3.js integrity="sha256-pnowlcjwZTIk92EL2r5qTKXjlATEupvN/XDWZEUB9PM=" crossorigin=anonymous></script></head><body dir=ltr><input type=checkbox class="hidden toggle" id=menu-control>
<input type=checkbox class="hidden toggle" id=toc-control><main class="container flex"><aside class=book-menu><div class=book-menu-content><nav><h2 class=book-brand><a class="flex align-center" href=/derivatives/><span>Derivatives</span></a></h2><div class=book-search><input type=text id=book-search-input placeholder=Search aria-label=Search maxlength=64 data-hotkeys=s/><div class="book-search-spinner hidden"></div><ul id=book-search-results></ul></div><ul><li><a href=https://csmeyns.github.io/derivatives/docs/readme/>README</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/architecture-human-orc/>Architecture of the Human Origin Recognition Complex*</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/amyloid-%CE%B2/>Origins of amyloid-β</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/arsenic/>Effects of arsenic and heavy metals on metabolic pathways in cells of human origin: Similarities and differences</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/back-to-future/>Back to the future: origins and directions of the “Agile Manifesto” – views of the originators</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/biopolymers/ class=active>Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers</a></li><li><input type=checkbox id=section-4e626a39f813bdfb0b0f11621a8d97e6 class=toggle>
<label for=section-4e626a39f813bdfb0b0f11621a8d97e6 class="flex justify-between"><a href=https://csmeyns.github.io/derivatives/docs/cc-by/>CC-BY</a></label><ul><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-2-summary/>Creative Commons License Deed Attribution 2.0 Generic (CC BY 2.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-2/>Creative Commons Legal Code Attribution 2.0</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-3-summary/>Creative Commons License Deed Attribution 3.0 Unported (CC BY 3.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-3/>Creative Commons Attribution 3.0 Unported</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-4-summary/>Creative Commons License Deed Attribution 4.0 International (CC BY 4.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-4-de/>Creative Commons Konzessionsurkunde Namensnennung 4.0 International (CC BY 4.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-4/>Creative Commons Legal Code Attribution 4.0 International</a></li></ul></li><li><a href=https://csmeyns.github.io/derivatives/docs/charon-smooth-plains/>The nature and origin of Charon's smooth plains</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/chinese-word-cell/>Origin of the Chinese word for “cell”: an unusual but wonderful idea of a mathematician</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/city-origins/>City origins</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/counterfeit-versus-original/>Counterfeit versus original patronage: Do emotional brand attachment, brand involvement, and past experience matter?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/creatine/>ON THE ORIGIN OF CREATINE</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/detecting-moisture/>Detecting the origins of moisture over Southeast China: Seasonal variation and heavy Rainfall</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/differentiation-spirits/>Differentiation Between Spirits According to Their Botanical Origin</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/food-taboos/>Food taboos: their origins and purposes</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/ganges/>Ganges: special at its origin</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/given-that-detailed/>Given that the detailed original criteria for deliberate practice have not changed, could the understanding of this complex concept have improved over time? A response to Macnamara and Hambrick (2020)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/hanwoo-cattle/>Hanwoo cattle: origin, domestication, breeding strategies and genomic selection</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/hoarding/>The developmental origins of hoarding disorder in adolescence: a longitudinal clinical interview study following an epidemiological survey</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/how-to-write/>How to write an original article for the *Journal of Orthopaedics and Traumatology*</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/in-out-qinghai-tibet-plateau/>“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/interview-patrick-o-brown/>An interview with Patrick O Brown on the origins and future of open access</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/manipulated-photos/>Can people identify original and manipulated photos of real-world scenes?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/neighborhood-spam/>Neighborhoods and bands: an analysis of the origins of spam</a></li><li><input type=checkbox id=section-18bd70f4fad61eb2742d114842aa84cf class=toggle>
<label for=section-18bd70f4fad61eb2742d114842aa84cf class="flex justify-between"><a href=https://csmeyns.github.io/derivatives/docs/letters/>Letters</a></label><ul><li><a href=https://csmeyns.github.io/derivatives/docs/letters/non-neanderthal/>Non-Neanderthal Origin of the HLA-DPB1*0401</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/letters/reply-to-ding/>Reply to Ding et al.: Non-Neanderthal Origin of the HLA-DPB1*0401</a></li></ul></li><li><a href=https://csmeyns.github.io/derivatives/docs/origin-of-the-ectopic-beat/>What is the origin of the ectopic beat?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/origins/>Origins</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/patient-safety/>The 5th anniversary of “Patient Safety in Surgery” – from the Journal’s origin to its future vision</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/rendang/>Tracing the origins of rendang and its development</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/soft-robot/>Assessing the Impact of an Original Soft Communicative Robot in a Nursing Home in Japan: Will Softness or Conversations Bring more Smiles to Older People?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/softwaretest-originaldaten/>Softwaretest mit Originaldaten Eine Analyse aus Sicht des Datenschutzes</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/spectacular-submarine/>Origin of spectacular fields of submarine sediment waves around volcanic islands</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/urban-conservation/>Urban Conservation Through Chinese Local Original Architecture: The Preservation and Renovation of Zhongshan Road in Hangzhou</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/vertebrate-gills/>The Origin of Vertebrate Gills</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/volcanic-origin-1741/>Volcanic origin of the 1741 Oshima-Oshima tsunami in the Japan Sea</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/what-is-morita-therapy/>What is Morita Therapy? The Nature, Origins, and Cross-Cultural Application of a Unique Japanese Psychotherapy</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/wildcat-reappearance/>Revealing the origin of wildcat reappearance after presumed long-term absence</a></li></ul></nav><script>(function(){var e=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(){localStorage.setItem("menu.scrollTop",e.scrollTop)}),e.scrollTop=localStorage.getItem("menu.scrollTop")})()</script></div></aside><div class=book-page><header class=book-header><div class="flex align-center justify-between"><label for=menu-control><img src=/derivatives/svg/menu.svg class=book-icon alt=Menu></label>
<strong>Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers</strong>
<label for=toc-control><img src=/derivatives/svg/toc.svg class=book-icon alt="Table of Contents"></label></div><aside class="hidden clearfix"><nav id=TableOfContents><ul><li><a href=#abstract>Abstract</a></li><li><a href=#chemical-cousins>Chemical Cousins</a></li><li><a href=#biosynthesis>Biosynthesis</a></li><li><a href=#living-dangerously>Living Dangerously</a><ul><li><a href=#fold>Fold</a></li><li><a href=#assemble>Assemble</a></li><li><a href=#degrade>Degrade</a></li></ul></li><li><a href=#complements-to-the-chef>Complements to the Chef</a><ul><li><a href=#complementarity>Complementarity</a></li><li><a href=#perturbationclarification>Perturbation—Clarification</a></li></ul></li><li><a href=#separated-at-birth>Separated at Birth</a><ul><li><a href=#adding-it-up>Adding It Up</a></li><li><a href=#functional-distance>Functional Distance</a></li></ul></li><li><a href=#fraternal-twins-dna-and-rna>Fraternal Twins: DNA and RNA</a></li><li><a href=#nature-chose-phosphate>Nature Chose Phosphate</a></li><li><a href=#molecules-in-mutualism>Molecules in Mutualism</a><ul><li><a href=#levels-of-mutualism>Levels of Mutualism</a></li><li><a href=#molecules>Molecules</a></li></ul></li><li><a href=#origins-of-biopolymersorigins-of-life>Origins of Biopolymers—Origins of Life</a><ul><li><a href=#why-biopolymers>Why Biopolymers?</a></li><li><a href=#origins-of-biopolymers>Origins of Biopolymers</a></li><li><a href=#selfish-molecules>Selfish Molecules</a></li><li><a href=#losers>Losers</a></li></ul></li><li><a href=#conclusion>Conclusion</a></li><li><a href=#references>References</a></li><li><a href=#acknowledgements>Acknowledgements</a></li><li><a href=#author-information>Author information</a><ul><li><a href=#affiliations>Affiliations</a></li><li><a href=#corresponding-author>Corresponding author</a></li></ul></li><li><a href=#ethics-declarations>Ethics declarations</a><ul><li><a href=#conflict-of-interest>Conflict of interest</a></li></ul></li><li><a href=#rights-and-permissions>Rights and permissions</a></li><li><a href=#reprints-and-permissions>Reprints and Permissions</a></li><li><a href=#about-this-article>About this article</a></li><li><a href=#download-citation>Download citation</a><ul><li><a href=#share-this-article>Share this article</a></li><li><a href=#get-shareable-link>Get shareable link</a></li></ul></li><li><a href=#keywords>Keywords</a></li><li><a href=#advertisement>Advertisement</a></li></ul></nav></aside></header><article class=markdown><h1><a href=/derivatives/docs/biopolymers/>Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers</a></h1><h2>Calvin M. Runnels, Kathryn A. Lanier, Justin Krish Williams, Jessica C. Bowman, Anton S. Petrov, Nicholas V. Hud & Loren Dean Williams</h2><p>Skip to main content
Advertisement
SpringerLink</p><p>Search
Log in
Original Article
Open Access
Published: 19 November 2018
Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers</p><p>Calvin M. Runnels, Kathryn A. Lanier, Justin Krish Williams, Jessica C. Bowman, Anton S. Petrov, Nicholas V. Hud & Loren Dean Williams
Journal of Molecular Evolution volume 86, pages 598–610 (2018)Cite this article</p><p>3227 Accesses
20 Citations
16 Altmetric
Metrics details</p><div class=abstract><h2 id=abstract>Abstract
<a class=anchor href=#abstract>#</a></h2><p>Life as we know it requires three basic types of polymers: polypeptide, polynucleotide, and polysaccharide. Here we evaluate both universal and idiosyncratic characteristics of these biopolymers. We incorporate this information into a model that explains much about their origins, selection, and early evolution. We observe that all three biopolymer types are pre-organized, conditionally self-complementary, chemically unstable in aqueous media yet persistent because of kinetic trapping, with chiral monomers and directional chains. All three biopolymers are synthesized by dehydration reactions that are catalyzed by molecular motors driven by hydrolysis of phosphorylated nucleosides. All three biopolymers can access specific states that protect against hydrolysis. These protected states are <em>folded</em>, using self-complementary interactions among recurrent folding elements within a given biopolymer, or <em>assembled</em>, in associations between the same or different biopolymer types. Self-association in a hydrolytic environment achieves self-preservation. Heterogeneous association achieves partner-preservation. These universal properties support a model in which life’s polymers emerged simultaneously and co-evolved in a common hydrolytic milieu where molecular persistence depended on folding and assembly. We believe that an understanding of the structure, function, and origins of any given type of biopolymer requires the context of other biopolymers.</p></div><p>Keywords Polynucleotide · Protein · Carbohydrate · Folding · Assembly · Self-complementarity</p><p>Polymers are large molecules formed by covalently linking small monomers into chains. Polyethylene, for example, is a synthetic polymer with molecular formula (–C2H4–)n and molecular weight around 5 million Daltons that is used to make plastic bottles and bags. Living systems are united by their expression and utilization of three types of polymers. These three biopolymers, the subject of this paper, are polynucleotide (DNA and RNA), polypeptide (protein), and polysaccharide (polymerized sugars). Biopolymers have special properties that distinguish them from other polymers.</p><p>Biopolymers:</p><ol><li>(i) spontaneously fold and assemble into precise and highly elaborate yet fragile assemblies with meager stabilities,</li><li>(ii) spontaneously degrade by hydrolysis in the aqueous environments characteristic of biological systems,</li><li>(iii) are self-protective against hydrolysis (by folding) and partner-protective (by heterogeneous assembly).</li></ol><p>The three biopolymer types differ profoundly in their properties and functions. Polypeptide and polynucleotide dominate the functional and informational machineries of life, while polysaccharide is important in physical structure, energy storage, and recognition. The three biopolymers occupy discrete chemical spaces. Yet, biopolymers share many critical “universalities.” An understanding of structure, function, and origins of a given biopolymer type requires recognition of these universalities and the context of the other two biopolymer types.</p><p>Universalities of biopolymers include the ability to fold and assemble spontaneously. All three biopolymer types are self-complementary and pre-organized. Biopolymer self-complementarity is conditional and can be switched on and off by sequence, composition or linkage chemistry. Biopolymers are chemically unstable in aqueous media but persist for long periods via kinetic trapping. The depths of these kinetic traps are modulated by folding and assembly. Biopolymers are chiral and directional, and are synthesized by condensation dehydration using phosphorylated intermediates in reactions mediated by divalent cations and driven by phosphate dependent motors. Lipids are not discussed here because they are not covalent polymers. However, our conclusions would be unaltered by their inclusion.</p><h2 id=chemical-cousins>Chemical Cousins
<a class=anchor href=#chemical-cousins>#</a></h2><p>In contrast to the usual approach of analyzing each biopolymer separately, we focus first on phenomena that are common to all biopolymers (Table 1). We use the same nomenclature to describe a given phenomenon or characteristic without regard to the type of biopolymer.</p><p>Table 1 Biopolymer Universalities and Idiosyncrasies
Full size table</p><table><thead><tr><th>Attribute</th><th>Polynucleotide</th><th>Polypeptide</th><th>Polyglucose</th></tr></thead><tbody><tr><td>Primary proficiency</td><td>Maintain, record, and transduce information, catalyze chemical reactions</td><td>Catalyze and regulate chemi- cal reactions, provide physical structure</td><td>Provide physical structure, energy storage, and recognition</td></tr><tr><td>Conditional self-complementa- rityU<sup id=fnref:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Yes</td><td>Yes</td><td>Yes</td></tr><tr><td>Condition for self-complemen- tarity</td><td>Nucleotide sequence</td><td>Amino acid composition</td><td>Linkage stereochemistry (β- vs. α-anomer)</td></tr><tr><td>Small number of types of folding elementsU<sup id=fnref1:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Yes</td><td>Yes</td><td>Yes</td></tr><tr><td>Folding element identities</td><td>Nitrogenous bases</td><td>Peptide linkage</td><td>Cyclic glucose</td></tr><tr><td>Enzymatic capability</td><td>Moderate</td><td>High</td><td>Low</td></tr><tr><td>Sidechain diversity</td><td>Low: four planer nitrogenous bases</td><td>High: 20 amino acid sidechains</td><td>N/A: no sidechains</td></tr><tr><td>Sidechain complementarity</td><td>Yes, base pairing</td><td>No</td><td>No</td></tr><tr><td>Backbone self-complementarity</td><td>No: anionic, self-repulsive back- bone</td><td>Yes: neutral, cohesive backbone</td><td>Yes: neutral, cohesive backbone</td></tr><tr><td>Complementary hydrogen bondingU<sup id=fnref2:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Unipolar, coplanar</td><td>Unipolar, coplanar</td><td>Bipolar, non-planar</td></tr><tr><td>Net hydrogen bond polarity</td><td>Large excess of acceptors over donors</td><td>Equivalent number of acceptors and donors</td><td>Excess of acceptors over donors</td></tr><tr><td>Selective self-interaction of hydro- lyzed monomersU<sup id=fnref3:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>No</td><td>No</td><td>No</td></tr><tr><td>Backbone linearity</td><td>Yes</td><td>Yes</td><td>Sometimes</td></tr><tr><td>Strand directionalityU<sup id=fnref4:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Yes, 5′–3′</td><td>Yes, N to C</td><td>Yes, 1–4</td></tr><tr><td>Secondary structureU<sup id=fnref5:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Helices, bulges, stem-loops, pseu- doknots, etc</td><td>α-Helices, β-sheets and turns</td><td>Elongated fibers</td></tr><tr><td>Conformational constraintsU<sup id=fnref6:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>“Rigid nucleotides,” planar bases</td><td>Planar peptide, allowed regions of φφ space</td><td>Conformational preferences within and between cyclic glucose</td></tr><tr><td>Self-destruct mechanism</td><td>RNA: Yes (2′ hydroxyl) DNA: No</td><td>No</td><td>No</td></tr><tr><td>Required folding cofactors</td><td>Cations</td><td>None</td><td>None</td></tr><tr><td>Degradation by hydrolysisU<sup id=fnref7:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Yes</td><td>Yes</td><td>Yes</td></tr><tr><td>Polymerized by</td><td>Protein</td><td>Ribozyme RNP complex</td><td>Protein</td></tr><tr><td>Polymerization is dependent on divalent cationsU<sup id=fnref8:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Yes</td><td>Yes</td><td>Yes</td></tr><tr><td>Polymerization mechanismU<sup id=fnref9:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Condensation dehydration</td><td>Condensation dehydration</td><td>Condensation dehydration</td></tr><tr><td>Polymerization intermediatesU<sup id=fnref10:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Phosphorylated</td><td>Phosphorylated</td><td>Phosphorylated</td></tr><tr><td>Driver of polymerization motorU<sup id=fnref11:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup></td><td>Phosphate release</td><td>Phosphate release</td><td>Phosphate release</td></tr><tr><td>Retention of phosphate during polymerization</td><td>Yes</td><td>No</td><td>No</td></tr></tbody></table><h2 id=biosynthesis>Biosynthesis
<a class=anchor href=#biosynthesis>#</a></h2><p>Biopolymers are universally formed by condensation dehydration reactions, which release water (Fig. 1) to link well-defined and modest sets of monomers. Proteins are formed by condensation of 20 types of amino acids. Polynucleotides are formed by condensation of four types of nucleotides. Cellulose, the most abundant polymer in the biosphere, is formed by condensation of one type of monomer—glucose (McNamara et al. 2015). Complex cell-surface polysaccharides contain fewer than 20 different monosaccharides (Gabius et al. 2011). Here we will limit our discussion of polysaccharide to polyglucose, encompassing cellulose, glycogen, amylose, amylopectin, and chitin (acetylated glucose). However, our conclusions apply to polysaccharide in general.</p><p><img src=/img/biopolymers_fig1.png alt="Net reactions for biopolymer formation by condensation dehydration and biopolymer degradation by hydrolysis."></p><figcaption><p><strong>Fig. 1</strong> figure1 Net reactions for biopolymer formation by condensation dehydration and biopolymer degradation by hydrolysis. <strong>a</strong> Protein. <strong>b</strong> RNA. <strong>c</strong> Polysaccharide. All biopolymers are chiral and directional with distinctive ends. Chiral centers (stars, shown in polymers only) and strand directionalities (arrows) are indicated. Blue boxes indicate (in toto) the atoms involved in the synthesis/degradation reactions. (Color figure online)
Full size image</p></figcaption><p>Another universal property of biopolymers is their synthesis via phosphorylated or pyrophosphorylated intermediates (Figs. 1, 2) in reactions catalyzed by processive divalent cation-dependent motors. In translation, the motor is the ribosome (Trappl and Polacek 2011). In replication, the motor is DNA polymerase (Steitz 1999). In transcription, the motor is RNA polymerase (Fuchs 1976). In cellulose synthesis, the motor is glycosyl transferase (Kang et al. 1984; McNamara et al. 2015; Morgan et al. 2016). RNA and DNA retain a phosphate during polymerization, forming anionic phosphodiester linkages, while other polymers eliminate phosphate groups and form neutral linkages.</p><p><img src=/img/biopolymers_fig2.png alt="Intermediates in the biosynthesis of a protein and b polyglucose"></p><figcaption><p><strong>Fig. 2</strong> figure2 Intermediates in the biosynthesis of <strong>a</strong> protein and <strong>b</strong> polyglucose
Full size image</p></figcaption><h2 id=living-dangerously>Living Dangerously
<a class=anchor href=#living-dangerously>#</a></h2><p>It is a fundamental paradox of biochemistry that biopolymers are chemically unstable in their native environment—aqueous solution. All biopolymers in water spontaneously hydrolyze to the level of monomers at equilibrium in dilute aqueous solution.</p><h3 id=fold>Fold
<a class=anchor href=#fold>#</a></h3><p>A random coil biopolymer (an ensemble of configurations) folds, by spontaneous conversion to precise three-dimensional structures characterized by specific intramolecular interactions, low configurational entropy, assignment of functional groups to exact locations and orientations in three-dimensional space, and specific interactions between functional groups. Biopolymers that occupy these precise low entropy states, stabilized by self-interactions, are called “folded.” The ability to fold is a universality of biopolymers. In our definition, a single polymer type folds. Multiple polymers of one type assemble into homogeneous assemblies. Multiple polymers of different types assemble into heterogeneous assemblies.</p><p>Elaborate folding and assembly are emergent properties of polymerization, and are possible for polymers but not for monomers. It is a universal characteristic of biopolymers that their hydrolyzed monomers do not specifically self-interact. Monomer nucleosides (Ts’o 1974), amino acids, and sugars do not pair or engage in self-complementary interactions in water. The formation of G-quadruplexes by monomeric guanosine (Gellert et al. 1962) is an exception to this universality.</p><p>Finely controlled molecular interactions allow proteins to fold into domains (Porter and Rose 2012) or fibers (Shoulders and Raines 2009) composed primarily of α-helices and β-sheets (Fig. 3) (Pauling et al. 1951; Pauling and Corey 1951; Eisenberg 2003). RNAs can fold into large domains (Woodson 2011) composed of duplexes, tetraloops, junctions, bulges, and pseudoknots (Moore 1999), which can be rigid (Ban et al. 2000) or flexible (Wan et al. 2011). Complementary DNA sequences assemble to double helices (Watson and Crick 1953) approaching a meter in length with billions of base pairs. Polyglucose assembles to microfibrils of indeterminate length containing multiple chains (Valeri 2010; Cosgrove 2014).</p><p><img src=/img/biopolymers_fig3.png alt="The self-complementarity of peptide linkages is the dominant molecular interaction in protein secondary structures and in folded protein."></p><figcaption><p><strong>Fig. 3</strong>
figure3
The self-complementarity of peptide linkages is the dominant molecular interaction in protein secondary structures and in folded protein. <strong>a</strong> Antiparallel β-sheet. <strong>b</strong> Parallel β-sheet. <strong>c</strong> α-Helix. <strong>d</strong> A globular protein showing α-helices (violet) and β-sheets (green). <strong>e</strong> An amyloid fibril showing dominance of β-sheets for any essentially amino acid sequence. Hydrogen bonding polarities are indicated by arrows. Each peptide linkage donates one hydrogen bond and accepts one hydrogen bond. (Color figure online)
Full size image</p></figcaption><h3 id=assemble>Assemble
<a class=anchor href=#assemble>#</a></h3><p>Biopolymers form heterogenous assemblies, containing multiple types of biopolymers, with specific three-dimensional structures and intermolecular interactions. For example, the prokaryotic ribosome is a heterogeneous assembly of three large rRNAs and around 50 ribosomal proteins. Nucleosomes, which are specific to eukaryotes and some archaea, are protein–DNA assemblies. Protein–saccharide assemblies are critical in cell–cell communication, cell adhesion, and host–pathogen interactions.</p><h3 id=degrade>Degrade
<a class=anchor href=#degrade>#</a></h3><p>Biopolymers are ephemeral. It is a universality that biopolymers hydrolyze in aqueous media and suffer a variety of other chemical assaults in vivo and in vitro, spontaneously degrading to the level of monomers and beyond. In dilute aqueous solution, degradation of biopolymers to monomers is always favored in the thermodynamic sense. However, biopolymers can persist for extended periods of time in non-equilibrium states via kinetic trapping; rates of degradation are reduced by folding and assembly. These phenomena increase the depths of the kinetic traps and decrease rates of hydrolysis and other chemical degradation (McKinley et al. 1983; Prusiner 1998; Nahvi et al. 2002; Shoulders and Raines 2009; van der Lee et al. 2014). The mechanisms of decrease in rates of hydrolysis are by depopulating high energy states along degradation reaction coordinates. Folding and assembly destabilize intermediates and transition states of biopolymer hydrolysis reactions.</p><p>Therefore, it is accurate to describe biopolymers as self-protective (by folding and homogeneous assembly) and partner-protective (by heterogeneous assembly). Biopolymers are both selfish and nurturing. These properties delay but do not avert the ultimate fate of any biopolymer—hydrolysis.</p><p>Folding-based and assembly-based protection from hydrolysis, allowing persistence in aqueous environments, is pre-programmed into biopolymers at several levels. At the most fundamental level, biopolymer backbones are pre-organized for folding by geometrically arrayed, self-complementary molecular interactions and geometric propensities to fold, induced by rotameric and steric restraints on conformation. Protein conformation is restrained by the planarity of the peptide linkage and by ϕφ restraints (Pauling and Corey 1951; Ramachandran and Sasisekharan 1968). Polynucleotides are restrained by planarities of bases and by “rigidity” of nucleotides. The available conformational space of the backbone is restricted by constraints on and correlations between torsional angles (Sundaralingam and Westhof 1979). Polysaccharides are restrained by conformational preferences within and between sugars (Stick and Williams 2010). Thus, even as random coil, in which high temperature or chemical denaturants disrupt intramolecular interactions, biopolymers retain a kinetic and thermodynamic propensity to fold. Folding is fast and spontaneous when the temperature is lowered or the denaturant is removed.</p><h2 id=complements-to-the-chef>Complements to the Chef
<a class=anchor href=#complements-to-the-chef>#</a></h2><h3 id=complementarity>Complementarity
<a class=anchor href=#complementarity>#</a></h3><p>Self-complementarity is a universality of biopolymers. Self-complementarity is proficiency for preferential self-binding, which is the ability to attract and associate with self to the exclusion of non-self. Three-dimensional structures of folded/assembled DNA, RNA, protein, and polysaccharide reveal extensive networks of highly specific molecular interactions in which biopolymers complement themselves.</p><p>The term “self-complementary” has traditionally referred only to the interactions between nucleic acid bases, such as those in the DNA duplex shown in Fig. 4. “Self-complementary” has not, to our knowledge, been used previously to describe the polypeptide backbone, apparently because the nomenclature for intramolecular interactions of nucleic acids is historically distinct and separate from that describing interactions of proteins. However, “self-complementary” is an exact and accurate description of the polypeptide backbone. Polypeptide selectively adheres to itself via extended arrays of hydrogen bond donors and acceptors that are geometrically matched in three-dimensional space. This donor/acceptor matching is realized by local interactions within α-helices, or by non-local interactions within β-sheets (Fig. 3). Thus, protein realizes self-complementarity in two fully distinct folded states, a remarkable feat.</p><p><img src=/img/biopolymers_fig4.png alt="The DNA duplex with sequence GATC in each strand is self-complementary, containing geometrically matched arrays of hydrogen bond donors and acceptors that link the two strands."></p><figcaption><p><strong>Fig. 4</strong>
figure4
The DNA duplex with sequence GATC in each strand is self-complementary, containing geometrically matched arrays of hydrogen bond donors and acceptors that link the two strands. On the left, the normals of the base pairs are within the plane of the page. On the right the normals of the base pairs are orthogonal to the page. Hydrogen bonds are indicated by dashed lines
Full size image</p></figcaption><p>Glucose in the polymerized state is intrinsically self-complementary. In cellulose, essentially all hydrogen bonding functionalities of each glucose are positively engaged with those of other glucose moieties (Fig. 5). Cellulose and chitin form stable intra-chain interfaces secured by large complementary arrays hydrogen bond donors and acceptors.</p><p><img src=/img/biopolymers_fig5.png alt="In cellulose, matched arrays of hydrogen bond donors and acceptors stabilize the folding of (1–4) polymerized glucose into homogeneous fibers."></p><figcaption><p><strong>Fig. 5</strong>
figure5
In cellulose, matched arrays of hydrogen bond donors and acceptors stabilize the folding of (1–4) polymerized glucose into homogeneous fibers. The β-anomer but not the α-anomer enables complementary glucose–glucose interactions in the polymer
Full size image</p></figcaption><p>Self-complementarity of biopolymers is conditional and can be switched between “on” or “off” states. For DNA and RNA, self-complementarity is conditional on nucleotide sequence, which can act as a controlling switch for the formation of elongated DNA duplexes or RNA stem-loops. Sequences such as r(CGA–UCG) use self-complementarity to form intramolecular stem-loops or intermolecular dimers, while those such as r(CGA–CGA) do not. Complementarity is achieved by hydrogen bonding interactions between nitrogenous bases, and is switched on and off by changing the sequence. The self-complementarity of protein is modulated by amino acid composition. Specifically, proline can switch the self-complementarity to the off state by unbalancing the ratio of hydrogen bond donors to acceptors. The anomeric linkage provides the on/off switch for the self-complementarity of polyglucose. β-Anomers such as cellulose and chitin are self-complementary. α-Anomers such as glycogen and amylose are not.</p><p>Molecular interactions that enable self-complementarity of DNA, RNA, or protein primarily utilize unipolar hydrogen bonds such as those of keto oxygens, amide and imine nitrogens, and polarized amino groups (Figs. 3, 4). The molecular interactions of polysaccharides are dominated by hydroxyl groups (Fig. 5). Hydroxyl groups are bipolar, with the ability to both donate and accept hydrogen bonds.</p><h3 id=perturbationclarification>Perturbation—Clarification
<a class=anchor href=#perturbationclarification>#</a></h3><p>Although certain amino acids (such proline) profoundly alter self-complementarity of polypeptide, amino acid sequence should be seen as a second-order perturbation of cohesive backbone interactions. Anfinsen described the native state of a globular protein as unique, stable, kinetically accessible, at a free energy minimum, and determined <em>only</em> by amino acid sequence (Anfinsen et al. 1961). However, essentially any amino acid sequence at high concentration forms fibrils in which β-sheet is the default mode of self-interaction (Fändrich and Dobson 2002; Pedersen et al. 2010). Globular proteins in dilute solution and amyloids at high concentrations follow the same organizing principle; both demonstrate the dominance of cohesive backbone interactions under all non-denaturing conditions.</p><h2 id=separated-at-birth>Separated at Birth
<a class=anchor href=#separated-at-birth>#</a></h2><p>Although DNA, RNA, protein, and polysaccharide have many chemical and structural similarities, they are distinguished by obvious differences. The backbone of protein is neutral, cohesive, and self-complementary, enabling formation of hydrophobic cores where water is excluded. The backbones of RNA and DNA are anionic and self-repulsive. RNA folds to globular structures with wet, salty cores, while DNA tends not to form globular structures at all. Polyglucose forms dry but hydrophilic cores stabilized by the vastness of the contact area. Protein and polysaccharide folding are largely independent of cofactors. RNA and DNA folding are dependent on cationic cofactors. The specific ordering of sidechains along monotonous backbones of RNA, DNA, and proteins are important devices for modulating and manipulating conformation and molecular interaction. Protein sidechains are many and chemically diverse. RNA and DNA sidechains are few and are chemically homogeneous. Polysaccharides lack sidechains altogether. RNA, DNA, protein, and some types of polyglucose (cellulose, amylose and chitin) are linear, while glycogen (animals) and amylopectin (plants) are branched. Each of the linear biopolymers folds to helical structures (Pauling et al. 1951; Pauling and Corey 1951; Watson and Crick 1953). Polypeptide has been selected by nature to fold predominantly via backbone interactions. Polynucleotides have been selected to fold predominantly via sidechain interactions. Evolution may have found it advantageous to include additional mechanisms for modulating biopolymer properties; post polymerization modifications of biopolymers can modulate their physicochemistry and biological functions.</p><h3 id=adding-it-up>Adding It Up
<a class=anchor href=#adding-it-up>#</a></h3><p>The net hydrogen bonding polarities of polypeptides sum to zero, with equivalent numbers of hydrogen bond donors and acceptors. Polyglucose has an excess of hydrogen bond acceptors over donors. Polynucleotides have a large excess of acceptors over donors.</p><h3 id=functional-distance>Functional Distance
<a class=anchor href=#functional-distance>#</a></h3><p>Is it possible to relate the functional roles of biopolymers to their structures? First, one must attempt to accurately describe biological functions. What does each biopolymer type do? There are no bright lines—functional roles are not rigidly proscribed by polymer type. The enormous diversity in the chemical transformations of biological systems are catalyzed and regulated primarily by proteins. Protein contributes enzymes, enzyme inhibitors, structural fibers, adhesives, pumps, pores, switches, and receptors. RNA is used for temporal and specific information transfer (i.e., mRNA) and performs more limited, but nonetheless critical, catalytic functions, for example in the ribosome. By contrast, DNA appears to be used exclusively for long-term and bulk information storage (i.e., whole genome) and transfer. On the whole, polynucleotides maintain, record, read and transmit sequence information. Polysaccharides contribute structure along with energy storage and elaborate recognition.</p><p>Ribozymes (Kruger et al. 1982; Guerrier-Takada et al. 1983), which are RNA-based “enzymes,” have correctly assumed a great deal of symbolic significance and importance in discussions of fundamentals of biology and the origin of life. However, thus far there has been no observation of a biological RNA-only ribozyme that is formally enzymatic; there are no RNA-only biological ribozymes that turn over (Kruger et al. 1982; Hutchins et al. 1986; Prody et al. 1986). All RNA-only catalytic elements discovered thus far in biological systems perform suicide (single turn-over) phosphoryl transfer functions. By contrast, highly abundant and critically important ribonuclear protein ribozymes (protein-assisted ribozymes), with RNA-only catalytic sites, do turn over and are thus fully enzymatic. These RNP ribozymes include the ribosome (Khaitovich et al. 1999), RNase P (Guerrier-Takada et al. 1983), and the spliceosome (Brody and Abelson 1985). No catalytic function of polysaccharide has been observed thus far, to our knowledge.</p><h2 id=fraternal-twins-dna-and-rna>Fraternal Twins: DNA and RNA
<a class=anchor href=#fraternal-twins-dna-and-rna>#</a></h2><p>DNA and RNA both fold and assemble to form double helices with central cores of paired and stacked nucleobases, framed by external, anionic backbones. DNA and RNA appear similar in chemical representations, differing only by a single atom on the backbone and by a methyl group on one base.</p><p>The 2′ hydroxyl group profoundly influences folding, providing a nucleation hook for base–backbone association, thus fostering diverse loops and junctions. The preponderance of hydrogen bond acceptors over donors of DNA is partially relaxed in RNA by the 2′ hydroxyl group, which provides a locus for intramolecular cohesion. A frequent folding motif involving base–backbone interactions of rRNA is the GNRA tetraloop. There are over 40 examples of this motif in the large ribosomal subunit of prokaryotes (Hsiao et al. 2009). These structures, and many other non-helical structures, are stabilized by intramolecular interactions between 2′ hydroxyl groups and RNA bases (Fig. 6b). These base–backbone interactions promote folding of RNA into local stem-loops, which are often further stabilized by tertiary interactions (Fig. 6c). Biological DNA, by contrast, is generally restrained to base–base associations, forming long, monotonous double helices (Fig. 6d).</p><p><img src=/img/biopolymers_fig6.png alt="The impact of the 2′ hydroxyl group on polynucleotide reactivity and structure."></p><figcaption><p><strong>Fig. 6</strong> figure6 The impact of the 2′ hydroxyl group on polynucleotide reactivity and structure. <strong>a</strong> Reactivity. RNA holds a gun to its own head. The 2′ oxygen is a nucleophile that is poised to attack the adjacent phosphorous atom, cleaving the RNA backbone. <strong>b</strong> Folding. The 2′ hydroxyl group nucleates folding of complex structures by enabling hydrogen bonding between the backbone and bases, as demonstrated in the GNRA tetraloop. The 2′ hydroxyl of a guanine forms a hydrogen bond with the N7 of an adenine. In addition, the N1 and N2 of the guanine form a hydrogen bond with a phosphate oxygen of the backbone. <strong>c</strong> Complexity. RNA folds into elaborate three-dimensional structures. <strong>d</strong> DNA folds to long double helices. In panels a and b, hydrogen bonding groups that do not form hydrogen bonds are omitted for clarity
Full size image</p></figcaption><p>Profound differences in reactivity distinguish RNA from DNA. RNA is recalcitrant to oxidative radical damage relative to DNA. However, RNA “holds a gun to its own head”; each 2′ hydroxyl of RNA is poised for nucleophilic attack at the adjacent phosphorus atom, causing cleavage of the backbone (Fig. 6a). The rate of RNA self-cleavage is modulated by local structure, flexibility, pH, and interactions with cations. Thus, the RNA and DNA backbones have distinctive lability profiles, which depend on many factors including on the chemistry of the cleavage process.</p><h2 id=nature-chose-phosphate>Nature Chose Phosphate
<a class=anchor href=#nature-chose-phosphate>#</a></h2><p>Westheimer suggested that phosphates dominate molecular biology because phosphate is a kinetically trapped (e.g., a phosphate ester), tunable, water-soluble leaving group that can be linked to small molecules, conferring anionic charge and blocking transit across membranes (Westheimer 1987). While correct, in our view this analysis should be extended to incorporate the role of phosphate in mechanochemical coupling.</p><p>All biopolymerization reactions utilize phosphorylated or pyrophosphorylated intermediates (Figs. 1, 2) in reactions catalyzed by processive enzymes. Phosphorylated intermediates appear to be necessary for the mechanochemical coupling required for processive polymerization. The polymerases that make DNA, RNA, protein, and polysaccharide are nanoscale motors. Translocation is energy-driven; the nascent polymer translocates relative to the polymerization enzyme. Mechanochemical coupling in motor proteins is commonly linked to association/dissociation of phosphate because phosphate has “claws” that reach out in three dimensions; phosphate can grab onto and deform proteins. The strength, directionality, and unipolarity of hydrogen bonding and electrostatic interactions between phosphate and protein cause linkage of phosphate association to protein conformation (Rice et al. 1999; Wittinghofer 2016). This coupling of directed molecular displacement (work) to association/dissociation of phosphate, which is in turn linked to pyrophosphate hydrolysis, has been characterized in myosin and kinesin, in the ribosome and in DNA, RNA, and cellulose polymerases (Wang et al. 1998; Morin et al. 2015; Morgan et al. 2016; Arias-Gonzalez 2017). During polymerizations of DNA, RNA, protein, and polyglucose, translocations are structurally and energetically coupled to phosphate association/dissociation.</p><h2 id=molecules-in-mutualism>Molecules in Mutualism
<a class=anchor href=#molecules-in-mutualism>#</a></h2><p>We (Williams) have previously proposed that formalisms for describing mutualisms on levels of cells, organisms, and ecosystems also apply to biopolymers (Lanier et al. 2017). Mutualisms are everywhere in the biosphere and are fundamentally important in evolution, ecology, and economy (Moran 2006; Bronstein 2015; Douglas 2015; Gray 2017). The mutual benefit, exchange of proficiencies, persistence, interdependence, co-evolution, and parasitism that characterize relationships on cellular, organismal, and ecological levels have direct parallels in the behaviors of biopolymers.</p><p>A mutualism is a persistent and intimate interaction that benefits multiple interactors (Douglas 2015). Because mutualisms are prolonged and intimate, partners in mutualism influence each other’s evolution. Evolutionary change of one partner triggers change of the other. We believe that biopolymers are mutually imprinted on each other in structure and function via their co-evolution, stabilizing the mutualism.</p><h3 id=levels-of-mutualism>Levels of Mutualism
<a class=anchor href=#levels-of-mutualism>#</a></h3><p>Mutualisms were previously understood to operate at the levels of cells, organisms, ecosystems, and even societies and economies. The eukaryotic cell is a culmination of mutualism between simpler prokaryotic cells (Sagan 1967; Poole and Gribaldo 2014; Gray 2017). Essentially every species on Earth is involved in mutualisms.</p><h3 id=molecules>Molecules
<a class=anchor href=#molecules>#</a></h3><p>Biopolymers satisfy all of the formalisms of mutualism. Biopolymers protect each other from hydrolysis and synthesize each other. Polypeptide synthesizes polynucleotide (polymerases) and polynucleotide synthesizes protein (the ribosome). During essential steps of translation, coding is performed by proteins (aaRS enzymes charge tRNAs), while decoding is performed by RNAs (mRNA and rRNA) in the ribosome. Molecules in Mutualism describes: (i) survival—extant biopolymers are more persistent than competing polymer types, which are now extinct; (ii) co-evolution—biopolymers created each other in an emergent and cooperative environment of chemical evolution; (iii) fitness—biopolymers are more ‘fit’ in combination than in isolation; (iv) distance—each biopolymer type has distinct proficiencies and chemical characteristics; (v) innovation—proficiencies of one type of biopolymer release constraints on partner biopolymer types; (vi) robustness—biopolymer types have been fixed for billions of years, meaning biopolymers compose seminal and ancient mutualism with profound stability; and (vii) parasitism—examples of molecular self-interest and escape from mutualism are seen in amyloids (McKinley et al. 1983) and phase-separated RNA gels (Jain and Vale 2017).</p><h2 id=origins-of-biopolymersorigins-of-life>Origins of Biopolymers—Origins of Life
<a class=anchor href=#origins-of-biopolymersorigins-of-life>#</a></h2><h3 id=why-biopolymers>Why Biopolymers?
<a class=anchor href=#why-biopolymers>#</a></h3><p>Biology requires polymers. Biopolymers allow processes of folding and assembly to be detached from the required investment of free energy. For biopolymers, prior free energy investment in synthesis is distributed over time and space, offsetting the subsequent cost of folding and assembly. Biopolymers appear to <em>spontaneously</em> fold and assemble, only because of prior free energy investments. For small molecules, by contrast, assembly and investment are directly coupled. The free energy of assembly is paid in real time, during molecular assembly. Therefore, small molecules cannot achieve the elaborate folds and assemblies, based on conditional self-complementarity, that appear to come naturally to biopolymers.</p><p>The data surveyed here suggest that polypeptide, polynucleotide, and polysaccharide arose by co-evolution. Biopolymer universalities, including (i) synthesis by condensation and degradation by hydrolysis, (ii) folding by pre-organization and self-complementarity, (iii) homogeneous and heterogeneous assembly, and (iv) protection by folding or homogeneous assembly (selfishness) and (v) protection by heterogeneous assembly (mutualism), point to simultaneous origins in a shared environment. The co-origins of biopolymers are consistent with previous reports of common chemistry of monomer formation (Miller and Urey 1959; Oró and Guidry 1960; Patel et al. 2015).</p><h3 id=origins-of-biopolymers>Origins of Biopolymers
<a class=anchor href=#origins-of-biopolymers>#</a></h3><p>In our view, observed biopolymer universalities and idiosyncracies support a model in which polymer synthesis by condensation cooperated with hydrolytic degradation, mediated by folding and assembly, to drive chemical evolution (Brack 1987; Abkevich et al. 1996; Hud and Anet 2000; Peters and Williams 2012). In contrast to the consensus, this model suggests that early selection operated at the level of hydrolytic degradation (mitigated by folding and assembly), rather than at the level of synthesis. After nearly 4 billion years of evolution, biopolymers continue to utilize self-complementarity to escape hydrolysis and increase persistence (Prusiner 1998; Jain and Vale 2017; Bai et al. 2018).</p><p>We (Hud) have proposed that the thermodynamic driver for synthesis and degradation on the ancient Earth would have been cycling water activity (Forsythe et al. 2015), which was and is ubiquitous over the landmass of the Earth. Thus, it seems possible that polymers originated via simple (non-redox) chemistry that remained near equilibrium, rocking gently in the cradle of day/night cycling (Hud and Anet 2000). Synthesis by condensation dehydration is favored in low water activity (day) and degradation by hydrolysis is favored in high water activity (night). The close analogy of biopolymer synthesis/degradation by hydrolysis/condensation in biochemical system to wet/dry cycling in geochemistry suggests that the origin of life, like extant life, was planet-wide phenomena of surfaces, and was not a function of exotic environments with constant, high water activity as in hydrothermal vents (Corliss et al. 1981).</p><h3 id=selfish-molecules>Selfish Molecules
<a class=anchor href=#selfish-molecules>#</a></h3><p>One can define biopolymer “self-interest”. Molecular self-interest is chemical persistence. Persistence of biopolymers in a hydrolytic environment is enhanced by folding and assembly. Self-complementarity is therefore an expression of self-interest, a method to escape from hydrolysis, a path to survival, and a property universal to biopolymers. Heterogeneous assemblies are expressions of partner-protection from hydrolysis, described here and elsewhere as molecular mutualisms (Lanier et al. 2017).</p><p>The observed nominal stability, rather than extreme stability, of biopolymer folds and assemblies, suggests that unfolding and disassembly confers advantage in some circumstances. The ability to unfold and disassemble provides pathways for prospecting for new folds and new partners and for recycling. Extremely stable folds and assemblies could persist for some period but ultimately form molecular dead-ends.</p><h3 id=losers>Losers
<a class=anchor href=#losers>#</a></h3><p>The juxtaposition of biopolymer universalities next to the diversity of chemical compositions of multiple biopolymer types is consistent with a model of simultaneous biopolymer origins via step-wise evolutionary processes, rather than from direct but improbable and singular phenomena (Ricardo et al. 2004). It seems likely that our small set of surviving biopolymers were chemically selected from diverse competing polymers (Hud et al. 2013), most of which failed to compete successfully because of their lesser ability to fold and assemble. Biopolymers, as indicated by spider webs, DNA nanodevices, chromatin, the ribosome, and cellulose, are masters of folding and assembly. It seems improbable that this mastery arose from good luck, rather than from chemical evolution. Loser polymer types, which were less accomplished at folding and assembly, were forced into hydrolytic extinction.</p><p>If so, ancestral polymers, which dominated in early stages, would have been supplanted by more successful second- or third-generation polymers. The scenario described here does not ascribe utility to catalysis or replication during the early origins of biopolymers and is agnostic on compartmentalization (Szostak 2017), although it does seem to require that compartments be competent to tolerate cycling water activity.</p><p>Biopolymer universalities are not inconsistent with conclusions of de la Escosura and coworkers, who argue that the origin of life involved a “system” (de la Escosura et al. 2015). Their <em>system</em>, a heterogeneous, functionally integrated, self-maintained, quasi-stationary state allowing for increases in complexity and elaboration, is a chemically vague but reasonable description of our shared environment of cycling water activity and co-evolution, with chemical selection at the level of degradation.</p><h2 id=conclusion>Conclusion
<a class=anchor href=#conclusion>#</a></h2><p>Although biopolymer types are traditionally studied and taught in isolation of each other, we believe that DNA, RNA, polypeptide, and polysaccharide are best understood in the context of their shared attributes and key differences. Recognition of biopolymer universalities explains their structures and functions and points to their origins. Foundational among these universalities is the ability of all biopolymers to fold via self-complementarity and assemble into structures that protect them (at least for a while) from their thermodynamic fate of chemical degradation in dilute aqueous solution. Only by examining biopolymers in context can we hope to achieve a reasonable understanding of the fundamental molecules of life.</p><h2 id=references>References
<a class=anchor href=#references>#</a></h2><ul><li>Abkevich V, Gutin A, Shakhnovich E (1996) How the first biopolymers could have evolved. <em>Proc Natl Acad Sci USA</em> 93:839–844
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Anfinsen CB, Haber E, Sela M, White FH Jr (1961) The kinetics of formation of native ribonuclease during oxidation of the reduced polypeptide chain. <em>Proc Natl Acad Sci USA</em> 47:1309–1314
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Arias-Gonzalez JR (2017) A DNA-centered explanation of the DNA polymerase translocation mechanism. <em>Sci Rep</em> 7:7566
Article</li></ul><p>Google Scholar</p><ul><li>Bai Y, Chotera A, Taran O, Liang C, Ashkenasy G, Lynn DG (2018) Achieving biopolymer synergy in systems chemistry. <em>Chem Soc Rev</em> 47:5444–5456
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Ban N, Nissen P, Hansen J, Moore PB, Steitz TA (2000) The complete atomic structure of the large ribosomal subunit at 2.4 Å resolution. <em>Science</em> 289:905–920
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Brack A (1987) Selective emergence and survival of early polypeptides in water. <em>Orig Life Evol Biosph</em> 17:367–379
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Brody E, Abelson J (1985) The spliceosome—yeast pre-messenger RNA associates with a 40S complex in a splicing-dependent reaction. <em>Science</em> 228:963–967
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Bronstein JL (2015) The study of mutualism. In: Bronstein JL (ed) <em>Mutualism</em>. Oxford Press, Oxford, pp 3–19
Chapter</li></ul><p>Google Scholar</p><ul><li><p>Corliss JB, Baross J, Hoffman S (1981) An hypothesis concerning the relationships between submarine hot springs and the origin of life on earth. <em>Oceanol Acta</em> 4:59–69
Google Scholar</p></li><li><p>Cosgrove DJ (2014) Re-constructing our models of cellulose and primary cell wall assembly. <em>Curr Opin Plant Biol</em> 22:122–131
CAS</p></li></ul><p>Article</p><p>Google Scholar</p><ul><li>de la Escosura A, Briones C, Ruiz-Mirazo K (2015) The systems perspective at the crossroads between chemistry and biology. <em>J Theor Biol</em> 381:11–22
Article</li></ul><p>Google Scholar</p><ul><li>Douglas AE (2015) The study of mutualism. In: Bronstein JL (ed) <em>Mutualism</em>. Oxford Press, Oxford, pp 20–34
Chapter</li></ul><p>Google Scholar</p><ul><li>Eisenberg D (2003) The discovery of the α-helix and β-sheet, the principal structural features of proteins. <em>Proc Natl Acad Sci USA</em> 100:11207–11210
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Fändrich M, Dobson CM (2002) The behaviour of polyamino acids reveals an inverse side chain effect in amyloid structure formation. <em>EMBO J</em> 21:5682–5690
Article</li></ul><p>Google Scholar</p><ul><li>Forsythe JG, Yu SS, Mamajanov I, Grover MA, Krishnamurthy R, Fernandez FM, Hud NV (2015) Ester-mediated amide bond formation driven by wet-dry cycles: a possible path to polypeptides on the prebiotic earth. <em>Angew Chem Int Ed</em> 54:9871–9875
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Fuchs E (1976) The interdependence of magnesium with spermidine and phosphoenolpyruvate in an enzyme-synthesizing system in vitro. <em>FEBS J</em> 63:15–22
CAS</li></ul><p>Google Scholar</p><ul><li>Gabius H-J, André S, Jiménez-Barbero J, Romero A, Solís D (2011) From lectin structure to functional glycomics: principles of the sugar code. <em>Trends Biochemical Sci</em> 36:298–313
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Gellert M, Lipsett MN, Davies DR (1962) Helix formation by guanylic acid. <em>Proc Natl Acad Sci USA</em> 48:2013–2018
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Gray MW (2017) Lynn Margulis and the endosymbiont hypothesis: 50 years later. <em>Mol Biol Cell</em> 28:1285–1287
Article</li></ul><p>Google Scholar</p><ul><li>Guerrier-Takada C, Gardiner K, Marsh T, Pace N, Altman S (1983) The RNA moiety of ribonuclease P is the catalytic subunit of the enzyme. <em>Cell</em> 35:849–857
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Hsiao C, Mohan S, Kalahar BK, Williams LD (2009) Peeling the onion: ribosomes are ancient molecular fossils. <em>Mol Biol Evol</em> 26:2415–2425
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Hud NV, Anet FA (2000) Intercalation-mediated synthesis and replication: a new approach to the origin of life. <em>J Theor Biol</em> 205:543–562
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Hud NV, Cafferty BJ, Krishnamurthy R, Williams LD (2013) The origin of RNA and “my grandfather’s axe”. <em>Chem Biol</em> 20:466–474
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Hutchins CJ, Rathjen PD, Forster AC, Symons RH (1986) Self-cleavage of plus and minus RNA transcripts of avocado sunblotch viroid. <em>Nucleic Acids Res</em> 14:3627–3640
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Jain A, Vale RD (2017) RNA phase transitions in repeat expansion disorders. <em>Nature</em> 546:243–247
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Kang M, Elango N, Mattia E, Au-Young J, Robbins P, Cabib E (1984) Isolation of chitin synthetase from saccharomyces cerevisiae. Purification of an enzyme by entrapment in the reaction product. <em>J Biol Chem</em> 259:14966–14972
CAS</li></ul><p>PubMed</p><p>Google Scholar</p><ul><li>Khaitovich P, Tenson T, Mankin AS, Green R (1999) Peptidyl transferase activity catalyzed by protein-free 23S ribosomal RNA remains elusive. <em>RNA</em> 5:605–608
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Kruger K, Grabowski PJ, Zaug AJ, Sands J, Gottschling DE, Cech TR (1982) Self-splicing RNA: autoexcision and autocyclization of the ribosomal RNA intervening sequence of tetrahymena. <em>Cell</em> 31:147–157
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Lanier KA, Petrov AS, Williams LD (2017) The central symbiosis of molecular biology. <em>J Mol Evol</em> 85:8–13
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>McKinley MP, Bolton DC, Prusiner SB (1983) A protease-resistant protein is a structural component of the scrapie prion. <em>Cell</em> 35:57–62
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>McNamara JT, Morgan JL, Zimmer J (2015) A molecular description of cellulose biosynthesis. <em>Annu Rev Biochem</em> 84:895–921
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Miller SL, Urey HC (1959) Origin of life: Organic compound synthesis on the primitive earth. <em>Science</em> 130:1622–1624
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Moore PB (1999) Structural motifs in RNA. <em>Annu Rev Biochem</em> 68:287–300
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Moran NA (2006) Symbiosis. <em>Curr Biolgy</em> 16:R866–R871
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Morgan JL, McNamara JT, Fischer M, Rich J, Chen H-M, Withers SG, Zimmer J (2016) Observing cellulose biosynthesis and membrane translocation in crystallo. <em>Nature</em> 531:329
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Morin JA, Cao FJ, Lázaro JM, Arias-Gonzalez JR, Valpuesta JM, Carrascosa JL, Salas M, Ibarra B (2015) Mechano-chemical kinetics of DNA replication: identification of the translocation step of a replicative DNA polymerase. <em>Nucleic Acids Res</em> 43:3643–3652
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Nahvi A, Sudarsan N, Ebert MS, Zou X, Brown KL, Breaker RR (2002) Genetic control by a metabolite binding mRNA. <em>Chem Biol</em> 9:1043–1049
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Oró J, Guidry C (1960) A novel synthesis of polypeptides. <em>Nature</em> 186:156
Article</li></ul><p>Google Scholar</p><ul><li>Patel BH, Percivalle C, Ritson DJ, Duffy CD, Sutherland JD (2015) Common origins of RNA, protein and lipid precursors in a cyanosulfidic protometabolism. <em>Nat Chem</em> 7:301–307
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Pauling L, Corey RB (1951) The pleated sheet, a new layer configuration of polypeptide chains. <em>Proc Natl Acad Sci USA</em> 37:251–256
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Pauling L, Corey RB, Branson HR (1951) The structure of proteins—2 hydrogen-bonded helical configurations of the polypeptide chain. <em>Proc Natl Acad Sci USA</em> 37:205–211
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Pedersen JS, Andersen CB, Otzen DE (2010) Amyloid structure—one but not the same: the many levels of fibrillar polymorphism. <em>FEBS J</em> 277:4591–4601
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Peters JW, Williams LD (2012) The origin of life: look up and look down. <em>Astrobiology</em> 12:1087–1092
Article</li></ul><p>Google Scholar</p><ul><li>Poole AM, Gribaldo S (2014) Eukaryotic origins: how and when was the mitochondrion acquired? <em>Cold Spring Harb Perspect Biol</em> 6:a015990
Article</li></ul><p>Google Scholar</p><ul><li>Porter LL, Rose GD (2012) A thermodynamic definition of protein domains. <em>Proc Natl Acad Sci USA</em> 109:9420–9425
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Prody GA, Bakos JT, Buzayan JM, Schneider IR, Bruening G (1986) Autolytic processing of dimeric plant–virus satellite RNA. <em>Science</em> 231:1577–1580
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Prusiner SB (1998) Prions. <em>Proc Natl Acad Sci USA</em> 95:13363–13383
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Ramachandran GN, Sasisekharan V (1968) Conformation of polypeptides and proteins. <em>Adv Protein Chem</em> 23:283–438
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Ricardo A, Carrigan M, Olcott A, Benner S (2004) Borate minerals stabilize ribose. <em>Science</em> 303:196–196
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Rice S, Lin AW, Safer D, Hart CL, Naber N, Carragher BO, Cain SM, Pechatnikova E, Wilson-Kubalek EM, Whittaker M, Pate E, Cooke R, Taylor EW, Milligan RA, Vale RD (1999) A structural change in the kinesin motor protein that drives motility. <em>Nature</em> 402:778–784
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Sagan L (1967) On the origin of mitosing cells. <em>J Theor Biol</em> 14:225–274
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Shoulders MD, Raines RT (2009) Collagen structure and stability. <em>Annu Rev Biochem</em> 78:929–958
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Steitz TA (1999) DNA polymerases: structural diversity and common mechanisms. <em>J Biol Chem</em> 274:17395–17398
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li><p>Stick RV, Williams S (2010) <em>Carbohydrates: the essential molecules of life</em>. Elsevier, Amsterdam
Google Scholar</p></li><li><p>Sundaralingam M, Westhof E (1979) The “rigid” nucleotide concept in perspective. <em>Int J Quantum Chem</em> 16:115–130
Article</p></li></ul><p>Google Scholar</p><ul><li>Szostak JW (2017) The narrow road to the deep past: in search of the chemistry of the origin of life. <em>Angew Chem Int Ed</em> 56:11037–11043
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Trappl K, Polacek N (2011) The ribosome: a molecular machine powered by RNA. <em>Metal Ions Life Sci</em> 9:253–275
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Ts’o POP (1974) Bases, nucleosides and nucleotides. In: Ts’o POP (ed) <em>Basic principles in nucleic acid chemistry</em>. Academic Press, New York, pp 453–584
Chapter</li></ul><p>Google Scholar</p><ul><li>Valeri IK (2010) Crystalline cellulose: structure and hydrogen bonds. <em>Russ Chem Rev</em> 79:231
Article</li></ul><p>Google Scholar</p><ul><li>van der Lee R, Lang B, Kruse K, Gsponer J, de Groot NS, Huynen MA, Matouschek A, Fuxreiter M, Babu MM (2014) Intrinsically disordered segments affect protein half-life in the cell and during evolution. <em>Cell Rep</em> 8:1832–1844
Article</li></ul><p>Google Scholar</p><ul><li>Wan Y, Kertesz M, Spitale RC, Segal E, Chang HY (2011) Understanding the transcriptome through RNA structure. <em>Nature Rev Genet</em> 12:641
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Wang HY, Elston T, Mogilner A, Oster G (1998) Force generation in RNA polymerase. <em>Biophys J</em> 74:1186–1202
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Watson JD, Crick FH (1953) Molecular structure of nucleic acids: a structure for deoxyribose nucleic acid. <em>Nature</em> 171:737–738
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Westheimer FH (1987) Why nature chose phosphates. <em>Science</em> 235:1173–1178
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Wittinghofer A (2016) GTP and ATP hydrolysis in biology. <em>Biopolymers</em> 105:419–421
CAS</li></ul><p>Article</p><p>Google Scholar</p><ul><li>Woodson SA (2011) RNA folding pathways and the self-assembly of ribosomes. <em>Acc Chem Res</em> 44:1312–1319
CAS</li></ul><p>Article</p><p>Google Scholar</p><p>Download references</p><h2 id=acknowledgements>Acknowledgements
<a class=anchor href=#acknowledgements>#</a></h2><p>The authors thank Drs. Gary Schuster and Roger Wartell for helpful discussions. This work was funded in part by NASA Exobiology grants NNX16AJ28G and NNX16AJ29G.</p><h2 id=author-information>Author information
<a class=anchor href=#author-information>#</a></h2><h3 id=affiliations>Affiliations
<a class=anchor href=#affiliations>#</a></h3><p>School of Chemistry and Biochemistry, Georgia Institute of Technology, Atlanta, GA, 30332, USA
Calvin M. Runnels, Kathryn A. Lanier, Justin Krish Williams, Jessica C. Bowman, Anton S. Petrov, Nicholas V. Hud & Loren Dean Williams</p><h3 id=corresponding-author>Corresponding author
<a class=anchor href=#corresponding-author>#</a></h3><p>Correspondence to Loren Dean Williams.</p><h2 id=ethics-declarations>Ethics declarations
<a class=anchor href=#ethics-declarations>#</a></h2><h3 id=conflict-of-interest>Conflict of interest
<a class=anchor href=#conflict-of-interest>#</a></h3><p>The authors declare they have no conflict of interest.</p><h2 id=rights-and-permissions>Rights and permissions
<a class=anchor href=#rights-and-permissions>#</a></h2><p>Open Access This article is distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href=http://creativecommons.org/licenses/by/4.0/%29>http://creativecommons.org/licenses/by/4.0/)</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made.</p><h2 id=reprints-and-permissions>Reprints and Permissions
<a class=anchor href=#reprints-and-permissions>#</a></h2><h2 id=about-this-article>About this article
<a class=anchor href=#about-this-article>#</a></h2><p>Verify currency and authenticity via CrossMark
Cite this article
Runnels, C.M., Lanier, K.A., Williams, J.K. et al. Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers. J Mol Evol 86, 598–610 (2018). <a href=https://doi.org/10.1007/s00239-018-9876-2>https://doi.org/10.1007/s00239-018-9876-2</a></p><h2 id=download-citation>Download citation
<a class=anchor href=#download-citation>#</a></h2><p>Received
29 June 2018</p><p>Accepted
09 November 2018</p><p>Published
19 November 2018</p><p>Issue Date
December 2018</p><p>DOI
<a href=https://doi.org/10.1007/s00239-018-9876-2>https://doi.org/10.1007/s00239-018-9876-2</a></p><h3 id=share-this-article>Share this article
<a class=anchor href=#share-this-article>#</a></h3><p>Anyone you share the following link with will be able to read this content:</p><h3 id=get-shareable-link>Get shareable link
<a class=anchor href=#get-shareable-link>#</a></h3><p>Provided by the Springer Nature SharedIt content-sharing initiative</p><h2 id=keywords>Keywords
<a class=anchor href=#keywords>#</a></h2><p>Polynucleotide
Protein
Carbohydrate
Folding
Assembly
Self-complementarity
Download PDF
Sections
Figures
References
Abstract
Chemical Cousins
Living Dangerously
Complements to the Chef
Separated at Birth
Fraternal Twins: DNA and RNA
Nature Chose Phosphate
Molecules in Mutualism
Origins of Biopolymers—Origins of Life
Conclusion
References
Acknowledgements
Author information
Ethics declarations
Rights and permissions
About this article</p><h2 id=advertisement>Advertisement
<a class=anchor href=#advertisement>#</a></h2><p>Over 10 million scientific documents at your fingertips Switch Edition
Academic Edition Corporate Edition
Home Impressum Legal information Privacy statement California Privacy Statement How we use cookies Manage cookies/Do not sell my data Accessibility Contact us
Not logged in - 81.205.198.192
Not affiliated
Springer Nature
© 2021 Springer Nature Switzerland AG. Part of Springer Nature.</p><div class=footnotes role=doc-endnotes><hr><ol><li id=fn:1><p><em>U</em> indicates universal property of all biopolymers&#160;<a href=#fnref:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref1:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref2:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref3:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref4:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref5:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref6:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref7:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref8:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref9:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref10:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref11:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li></ol></div></article><footer class=book-footer><div class="flex flex-wrap justify-between"></div><script>(function(){function e(e){const t=window.getSelection(),n=document.createRange();n.selectNodeContents(e),t.removeAllRanges(),t.addRange(n)}document.querySelectorAll("pre code").forEach(t=>{t.addEventListener("click",function(){e(t.parentElement),navigator.clipboard&&navigator.clipboard.writeText(t.parentElement.textContent)})})})()</script></footer><div class=book-comments></div><label for=menu-control class="hidden book-menu-overlay"></label></div><aside class=book-toc><div class=book-toc-content><nav id=TableOfContents><ul><li><a href=#abstract>Abstract</a></li><li><a href=#chemical-cousins>Chemical Cousins</a></li><li><a href=#biosynthesis>Biosynthesis</a></li><li><a href=#living-dangerously>Living Dangerously</a><ul><li><a href=#fold>Fold</a></li><li><a href=#assemble>Assemble</a></li><li><a href=#degrade>Degrade</a></li></ul></li><li><a href=#complements-to-the-chef>Complements to the Chef</a><ul><li><a href=#complementarity>Complementarity</a></li><li><a href=#perturbationclarification>Perturbation—Clarification</a></li></ul></li><li><a href=#separated-at-birth>Separated at Birth</a><ul><li><a href=#adding-it-up>Adding It Up</a></li><li><a href=#functional-distance>Functional Distance</a></li></ul></li><li><a href=#fraternal-twins-dna-and-rna>Fraternal Twins: DNA and RNA</a></li><li><a href=#nature-chose-phosphate>Nature Chose Phosphate</a></li><li><a href=#molecules-in-mutualism>Molecules in Mutualism</a><ul><li><a href=#levels-of-mutualism>Levels of Mutualism</a></li><li><a href=#molecules>Molecules</a></li></ul></li><li><a href=#origins-of-biopolymersorigins-of-life>Origins of Biopolymers—Origins of Life</a><ul><li><a href=#why-biopolymers>Why Biopolymers?</a></li><li><a href=#origins-of-biopolymers>Origins of Biopolymers</a></li><li><a href=#selfish-molecules>Selfish Molecules</a></li><li><a href=#losers>Losers</a></li></ul></li><li><a href=#conclusion>Conclusion</a></li><li><a href=#references>References</a></li><li><a href=#acknowledgements>Acknowledgements</a></li><li><a href=#author-information>Author information</a><ul><li><a href=#affiliations>Affiliations</a></li><li><a href=#corresponding-author>Corresponding author</a></li></ul></li><li><a href=#ethics-declarations>Ethics declarations</a><ul><li><a href=#conflict-of-interest>Conflict of interest</a></li></ul></li><li><a href=#rights-and-permissions>Rights and permissions</a></li><li><a href=#reprints-and-permissions>Reprints and Permissions</a></li><li><a href=#about-this-article>About this article</a></li><li><a href=#download-citation>Download citation</a><ul><li><a href=#share-this-article>Share this article</a></li><li><a href=#get-shareable-link>Get shareable link</a></li></ul></li><li><a href=#keywords>Keywords</a></li><li><a href=#advertisement>Advertisement</a></li></ul></nav></div></aside></main></body></html>