<!doctype html><html lang=en dir=ltr><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="Skip to main content Advertisement BMC Part of Springer Nature
Search Explore journals Get published About BMC Login BMC Genomics Home About Articles Submission Guidelines Research article Open Access Published: 30 April 2013 Origins of amyloid-β
RESEARCH ARTICLE Open Access
William G Tharp1,2 & Indra Neil Sarkar1,3,4, * BMC Genomics volume 14, Article number: 290 (2013) Cite this article
Tharp and Sarkar BMC Genomics 2013, 14:290 http://www.biomedcentral.com/1471-2164/14/290
* Correspondence: neil.sarkar@uvm.edu"><meta name=theme-color content="#FFFFFF"><meta name=color-scheme content="light dark"><meta property="og:title" content="Origins of amyloid-β"><meta property="og:description" content="Skip to main content Advertisement BMC Part of Springer Nature
Search Explore journals Get published About BMC Login BMC Genomics Home About Articles Submission Guidelines Research article Open Access Published: 30 April 2013 Origins of amyloid-β
RESEARCH ARTICLE Open Access
William G Tharp1,2 & Indra Neil Sarkar1,3,4, * BMC Genomics volume 14, Article number: 290 (2013) Cite this article
Tharp and Sarkar BMC Genomics 2013, 14:290 http://www.biomedcentral.com/1471-2164/14/290
* Correspondence: neil.sarkar@uvm.edu"><meta property="og:type" content="article"><meta property="og:url" content="localhost:1313/docs/amyloid-%CE%B2/"><meta property="article:section" content="docs"><title>Origins of amyloid-β | Derivatives</title><link rel=manifest href=/manifest.json><link rel=icon href=/favicon.png type=image/x-icon><link rel=stylesheet href=/book.min.1a6fac2df464e68d664cb846d822c678cd622692d6fae24ee18901a08bfc8eb6.css><script defer src=/flexsearch.min.js></script>
<script defer src=/en.search.min.0c0371d34f65cdc71a05aefc0008b3ed60ae9cb92f5ac1679f81eaed539cc792.js></script></head><body dir=ltr><input type=checkbox class="hidden toggle" id=menu-control>
<input type=checkbox class="hidden toggle" id=toc-control><main class="container flex"><aside class=book-menu><div class=book-menu-content><nav><h2 class=book-brand><a class="flex align-center" href=/><span>Derivatives</span></a></h2><div class=book-search><input type=text id=book-search-input placeholder=Search aria-label=Search maxlength=64 data-hotkeys=s/><div class="book-search-spinner hidden"></div><ul id=book-search-results></ul></div><ul><li><a href=#ZgotmplZ>README</a></li><li><a href=#ZgotmplZ>Architecture of the Human Origin Recognition Complex*</a></li><li><a href=#ZgotmplZ class=active>Origins of amyloid-β</a></li><li><a href=#ZgotmplZ>Effects of arsenic and heavy metals on metabolic pathways in cells of human origin: Similarities and differences</a></li><li><a href=#ZgotmplZ>Back to the future: origins and directions of the “Agile Manifesto” – views of the originators</a></li><li><a href=#ZgotmplZ>Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers</a></li><li><input type=checkbox id=section-4e626a39f813bdfb0b0f11621a8d97e6 class=toggle>
<label for=section-4e626a39f813bdfb0b0f11621a8d97e6 class="flex justify-between"><a href=#ZgotmplZ>CC-BY</a></label><ul><li><a href=#ZgotmplZ>Creative Commons License Deed Attribution 2.0 Generic (CC BY 2.0)</a></li><li><a href=#ZgotmplZ>Creative Commons Legal Code Attribution 2.0</a></li><li><a href=#ZgotmplZ>Creative Commons License Deed Attribution 3.0 Unported (CC BY 3.0)</a></li><li><a href=#ZgotmplZ>Creative Commons Attribution 3.0 Unported</a></li><li><a href=#ZgotmplZ>Creative Commons License Deed Attribution 4.0 International (CC BY 4.0)</a></li><li><a href=#ZgotmplZ>Creative Commons Konzessionsurkunde Namensnennung 4.0 International (CC BY 4.0)</a></li><li><a href=#ZgotmplZ>Creative Commons Legal Code Attribution 4.0 International</a></li></ul></li><li><a href=#ZgotmplZ>The nature and origin of Charon's smooth plains</a></li><li><a href=#ZgotmplZ>Origin of the Chinese word for “cell”: an unusual but wonderful idea of a mathematician</a></li><li><a href=#ZgotmplZ>City origins</a></li><li><a href=#ZgotmplZ>Counterfeit versus original patronage: Do emotional brand attachment, brand involvement, and past experience matter?</a></li><li><a href=#ZgotmplZ>ON THE ORIGIN OF CREATINE</a></li><li><a href=#ZgotmplZ>Detecting the origins of moisture over Southeast China: Seasonal variation and heavy Rainfall</a></li><li><a href=#ZgotmplZ>Differentiation Between Spirits According to Their Botanical Origin</a></li><li><a href=#ZgotmplZ>Food taboos: their origins and purposes</a></li><li><a href=#ZgotmplZ>Ganges: special at its origin</a></li><li><a href=#ZgotmplZ>Given that the detailed original criteria for deliberate practice have not changed, could the understanding of this complex concept have improved over time? A response to Macnamara and Hambrick (2020)</a></li><li><a href=#ZgotmplZ>Hanwoo cattle: origin, domestication, breeding strategies and genomic selection</a></li><li><a href=#ZgotmplZ>The developmental origins of hoarding disorder in adolescence: a longitudinal clinical interview study following an epidemiological survey</a></li><li><a href=#ZgotmplZ>How to write an original article for the *Journal of Orthopaedics and Traumatology*</a></li><li><a href=#ZgotmplZ>“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds</a></li><li><a href=#ZgotmplZ>An interview with Patrick O Brown on the origins and future of open access</a></li><li><a href=#ZgotmplZ>Can people identify original and manipulated photos of real-world scenes?</a></li><li><a href=#ZgotmplZ>Neighborhoods and bands: an analysis of the origins of spam</a></li><li><input type=checkbox id=section-18bd70f4fad61eb2742d114842aa84cf class=toggle>
<label for=section-18bd70f4fad61eb2742d114842aa84cf class="flex justify-between"><a href=#ZgotmplZ>Letters</a></label><ul><li><a href=#ZgotmplZ>Non-Neanderthal Origin of the HLA-DPB1*0401</a></li><li><a href=#ZgotmplZ>Reply to Ding et al.: Non-Neanderthal Origin of the HLA-DPB1*0401</a></li></ul></li><li><a href=#ZgotmplZ>What is the origin of the ectopic beat?</a></li><li><a href=#ZgotmplZ>Origins</a></li><li><a href=#ZgotmplZ>The 5th anniversary of “Patient Safety in Surgery” – from the Journal’s origin to its future vision</a></li><li><a href=#ZgotmplZ>Tracing the origins of rendang and its development</a></li><li><a href=#ZgotmplZ>Assessing the Impact of an Original Soft Communicative Robot in a Nursing Home in Japan: Will Softness or Conversations Bring more Smiles to Older People?</a></li><li><a href=#ZgotmplZ>Softwaretest mit Originaldaten Eine Analyse aus Sicht des Datenschutzes</a></li><li><a href=#ZgotmplZ>Origin of spectacular fields of submarine sediment waves around volcanic islands</a></li><li><a href=#ZgotmplZ>Urban Conservation Through Chinese Local Original Architecture: The Preservation and Renovation of Zhongshan Road in Hangzhou</a></li><li><a href=#ZgotmplZ>The Origin of Vertebrate Gills</a></li><li><a href=#ZgotmplZ>Volcanic origin of the 1741 Oshima-Oshima tsunami in the Japan Sea</a></li><li><a href=#ZgotmplZ>What is Morita Therapy? The Nature, Origins, and Cross-Cultural Application of a Unique Japanese Psychotherapy</a></li><li><a href=#ZgotmplZ>Revealing the origin of wildcat reappearance after presumed long-term absence</a></li></ul></nav><script>(function(){var e=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(){localStorage.setItem("menu.scrollTop",e.scrollTop)}),e.scrollTop=localStorage.getItem("menu.scrollTop")})()</script></div></aside><div class=book-page><header class=book-header><div class="flex align-center justify-between"><label for=menu-control><img src=/svg/menu.svg class=book-icon alt=Menu></label>
<strong>Origins of amyloid-β</strong>
<label for=toc-control><img src=/svg/toc.svg class=book-icon alt="Table of Contents"></label></div><aside class="hidden clearfix"><nav id=TableOfContents><ul><li><a href=#abstract>Abstract</a></li><li><a href=#background>Background</a></li><li><a href=#results>Results</a><ul><li><a href=#phylogenetics-of-aβpp-gene-family>Phylogenetics of AβPP gene family</a></li><li><a href=#persistence-of-amyloid-β>Persistence of amyloid-β</a></li><li><a href=#evolutionary-relationships-of-amyloid-β-formation-potential>Evolutionary relationships of amyloid-β formation potential</a></li></ul></li><li><a href=#discussion>Discussion</a></li><li><a href=#methods>Methods</a><ul><li><a href=#dataset-collection-and-alignment>Dataset collection and alignment</a></li><li><a href=#generation-of-phylogenetic-trees>Generation of phylogenetic trees</a></li><li><a href=#synapomorphic-frequencies>Synapomorphic frequencies</a></li><li><a href=#aggregation-modeling>Aggregation modeling</a></li></ul></li><li><a href=#electronic-supplementary-material>Electronic supplementary material</a><ul><li><a href=#additional-file-figure-s1>Additional file: Figure S1.</a></li><li><a href=#table-s1-taxa-species-names-and-sequence-accession-numbers-pdf-1-mb>Table S1. Taxa Species Names and Sequence Accession Numbers. (PDF 1 MB)</a></li></ul></li><li><a href=#abbreviations>Abbreviations</a></li><li><a href=#references>References</a></li><li><a href=#acknowledgements>Acknowledgements</a></li><li><a href=#author-information>Author information</a><ul><li><a href=#affiliations>Affiliations</a></li><li><a href=#corresponding-author>Corresponding author</a></li></ul></li><li><a href=#additional-information>Additional information</a><ul><li><a href=#competing-interests>Competing interests</a></li><li><a href=#authors-contributions>Authors’ contributions</a></li></ul></li><li><a href=#rights-and-permissions>Rights and permissions</a></li><li><a href=#reprints-and-permissions>Reprints and Permissions</a></li><li><a href=#about-this-article>About this article</a><ul><li><a href=#cite-this-article>Cite this article</a></li><li><a href=#share-this-article>Share this article</a></li><li><a href=#get-shareable-link>Get shareable link</a></li></ul></li><li><a href=#keywords>Keywords</a></li><li><a href=#figures>Figures</a></li><li><a href=#submission-enquiries>Submission enquiries:</a></li><li><a href=#general-enquiries>General enquiries:</a></li><li><a href=#submit-your-next-manuscript-to-biomed-central-and-take-full-advantage-of>Submit your next manuscript to BioMed Central and take full advantage of:</a></li></ul></nav></aside></header><article class=markdown><h1><a href=/docs/amyloid-%CE%B2/>Origins of amyloid-β</a></h1><h2>William G Tharp & Indra Neil Sarkar</h2><div><a href=/tags/amyloid/>Amyloid</a>,
<a href=/tags/alzheimer-disease/>Alzheimer disease</a>,
<a href=/tags/phylogenetics/>Phylogenetics</a>,
<a href=/tags/in-silico/>In silico</a>,
<a href=/tags/aggregation/>Aggregation</a>,
<a href=/tags/maximum-parsimony/>Maximum parsimony</a>,
<a href=/tags/bayesian-inference/>Bayesian inference</a></div><p>Skip to main content
Advertisement
BMC
Part of Springer Nature</p><p>Search
Explore journals
Get published
About BMC
Login
BMC Genomics
Home
About
Articles
Submission Guidelines
Research article
Open Access
Published: 30 April 2013
Origins of amyloid-β</p><p>RESEARCH ARTICLE Open Access</p><p>William G Tharp<sup id=fnref:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>,<sup id=fnref:2><a href=#fn:2 class=footnote-ref role=doc-noteref>2</a></sup> & Indra Neil Sarkar<sup id=fnref1:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>,<sup id=fnref:3><a href=#fn:3 class=footnote-ref role=doc-noteref>3</a></sup>,<sup id=fnref:4><a href=#fn:4 class=footnote-ref role=doc-noteref>4</a></sup>, *
BMC Genomics volume 14, Article number: 290 (2013) Cite this article</p><p>Tharp and Sarkar <em>BMC Genomics</em> 2013, <strong>14</strong>:290 <a href=http://www.biomedcentral.com/1471-2164/14/290>http://www.biomedcentral.com/1471-2164/14/290</a></p><p>* Correspondence: <a href=mailto:neil.sarkar@uvm.edu>neil.sarkar@uvm.edu</a></p><p>Full list of author information is available at the end of the article</p><p>7408 Accesses
46 Citations
15 Altmetric
Metrics details</p><h2 id=abstract>Abstract
<a class=anchor href=#abstract>#</a></h2><div class=abstract><p><strong>Background:</strong> Amyloid-β plaques are a defining characteristic of Alzheimer Disease. However, Amyloid-β deposition is also found in other forms of dementia and in non-pathological contexts. Amyloid-β deposition is variable among vertebrate species and the evolutionary emergence of the amyloidogenic property is currently unknown. Evolutionary persistence of a pathological peptide sequence may depend on the functions of the precursor gene, conservation or mutation of nucleotides or peptide domains within the precursor gene, or a species-specific physiological environment.</p><p><strong>Results:</strong> In this study, we asked when amyloidogenic Amyloid-β first arose using phylogenetic trees constructed for the Amyloid-β Precursor Protein gene family and by modeling the potential for Amyloid-β aggregation across species <em>in silico</em>. We collected the most comprehensive set of sequences for the Amyloid-β Precursor Protein family using an automated, iterative meta-database search and constructed a highly resolved phylogeny. The analysis revealed that the ancestral gene for invertebrate and vertebrate Amyloid-β Precursor Protein gene families arose around metazoic speciation during the Ediacaran period. Synapomorphic frequencies found domain-specific conservation of sequence. Analyses of aggregation potential showed that potentially amyloidogenic sequences are a ubiquitous feature of vertebrate Amyloid-β Precursor Protein but are also found in echinoderm, nematode, and cephalochordate, and hymenoptera species homologues.</p><p><strong>Conclusions:</strong> The Amyloid-β Precursor Protein gene is ancient and highly conserved. The amyloid forming Amyloid-β domains may have been present in early deuterostomes, but more recent mutations appear to have resulted in potentially unrelated amyoid forming sequences. Our results further highlight that the species-specific physiological environment is as critical to Amyloid-β formation as the peptide sequence.</p><p><strong>Keywords:</strong> Amyloid, Alzheimer disease, Phylogenetics, In silico, Aggregation, Maximum parsimony, Bayesian inference</p></div><h2 id=background>Background
<a class=anchor href=#background>#</a></h2><p>The Amyloid-β Precursor Protein (AβPP, APP) has been intensively studied due to its role in the generation of pathogenic cortical plaques in Alzheimer Disease [1]. It belongs to a gene family with deep evolutionary origins and is a member of a highly conserved protein family of type-1 transmembrane proteins [2–4]. The AβPP family consists of up to three homologues in vertebrate species: AβPP, amyloid precursor like protein 1 (APLP-1), and amyloid precursor like protein 2 (APLP-2). Invertebrate species genomes encode a single homologue referred to as either amyloid precursor like 1 protein (APL-1) or AβPP-like 1 protein (APPL-1). Vertebrates and flatworms exhibit ubiquitous expression of at least one member of the AβPP family, while fruit flies express APPL-1 only in neurons. In all species, AβPP proteins are cleaved into multiple peptides and fragments by a series of proteases, but only vertebrate AβPP contains the sequence coding the pathological Amyloid-β (Aβ) peptide fragment.</p><p>The β-fold intrinsic to amyloid formation is a commonly observed biochemical property [5–7]. Amyloid formation is observed in non-pathological contexts from an efficient steric mechanism for storage of small peptide hormones to rudimentary forms of biological compartmentalization [5, 8]. The neuropathological changes observed in the brains of patients with Alzheimer Disease led to the formation of the Amyloid Hypothesis, which implicates both extracellular deposits of Aβ fibrils and low-order intracellular Aβ oligomers in the disruption of neuronal function, distortion of neural architecture, and induction of inflammation [1].</p><p>Mutations in the AβPP sequence and in associated proteases have been independently associated with familial early onset Alzheimer Disease characterized by rapidly progressive dementia and heavy Aβ plaque burden [9]. Recently, a protective mutation in AβPP reducing the formation of Aβ was identified [10]. However, >95% of sporadic Alzheimer Disease exhibits no mutation in the AβPP gene sequence. Further, deposition of Aβ is not limited to Alzheimer Disease. Aβ plaques have been observed in vascular dementias, Lewy body dementia, and Parkinson Disease with dementia, as well as in the brains of aged individuals without any cognitive deficits [11–14]. Together, these studies indicate that while the sequence of Aβ can contribute to the progression and severity of disease factors regulating the production by proteolysis and the degradation and clearance of Aβ, it also plays a critical role in generation of Aβ pathology.</p><p>Beyond the eponymous production of Aβ, AβPP processing produces other active peptides with functions ranging from hemostatic modulators to trophic factors to pro-apoptotic proteins [15–18]. There is a substantial body of knowledge focusing on the neural impacts of AβPP and Aβ. However, this family of proteins is also widely expressed in peripheral tissues of vertebrate species including skin, skeletal muscle, leukocytes, platelets, intestinal epithelia, pancreas, and adipose tissue. The function and regulation of non-neuronal AβPP are not fully understood [19–26].</p><p>The AβPP family is variably essential for viability among species. Experimental data show that the N-terminus of APL-1 is necessary for progression through molting stages by nematodes [27]. The C-terminus of at least one member of the AβPP family is necessary for viability in early parturition of knockout mouse models [28–30]. Drosophila models without APPL-1 show subtle neuronal patterning defects but remain viable and able to reproduce [31]. Zebrafish knockout models have impaired body development and synaptogenesis [32, 33]. Each of these models can be rescued by expression of truncated portions of AβPPs, indicating that absences of different domains are responsible for the observed lethality or defects in each model. Thus, the persistence of this protein family appears domain-dependent among species despite high evolutionary conservation of the entire gene.</p><p>The major conserved regions found in AβPP family proteins include two ectoplasmic domains (E1 and E2), which contain extracellular matrix and divalent cation binding regions and a growth-factor-like domain (GFLD), and the cytoplasmic region (E3) that contains a basolateral sorting signal (BLS) and an NPXY internalization sequence (YENPTY) (Figure 1). The corresponding nucleic acids coding the domains are termed D1, D2, and D3, respectively. Other important conserved domains include a Kunitz-protease inhibitor (KPI) domain found only in vertebrate AβPP and APLP-2, as well as the βA4 region that gives rise to Aβ in certain vertebrate species. Interestingly, it has been demonstrated that the corresponding region of APPL-1 in Drosophila melanogaster can form amyloid deposits when co-expressed in high levels with the Drosophila β-secretase [34]. Similar to the conservation of the E1, E2, and E3 domains, the βA4 region and corresponding regions may have arisen from a common ancestral domain [35]. It is not known when the amyloidogenic trait first appeared in this gene family nor why species with nearly identical Aβ sequences do not develop Aβ deposits.</p><p><img src=/img/amyloid-%ce%b2_fig1.jpg alt="Figure 1"></p><p>figure1</p><figcaption><p><strong>Figure 1 Conserved Regions of the Amyloid-β Precursor Protein Gene Family.</strong> Schematic representations of the five members of the Amyloid-β Precursor Protein (AβPP) gene family show multiple conserved domains: N-Terminal Signal peptide (NTS), growth factor-like domain (GFLD), heparin binding domain (Hep), copper binding domain (Cu), zinc binding domain (Zn), an acidic amino-acid rich region (D/E), collagen binding domain (Col), a basolateral sorting signal (BLS), and a clathrin-binding internalization signal domain (YENPTY). Certain members of the gene family also contain a Kunitz-protease inhibitor domain (KPI), the amyloid-β forming region (βA4), an OX-2 domain (diagonal hashmarks), a putative collagen binding domain (horizontal white hashmarks), and/or a glutamine and serine-rich region (Q/S).</p></figcaption><p>Full size image</p><p>Previous phylogenetic studies showed that this ancient protein family has been widely distributed among multicellular eukaryotes since at least the divergence of protostomia and deuterostomia [36]. These studies and corresponding conclusions are based on at most ten sequences that were trimmed and concatenated, focusing solely on the major conserved domains (D1, D2, and D3 only). Use of trimmed sequences does yield cleaner sequence alignments and better branch supports on the phylogenetic tree, but ignores potentially valuable evolutionary data encoded in adjoining regions. For the AβPP gene family in particular, the omission of Aβ from the analyses occludes an understanding of the evolution of the pathological eponymous domain. Despite wide distribution of the AβPP family across species, it is not know when amyloidogenic Aβ peptides first evolved. This study uses the full complement of available molecular sequence data to provide an <em>in silico</em> model of the evolutionary history of this essential gene family and the origin of the Aβ peptide.</p><h2 id=results>Results
<a class=anchor href=#results>#</a></h2><h3 id=phylogenetics-of-aβpp-gene-family>Phylogenetics of AβPP gene family
<a class=anchor href=#phylogenetics-of-a%ce%b2pp-gene-family>#</a></h3><p>Amino acid and nucleotide sequences were collected using an automated, iterative search method from <em>Entrez</em> Protein (GenPept) and <em>Entrez</em> Nucleotide (GenBank) (see Methods and Additional file 1: Table S1). Character matrices were generated in Mesquite 2.75 and aligned using Muscle 3.8.31) and the longest sequence for each species’ homologue(s) was retained [37, 38]. Amino acid and nucleotide trees were generated under maximum parsimony using TNT 1.1 and Bayesian inference using MrBayes 3.2 [39, 40].</p><p>The overall topology of the nucleotide and amino acid trees is similar among trees generated by maximum parsimony (Figure 2a and b) and Bayesian inference (Additional file 1: Figure S1a and b). Branch support data for the Bayesian analyses are found in Additional file 1: Figure S1 and support for the maximum parsimony analyses are found in Additional file 1: Figure S2.</p><p>figure2</p><p><img src=/img/amyloid-%ce%b2_fig2.jpg alt="Figure 2"></p><figcaption><p><strong>Figure 2 Phylogenetic Relationships of 103 members of the Amyloid-β Precursor Protein Gene Family.</strong> Shown are: (<strong>a</strong>) Phylogram showing the evolutionary relationships among the nucleotide sequences of the AβPP gene family; (<strong>b</strong>) Phylogram for the corresponding protein sequences. Trees were generated by maximum parsimony methods. Scale bars indicate character changes contributing to branch lengths.</p></figcaption><p>Full size image</p><p>The presence of an AβPP-like sequence in hydra (<em>Hydra magnipapillata</em>) and sea anemone (<em>Nematostella vectensis</em>) genomes suggests that the ancestral gene arose around metazoic divergence in the Ediacaran period, between 630–540 million years ago (Mya). No related sequences from single-celled organisms were found. A single member of the gene family has persisted across invertebrate species with a major divergence around the evolution of arthropods during the Cambrian period giving rise to APPL-1 (~500 Mya). Two gene duplication events occurred during the evolution of vertebrate species. Our search recovered a single AβPP-like gene for the cephalochordate lancelet (<em>Branchiostomidae floridae</em>) genome that was more closely related to mollusks and cnidarians than vertebrate sequences. The cartilaginous ray (<em>Narke japonica</em>) genome contains a single AβPP gene with high homology to human AβPP. The results indicate that AβPP and APLP-2 genes are present in the zebrafish (<em>Danio rerio</em>) but only AβPP was recovered for other members of class Osteichthyes (<em>Takifugu rubripes</em>, <em>Tetraodon fluviatilis</em>, and <em>Perca flavescens</em>). The majority of tetrapod genomes in this study contained all three members of the vertebrate AβPP gene family. Sequences for all three genes were found for Xenopus species, but APLP-1 sequences were not found for any members of class Aves or Reptilia.</p><p>Within the gene family, the nucleotide sequence phylogenetic trees (Figure 2a) indicate that AβPP and APLP-2 are more closely related than APLP-1 and APLP-2. Furthermore, the placement in the nucleotide phylogenetic tree suggests that APLP-1 may be the original vertebrate sequence. However, placement of the AβPP branch on the nucleotide tree is weakly supported under both maximum parsimony (47% resampling support, 12% relative Bremer; see Additional file 1: Figure S2a, b) and Bayesian inference (60% posterior probability; see Additional file 1: Figure S1a). In the amino acid sequence phylogenetic trees, APLP-1 and APLP-2 are more closely related and AβPP appears to be the original vertebrate peptide (Figure 2b). This arrangement has higher support for the placement of AβPP (65% resampling support, 100% relative Bremer support [maximum parsimony] Additional file 1: Figure S2c, d; 100% posterior probability [Bayesian inference]; Additional file 1: Figure S1b).</p><h3 id=persistence-of-amyloid-β>Persistence of amyloid-β
<a class=anchor href=#persistence-of-amyloid-%ce%b2>#</a></h3><p>The variability in the essential nature of the AβPP gene family can be observed by analyzing the evolutionary differences between related genes and shared residues according to specific functional domains. This was accomplished using synapomorphy frequency histograms. A synapomorphy is a trait or character shared by sister taxa of a clade that was derived from a previous common ancestor but not shared by taxa from another clade. Thus, synapomorphies contribute to the topology of a phylogenetic tree as factors in defining nodes on the tree [41]. Using the TNT program we collected synapomorphies present at each node of the consensus amino acid tree and examined the frequency of synapomorphy for each character across the sequence matrix. High frequencies of synapomorphy indicate residue changes at a given position make large contributions to the topology of the phylogenetic tree (conversely, low frequencies on the plots are associated with highly conserved domains/characters present in all terminal taxa groupings on the tree).</p><p>In this study, synapomorphies were first analyzed across all positions in the dataset (Figure 3) and then analyses were stratified by the major branches corresponding to APPL-1, APL-1, AβPP, APLP-2, and APLP-1 (Figure 4, Tables 1 and 2). The most highly variable region was the E2 domain, which accounted for 18.3% of synapomorphies on the whole tree, while the most highly conserved domain was the E3 domain (which had 6.9% of the residue synapomorphies). The E3 region of AβPP, encoded by exons 17 and 18, contributed 1.7% of the synapomorphic frequencies to that branch and 0.3% for the whole tree. By contrast, the same region of the other homologues contributed between 1.2 – 2.4% for the whole tree and 6.3 – 12.5% for each major branch. The βA4 domain, encoded by exons 16 and 17, contributed 0.5% and 3.1% of synapomorphies to the whole tree and AβPP branch, respectively. The exon 16–17 regions of APLP-1 and APLP-2 contributed 1.5% and 1% of synapomorphies to the whole tree, respectively, and 8.8% and 5.4% to their respective branches. These data showed the E3 region to be the most highly conserved part of the entire gene family and conservation of the βA4/E3 region is even stronger for vertebrate species.</p><p>figure3
<img src=/img/amyloid-%ce%b2_fig3.jpg alt="Figure 3"></p><figcaption><p><strong>Figure 3 Synapomorphic Character Frequencies for the Amyloid-β Precursor Protein Gene Family.</strong> Histogram of synapomorphic frequency generated for the whole gene family above the aligned amino acid character map. The colors of the character map were arbitrarily assigned by Mesquite. Lack of a colored line indicates a gap in the aligned sequences. The five major branches of the AβPP phylogenetic tree are indicated to the left of the map. The histogram is binned at 5 residues and scaled as a percentage.</p></figcaption><p>Full size image</p><p>figure4</p><p><img src=/img/amyloid-%ce%b2_fig4.jpg alt="Figure 4"></p><figcaption><p><strong>Figure 4 Synapomorphic Frequencies Correspond to Conserved Sequences in the Amyloid-β Precursor Protein Gene Family.</strong> Histograms of synapomorphic frequency generated for each major branch of the gene family above the representative schematic for each member and the amino acid character map. The colors of the character map were arbitrarily assigned by Mesquite. Lack of a colored line indicates a gap in the aligned sequences. Relevant taxonomic/cladistic classifications are indicated to the left of the maps. (<strong>a</strong>) APL-1; (<strong>b</strong>) APPL-1; (<strong>c</strong>) AbPP; (<strong>d</strong>) APLP-2; and (<strong>e</strong>) APLP-1 histograms are binned at 5 residues and scaled as percentages. Descriptions of the schematic regions are found in Figure 1.</p></figcaption>Full size image<p>Full size table</p><table><thead><tr><th>Domain</th><th>Whole tree</th><th>APL-1</th><th>APPL-1</th><th>AβPP</th><th>APLP-2</th><th>APLP-1</th></tr></thead><tbody><tr><td>N-Terminal Signal Peptide</td><td>11.61</td><td>1.25</td><td>1.49</td><td>2.05</td><td>3.31</td><td>3.57</td></tr><tr><td>E1</td><td>14.86</td><td>2.71</td><td>5.71</td><td>3.50</td><td>2.43</td><td>1.09</td></tr><tr><td>E2</td><td>18.33</td><td>4.95</td><td>4.18</td><td>3.22</td><td>2.83</td><td>4.17</td></tr><tr><td>E3</td><td>6.89</td><td>2.44</td><td>1.96</td><td>0.30</td><td>1.19</td><td>1.39</td></tr></tbody></table><figcaption><p><strong>Table 1 Synapomorphic frequencies within conserved domains</strong> APL-1: amyloid precursor like 1 protein; APPL-1: AβPP-like 1 protein; AβPP: amyloid-β precursor protein; APLP-2: amyloid precursor protein like protein 2; APLP-1: amyloid precursor protein like protein 1. This table shows the percentage of synapomorphy accounted for by the N-terminal signal peptide and the three major conserved regions of the peptide sequences for the entire phyologenetic tree and the relative contribution for the five major branches.</p></figcaption><table><thead><tr><th>Domain</th><th>APL-1</th><th>APPL-1</th><th>AβPP</th><th>APLP-2</th><th>APLP-1</th></tr></thead><tbody><tr><td>N-Terminal Signal Peptide</td><td>6.41</td><td>4.80</td><td>11.74</td><td>18.47</td><td>21.38</td></tr><tr><td>E1</td><td>13.91</td><td>18.36</td><td>20.07</td><td>13.56</td><td>6.54</td></tr><tr><td>E2</td><td>25.38</td><td>13.42</td><td>18.49</td><td>15.81</td><td>24.94</td></tr><tr><td>E3</td><td>12.50</td><td>6.30</td><td>1.70</td><td>6.63</td><td>8.31</td></tr></tbody></table><figcaption><p><strong>Table 2 Synapomorphic frequencies of domains within each branch</strong> APL-1: amyloid precursor like 1 protein; APPL-1: AβPP-like 1 protein; AβPP: amyloid-β precursor protein; APLP-2: amyloid precursor protein like protein 2; APLP-1: amyloid precursor protein like protein 1. This table shows the percentage of synapomorphy accounted for by the N-terminal signal peptide and the three major conserved regions of the peptide sequences within each branch of the phylogenetic trees.</p></figcaption><p>Full size table</p><h3 id=evolutionary-relationships-of-amyloid-β-formation-potential>Evolutionary relationships of amyloid-β formation potential
<a class=anchor href=#evolutionary-relationships-of-amyloid-%ce%b2-formation-potential>#</a></h3><p>Deposition of Aβ has been well documented in mammals; the sequence is generally >95% identical across mammals and all vertebrates express β- and γ-secretases [42–44]. The Guinea pig rodent (<em>Cavia porcelus</em>) and the common hare (<em>Oryctolagus cuniculus</em>) have been shown to generate Aβ plaques, but neither the <em>Mus musculus</em> nor <em>Rattus norvegicus</em> rodents naturally produce Aβ plaques [45–48]. Evidence of Aβ accumulation in other vertebrate species is sparse. Deposition of extracellular Aβ has only been documented in one member of class Osteichthyes: <em>Onchyrus</em> sockeye salmon; the sockeye AβPP gene has not been sequenced [49]. While some species of birds may generate Aβ plaques or vascular amyloid deposition, there is no evidence of plaque formation or extracellular deposition in reptiles and amphibians despite >90% sequence homology [47, 50]. No natural invertebrate amyloid-β plaques have been documented. Recently it was shown that the corresponding peptide from Drosophila can form an amyloid <em>in vivo</em> when co-expressed at high levels with the endogenous β-secretase gene [34].</p><p>In order to determine when Aβ formation first arose in evolution, we modeled β-sheet aggregation and amyloid formation probabilities for sequences corresponding to the human βA4 region using the AmylPred tool and PASTA server, both of which have been designed and validated using Aβ [51–54]. We found the C-terminal region of βA4 domains to have a high probability to form an amyloid or aggregate for nearly all sequences (Figures 5, 6, and 7). Only sequence from the silkmoth <em>Bombyx mori</em> had no amyloidogenic potential using both methods, although AmylPred predicted amyloidogenicity in a short C-terminal region. Very few sequences with amyloidogenic potential were found in the N-terminal region of invertebrate βA4 domains. The cnidarian sea anemone <em>Nematostella vectensis</em> and hydra <em>Hydra magnipapillata</em> exhibit strong C-terminal amyloid potential but little potential for amyloid formation in their N-terminal βA4 (Figure 6a and b). The nematode <em>Trichinella spiralis</em> is the only worm in our dataset with strongly amyloidogenic βA4 N-terminal sequences predicted by both methods (Figure 6d). The <em>Neohelice granulata</em> crab has a short N-terminal amyloid prone region (Figure 6e), but the water flea <em>Daphnia pulex</em> does not (Figure 6f). The Drosophila flies all express a potentially amyloidogenic N-terminal sequence predicted by AmylPred but with a PASTA energy of – 3.01 (Figure 6g), but no other members of Hymenoptera express amyloid prone sequence at the N-terminus. The squid <em>Loligo pealei</em> has PASTA energies &lt; − 4 for long stretches of the N-terminal βA4 region but no consensus support from AmylPred (Figure 6i). The sea slug <em>Aplysia californica</em> has a short region with probable amyloid forming potential supported by AmylPred and with a PASTA energy of – 3.16 (Figure 6j). The sea urchins <em>Stronglyocentrotus purpuratus</em> and <em>Paracentrotus lividus</em> also had a short N-terminal region predicted to form amyloid (Figure 6k). The cephalochordate lancelet <em>Branchiostoma floridae</em> had two long N-terminal regions with high amyloidogenic potential (Figure 6l). All AβPP sequences in the dataset exhibited a strongly amyloidogenic N-terminal region, though the rodent <em>M musculus</em> and <em>R novergicus</em> sequences had reduced PASTA energies compared to other vertebrates for their N-terminal regions (Figures 5 and 7). Interestingly, <em>Danio rerio</em> APLP2 showed an N-terminal amyloidogenic region (Figure 7e) while all other APLP2 sequences were identical at these residues and had a lower probability of forming an amyloid (Figure 7f). The APLP1 sequences for <em>Xenopus laevis</em> and <em>Monodelphis domestica</em> showed long sections of aggregation prone sequence (Figure 7g and h). The remaining APLP1 sequences, representing only placental mammals, show a region with lowered probability of aggregation or fibril formation (Figure 7i).</p><p><img src=/img/amyloid-%ce%b2_fig5.jpg alt="Figure 5"></p><figcaption><p><strong>Figure 5</strong> (See legend on next page.)</p><p>(See figure on previous page.)</p><p><strong>Figure 5 figure5 Amyloidogenic Potential in the Amyloid-β Sequence.</strong> Aligned representative amino acid sequences for the regions corresponding to exons 16 – 17 of human AβPP. Sequences tested are marked with boxes. Residues with AmylPred consensus and PASTA energies &lt; − 4 are in red; residues with AmylPred consensus and PASTA energies between – 3 and – 4 are in blue. Known secretase cleavage sites are marked by arrows.
Full size image</p></figcaption><p><img src=/img/amyloid-%ce%b2_fig6.jpg alt="Figure 6"></p><figcaption><p><strong>Figure 6 figure6 Amyloid Potential in Invertebrate Amyloid-β Sequence.</strong> Plots of probability of aggregation and stabilization of β-fibrils for each amino acid residue from PASTA for representative species. (a) Hydra magnipapillata, (b) Nematostella vectensis, (c) Caenorhabditis elegans, (d) Trichinella spiralis, (e) Neohelice granulata, (f) Daphnia pulex, (g) Drosophila melanogaster, (h) Aedes aegypti, (i) Loligo pealei, (j) Aplysia californica, (k) Stronglyocentrotus pupuratus, and (l) Branchiostoma floridae. Residues with PASTA energies &lt; − 4 and AmylPred consensus are marked with a black line; residues with PASTA energies between – 3 and – 4 and AmylPred consensus are marked with a grey line.
Full size image</p></figcaption><p><img src=/img/amyloid-%ce%b2_fig7.jpg alt="Figure 7"></p><figcaption><p><strong>Figure 7 figure7 Amyloid Potential in Vertebrate Amyloid-β Sequence.</strong> Plots of probability of aggregation and stabilization of β-fibrils for each amino acid residue from PASTA for representative species. (<strong>a</strong>) Narke japonica AβPP, (<strong>b</strong>) Danio rerio AβPP, (<strong>c</strong>) Homo sapiens AβPP, (<strong>d</strong>) Mus musculus AβPP, (<strong>e</strong>) Danio rerio APLP2, (<strong>f</strong>) Homo sapiens APLP2, (<strong>g</strong>) Xenopus laevis APLP1, (<strong>h</strong>) Monodelphis domestica APLP1, and (<strong>i</strong>) Homo sapiens APLP1. Residues with PASTA energies &lt; − 4 and AmylPred consensus are marked with a black line; residues with PASTA energies between – 3 and – 4 and AmylPred consensus are marked with a grey line.
Full size image</p></figcaption><h2 id=discussion>Discussion
<a class=anchor href=#discussion>#</a></h2><p>This study provides the most comprehensive phylogeny of the AβPP gene family based on available data to date. The analysis reveals that the ancestral sequence evolved during metazoic divergence, which is much earlier than previously thought. The results further suggest that AβPP itself was the first vertebrate sequence and that APLP-1 and 2 are likely derived from gene duplication of AβPP.</p><p>It is possible that the vertebrate gene family arose as a duplication of APLP-1 followed by a second duplication to form APLP-2 and AβPP. However, it is also possible that the original duplication gave rise to APLP-2 and AβPP after which a duplication of APLP-2 gave rise to APLP-1. The search strategy used in this study found APLP-1 sequences only in tetrapods, AβPP in both cartilaginous and bony fish, and APLP-2 in one bony fish and most tetrapods.</p><p>We found that the E3 C-terminal region of the protein is essentially unchanged since the divergence of jawed vertebrates during the Ordovician period and that amyloidogenic Aβ was present around this evolutionary step. Its persistence is likely due to the overlap with the E3 domain. It has been shown that the E3 domain is essential for life in mammals and the βA4 domain contains an HD motif with evidence of positive selection, both of which may explain some of the persistence of amyloidogenic Aβ in the mammalian genome [30, 35]. Our analysis also found evidence of aggregation prone C-terminal regions in nearly all sequences in the dataset, which is not surprising as this is part of the transmembrane region high in hydrophobic residues, but a stable β-fold requires two regions within the peptide. Sequences with two separate domains capable of forming and stabilizing an amyloid were rare in protostomes, suggesting the characteristic developed after the divergence of deuterostomes and protostomes or was subsequently lost through mutation (Figure 8). Of particular note, the Drosophila sequence is predicted to form an amyloid but at a lowered probability than mammalian Aβ and there is experimental evidence that it can form fibrillar Aβ <em>in vivo</em> [34]. As no other Hymenoptera species in the study show amyloid potential, this likely represents a new mutation in the development of the fruitfly species. Interestingly, non-vertebrate deuterostome species in this study have amyloidogenic sequence but little homology to the mammalian Aβ sequence, suggesting that early amyloid prone regions may have evolved prior to the divergence of echinoderms, hemichordate, and chordate species. The main sequence variations arise from the N-terminal region aligned to <em>Homo sapiens</em> AβPP exon 16. APLP-2 from the zebrafish <em>Danio rerio</em> also showed amyloidogenic potential and all other APLP-2 had reduced potential to form Aβ. This is may be a result of mutation or indels in the exon 16 region during or after the gene duplication events giving rise to APLP-1 and APLP-2.</p><p><img src=/img/amyloid-%ce%b2_fig8.jpg alt="Figure 8"></p><figcaption><p><strong>Figure 8</strong> (See legend on next page.)</p><p>(See figure on previous page.)
<strong>Figure 8 figure8 Evolutionary Relationship of Amyloid-β Formation Potential.</strong> Phylogram of the protein sequence for the AβPP family color-coded by prediction of Amyloid-β formation. Red: High potential (PASTA energies &lt; − 4 and AmylPred consensus); Blue: Low potential (PASTA energies between – 3 and – 4 and AmylPred consensus); Black: No potential. Species with unique sequences demonstrating potential for Amyloid-β formation are labeled.
Full size image</p></figcaption><p>Because the data used in this study were based on <em>in silico</em> search strategies from deposited sequences in public repositories (GenBank and GenPept), it cannot be assumed that these data are necessarily complete for each species (i.e., a <em>de novo</em> sequencing was not performed for each species studied). Nonetheless, these data support the hypotheses that AβPP is the ancestral sequence for vertebrates, gene duplication after the speciation of cartilaginous and bony fish gave rise to APLP-2, and a subsequent partial or degenerate duplication of APLP-2 following the speciation of tetrapods gave rise to APLP-1. Some species may have subsequently lost either APLP-1 or APLP-2 genes.</p><p>The sequence difference in <em>Mus musculus</em> and <em>Rattus norvegicus</em> results from three amino acid substitutions from three single nucleotide changes. Whether the lack of amyloidogenesis in these particular rodents comes from these three changes or from other physiological considerations is unclear, but the presence of identical sequence in other rodents and mammals in general suggests that the ancestral species to mice and rats evolved around amyloidogenic Aβ. The lack of data on Aβ deposition in fish, birds, reptiles, and amphibians also suggests unknown physiological adaptations may limit Aβ production or deposition. Recently a mutation encoding a change from alanine to threonine at position 673 of AβPP was found to be protective against developing Alzheimer Disease, likely through reduction of β-secretase processing at that site [10]. It is interesting to note that all fish sequences in this study, with the exception of <em>Danio rerio</em>, have a threonine at this position, suggesting β-secretase processing may be reduced in these animals. In addition to processes that may increase or decrease Aβ production by regulating secretase efficiency or transcription, the presence of a β-secretase in the gene repertoire is an important consideration.</p><p>A whole genome assembly for <em>Nematostella vectensis</em> indicate the presence of the secretases but no studies have examined amyloid formation [55]. A genome for <em>Hydra magnipapillata</em> predicted the presence of a γ-secretase, but not a β-secretase (REFSEQ NW_002165109). Experimental evidence suggests that the nematode <em>Caenorhabditis elegans</em> does not express a β-secretase, although both α- and γ-secretases have been identified [56]. A search of Entrez Nucleotide returned no β-secretase sequences for other nematodes, crustaceans, hymenoptera, or lepidoptera in our dataset.</p><p>The increased understanding of disease genetics and increasing availability of molecular sequence data provide an opportunity to harness evolutionary approaches to provide deep insights pertaining to the etiology of disease. Using this approach we found the AβPP family to have origins in the speciation of the metazoic lineage and propose that ancestral Aβ may have arisen as deuterostomia and protostomia diverged. However, other mutations may continue to produce amyloidogenic sequences in this domain, as seen with Drosophila or unknown physiological factors may play a role in preventing Aβ formation as in mice and rats. The approach developed here may be widely applicable to the study of other critical disease genes and builds a foundation for further studies on the co-evolution of Alzheimer Disease associated proteins (e.g., co-evolution of ApoE or β-secretase with AβPP) that may yield novel approaches to treating or preventing Aβ formation.</p><h2 id=methods>Methods
<a class=anchor href=#methods>#</a></h2><h3 id=dataset-collection-and-alignment>Dataset collection and alignment
<a class=anchor href=#dataset-collection-and-alignment>#</a></h3><p>Amino acid sequences were collected through Entrez Protein using a combination of search terms and sequence similarity searches. First, based on previous studies of sequences from the Amyloid-β Precursor Protein [2–4, 36] family five sets of metadata-based search terms developed and used to identify those sequences from across the Amyloid-β Precursor Protein family: (1) <code>"App"[gene name] AND "animals"[porgn:__txid33208];</code> (2) <code>"aplp1"[gene name] and "animals"[porgn:__txid33208];</code> (3) <code>"aplp2"[gene name] and "animals"[porgn:__txid33208];</code> (4) <code>"apl-1"[gene name] and "nematodes"[porgn:__txid6231];</code> and (5) <code>"app_amyloid"</code>. Sequences for which the organism was either “Unknown” or listed as a “synthetic construct” were removed. Next, a stringent (E-value = 0.0) blastp (BLAST+ v.2.2.26) was used to search Entrez Protein for potential orthologous amino acid sequences for each of the sequences identified in the metadata-based search from the non-redundant protein database. An additional stringent blastp search was then done iteratively for each new sequence identified, until no additional sequences were found. The resulting dataset (which contained 435 sequences) was then subjected to multiple sequence alignment using MUSCLE v.3.8.31 [38]. The multiple sequence alignment was manually inspected (by viewing the data in Mesquite 2.75 [37]) to identify the one longest representative sequence per taxon (e.g., only the sequence for human AβPP770 which contains all transcribed and translated exons was kept). As sequences were removed from the dataset, the multiple sequence alignment was redone. The resulting dataset reflected 103 taxa corresponding to 67 species. Based on identifiers within GenPept records, corresponding nucleic acid sequences were then collected for each amino acid sequence. These nucleotide sequences were also subjected to multiple sequence alignment using MUSCLE. Character maps were generated using the Mesquite character matrices.</p><h3 id=generation-of-phylogenetic-trees>Generation of phylogenetic trees
<a class=anchor href=#generation-of-phylogenetic-trees>#</a></h3><p>Trees were obtained by maximum parsimony using TNT 1.1 and Bayesian inference using MrBayes 3.2.0 [39, 40]. For analyses in TNT, the ‘aquickie.run’ script was used to guide the search, which aimed to find the optimal score 20 times independently, using defaults of &ldquo;xmult&rdquo; plus 10 cycles of tree-drifting. This resulted in 131 nucleotide trees from more than 8x108 rearrangements and 103714 amino acid trees from more than 7x108 rearrangements. For consensus tree calculation, trees were TBR-collapsed, Bremer group supports calculated by TBR-swapping, and bootstrap resampling by 100 replications of symmetric resampling with a single random addition (see Additional file 1).</p><p>For MrBayes, the Metropolis-coupled Markov chain Monte Carlo analysis was set for 2 runs with 4 chains each with a temperature of 0.2 degrees. A General Time Reversible (GTR) model with a Dirichlet (flat) probability distribution of nucleotide rate change parameters, stationary nucleotide frequencies, no specified shape parameter for the gamma distribution of rate variation, and no invariable sites was used for the nucleotide analyses; this is the default prior model for nucleotide matrices in MrBayes. All runs favored the WAG rate matrix as the prior model for the amino acid analyses [57].</p><p>Markov Chain analysis was continued until the runs converged, when the standard deviation of the split frequencies remained &lt;0.01 and likelihood analysis found the potential scale reduction factor approached 1.0 [58]. For the nucleotide modeling this took more than 3x10<sup>6</sup> generations; for the protein analysis this took more than 2x10<sup>6</sup> generations. Consensus trees were constructed using the 50% majority rule with 95% cumulative posterior probability from 925 nucleotide trees and 1,591 amino acid trees (see Additional file 1). All tree diagrams were generated in either Dendroscope 3.1.0 or FigTree 1.3.1 [59, 60].</p><h3 id=synapomorphic-frequencies>Synapomorphic frequencies
<a class=anchor href=#synapomorphic-frequencies>#</a></h3><p>Unambiguous synapomorphies at each node were generated in TNT for the maximum parsimony consensus trees. The frequency of a given character being synapomorphic at a given node was examined for the entire amino acid tree and for each of the five major branches. Probabilistic models of synapomorphy have been developed to address the confounding of homoplasy and lend statistical support to defining a character as synapomorphic as opposed to homoplasious [41]. While these are important considerations for higher resolution analysis of a gene family, use of simple statistical analysis for such a large and diverse dataset is a reasonable approach to defining areas of conservation or change, accepting internal error for random mutation producing homoplasy or loss of an actual synapomorphy.</p><h3 id=aggregation-modeling>Aggregation modeling
<a class=anchor href=#aggregation-modeling>#</a></h3><p>There are a number of programs available for modeling β-folding and aggregation of amyloidogenic peptides [61]. AmylPred is a consensus tool that predicts β-folding and aggregation based on a set of five published methods and uses agreement of 2 or more methods for determining consensus [54]. PASTA predicts stabilizing sequences in β-fibrillar structures using a calculation of the change of energy from pairing between amino acid sequences [53]. Regions that are known to form ordered β-fibril structures have a PASTA energy less than – 4. Using aligned amino acid sequences coded by <em>Homo sapiens</em> AβPP exons 16 and 17, we examined the corresponding βA4 region across all taxa and used known secretase cleavage sites to determine the aligned sequences for submission to AmylPred and PASTA [62–64]. Where cleavage sites are not known from previous studies, boundaries were chosen based on similar species and sequences. In cases where there was no clear similarity, boundaries were extended to correspond with Homo sapiens Aβ42. PASTA energies were collected until greater than – 2 by sequential truncation of the C-terminus for each sequence.
There are a number of programs available for modeling β-folding and aggregation of amyloidogenic peptides [61]. AmylPred is a consensus tool that predicts β-folding and aggregation based on a set of five published methods and uses agreement of 2 or more methods for determining consensus [54]. PASTA predicts stabilizing sequences in β-fibrillar structures using a calculation of the change of energy from pairing between amino acid sequences [53]. Regions that are known to form ordered β-fibril structures have a PASTA energy less than – 4. Using aligned amino acid sequences coded by <em>Homo sapiens</em> AβPP exons 16 and 17, we examined the corresponding βA4 region across all taxa and used known secretase cleavage sites to determine the aligned sequences for submission to AmylPred and PASTA [62–64]. Where cleavage sites are not known from previous studies, boundaries were chosen based on similar species and sequences. In cases where there was no clear similarity, boundaries were extended to correspond with <em>Homo sapiens</em> Aβ42. PASTA energies were collected until greater than – 2 by sequential truncation of the C-terminus for each sequence.</p><h2 id=electronic-supplementary-material>Electronic supplementary material
<a class=anchor href=#electronic-supplementary-material>#</a></h2><h3 id=additional-file-figure-s1>Additional file: Figure S1.
<a class=anchor href=#additional-file-figure-s1>#</a></h3><p><strong>Additional file 1: Figure S1.</strong> Phylogenetic Relationships among the Amyloid-β Precursor Protein Gene Family from Baysian Inference. <strong>a</strong>, Phylogram showing the evolutionary relationships among the nucleotide sequences of the AβPP gene family. <strong>b</strong>, Phylogram for the corresponding protein sequences. Trees were generated by Bayesian inference methods and show posterior probability values are each node. <strong>Figure S2.</strong> Branch Supports for Phylogenetic Trees. Symmetric bootstrap re-sampling and Bremer supports, for nucleotide trees (<strong>a</strong> and <strong>b</strong>, respectively) and for amino acid trees (<strong>c</strong> and <strong>d</strong>, respectively).</p><h3 id=table-s1-taxa-species-names-and-sequence-accession-numbers-pdf-1-mb>Table S1. Taxa Species Names and Sequence Accession Numbers. (PDF 1 MB)
<a class=anchor href=#table-s1-taxa-species-names-and-sequence-accession-numbers-pdf-1-mb>#</a></h3><p><strong>Table S1.</strong> Taxa Species Names and Sequence Accession Numbers.</p><p>Authors’ original submitted files for images</p><p>Below are the links to the authors’ original submitted files for images.</p><p>Authors’ original file for figure 1
Authors’ original file for figure 2
Authors’ original file for figure 3
Authors’ original file for figure 4
Authors’ original file for figure 5
Authors’ original file for figure 6
Authors’ original file for figure 7
Authors’ original file for figure 8</p><h2 id=abbreviations>Abbreviations
<a class=anchor href=#abbreviations>#</a></h2><dl><dt>Aβ:</dt><dd>Amyloid-β</dd><dt>AβPP:</dt><dd>APP, Amyloid-β Precursor Protein</dd><dt>APPL-1:</dt><dd>Amyloid-β Precursor Protein-like 1 protein</dd><dt>APLP-1:</dt><dd>Amyloid precursor like protein 1</dd><dt>APLP-2:</dt><dd>Amyloid precursor like protein 2</dd><dt>APL-1:</dt><dd>Amyloid precursor like 1 protein</dd><dt>BLS:</dt><dd>Basolateral sorting signal</dd><dt>GFLD:</dt><dd>Growth-factor-like domain</dd><dt>KPI:</dt><dd>Kunitz-protease inhibitor</dd><dt>Mya:</dt><dd>Million years ago.</dd></dl><h2 id=references>References
<a class=anchor href=#references>#</a></h2><ol><li>Hardy J, Selkoe DJ: <strong>The amyloid hypothesis of Alzheimer&rsquo;s disease: progress and problems on the road to therapeutics.</strong> <em>Science</em>. 2002, <strong>297</strong> (5580): 353-356. 10.1126/science.1072994.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=2><li>Guo Q, Wang Z, Li H, Wiese M, Zheng H: <strong>APP physiological and pathophysiological functions: insights from animal models.</strong> <em>Cell Res</em>. 2012, <strong>22</strong> (1): 78-89. 10.1038/cr.2011.116.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=3><li>Jacobsen KT, Iverfeldt K: <strong>Amyloid precursor protein and its homologues: a family of proteolysis-dependent receptors.</strong> <em>Cell Mol Life Sci</em>. 2009, <strong>66</strong> (14): 2299-2318. 10.1007/s00018-009-0020-8.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=4><li>Walsh DM, Minogue AM, Sala Frigerio C, Fadeeva JV, Wasco W, Selkoe DJ: <strong>The APP family of proteins: similarities and differences.</strong> <em>Biochem Soc Trans</em>. 2007, <strong>35</strong> (Pt 2): 416-420.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=5><li>Greenwald J, Riek R: <strong>Biology of amyloid: structure, function, and regulation.</strong> <em>Structure</em>. 2010, <strong>18</strong> (10): 1244-1260. 10.1016/j.str.2010.08.009.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=6><li>Chernoff YO: <strong>Amyloidogenic domains, prions and structural inheritance: rudiments of early life or recent acquisition?.</strong> <em>Curr Opin Chem Biol</em>. 2004, <strong>8</strong> (6): 665-671. 10.1016/j.cbpa.2004.09.002.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=7><li>Brack A, Orgel LE: <strong>Beta structures of alternating polypeptides and their possible prebiotic significance.</strong> <em>Nature</em>. 1975, <strong>256</strong> (5516): 383-387. 10.1038/256383a0.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=8><li>Inge-Vechtomov SG, Zhouravleva GA, Chernoff YO: <strong>Biological roles of prion domains.</strong> <em>Prion</em>. 2007, <strong>1</strong> (4): 228-235. 10.4161/pri.1.4.5059.
PubMed Central</li></ol><p>Article</p><p>PubMed</p><p>Google Scholar</p><ol start=9><li>Rosenberg RN: <strong>The molecular and genetic basis of AD: the end of the beginning: the 2000 Wartenberg lecture.</strong> <em>Neurology</em>. 2000, <strong>54</strong> (11): 2045-2054. 10.1212/WNL.54.11.2045.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=10><li><p>Jonsson T, Atwal JK, Steinberg S, Snaedal J, Jonsson PV, Bjornsson S, Stefansson H, Sulem P, Gudbjartsson D, Maloney J, et al: <strong>A mutation in APP protects against Alzheimer&rsquo;s disease and age-related cognitive decline.</strong> <em>Nature</em>. 2012, advance online publication
Google Scholar</p></li><li><p>Barrachina M, Dalfo E, Puig B, Vidal N, Freixes M, Castano E, Ferrer I: <strong>Amyloid-beta deposition in the cerebral cortex in Dementia with Lewy bodies is accompanied by a relative increase in AbetaPP mRNA isoforms containing the Kunitz protease inhibitor.</strong> <em>Neurochem Int</em>. 2005, <strong>46</strong> (3): 253-260. 10.1016/j.neuint.2004.08.006.
Article</p></li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=12><li>Yamada M, Naiki H: <strong>Cerebral amyloid angiopathy.</strong> <em>Prog Mol Biol Transl Sci</em>. 2012, <strong>107</strong>: 41-78.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=13><li>Liang WS, Dunckley T, Beach TG, Grover A, Mastroeni D, Ramsey K, Caselli RJ, Kukull WA, McKeel D, Morris JC, et al: <strong>Neuronal gene expression in non-demented individuals with intermediate Alzheimer&rsquo;s Disease neuropathology.</strong> <em>Neurobiol Aging</em>. 2010, <strong>31</strong> (4): 549-566. 10.1016/j.neurobiolaging.2008.05.013.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=14><li><p>Kotzbauer PT, Cairns NJ, Campbell MC, Willis AW, Racette BA, Tabbal SD, Perlmutter JS: <strong>Pathologic Accumulation of alpha-Synuclein and Abeta in Parkinson Disease Patients With Dementia.</strong> <em>Arch Neurol</em>. 2012, 1-6.
Google Scholar</p></li><li><p>Nikolaev A, McLaughlin T, O&rsquo;Leary DD, Tessier-Lavigne M: <strong>APP binds DR6 to trigger axon pruning and neuron death via distinct caspases.</strong> <em>Nature</em>. 2009, <strong>457</strong> (7232): 981-989. 10.1038/nature07767.
PubMed Central</p></li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=16><li>Sinha S, Lieberburg I: <strong>Cellular mechanisms of beta-amyloid production and secretion.</strong> <em>Proc Natl Acad Sci U S A</em>. 1999, <strong>96</strong> (20): 11049-11053. 10.1073/pnas.96.20.11049.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=17><li>Thinakaran G, Koo EH: <strong>APP trafficking, processing and function.</strong> <em>J Biol Chem</em>. 2008, <strong>283</strong> (44): 29615-10.1074/jbc.R800019200.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=18><li>Xu F, Davis J, Miao J, Previti ML, Romanov G, Ziegler K, Van Nostrand WE: <strong>Protease nexin-2/amyloid beta-protein precursor limits cerebral thrombosis.</strong> <em>Proc Natl Acad Sci U S A</em>. 2005, <strong>102</strong> (50): 18135-18140. 10.1073/pnas.0507798102.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=19><li>Bush AI, Martins RN, Rumble B, Moir R, Fuller S, Milward E, Currie J, Ames D, Weidemann A, Fischer P, et al: <strong>The amyloid precursor protein of Alzheimer&rsquo;s disease is released by human platelets.</strong> <em>J Biol Chem</em>. 1990, <strong>265</strong> (26): 15977-15983.
CAS</li></ol><p>PubMed</p><p>Google Scholar</p><ol start=20><li>Joachim CL, Mori H, Selkoe DJ: <strong>Amyloid beta-protein deposition in tissues other than brain in Alzheimer&rsquo;s disease.</strong> <em>Nature</em>. 1989, <strong>341</strong> (6239): 226-230. 10.1038/341226a0.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=21><li>Lee YH, Tharp WG, Maple RL, Nair S, Permana PA, Pratley RE: <strong>Amyloid precursor protein expression is upregulated in adipocytes in obesity.</strong> <em>Obesity (Silver Spring, Md)</em>. 2008, <strong>16</strong> (7): 1493-1500. 10.1038/oby.2008.267.
Article</li></ol><p>CAS</p><p>Google Scholar</p><ol start=22><li>Galloway S, Jian L, Johnsen R, Chew S, Mamo JC: <strong>Deta-amyloid or its precursor protein is found in epithelial cells of the small intestine and is stimulated by high-fat feeding.</strong> <em>J Nutr Biochem</em>. 2007, <strong>18</strong> (4): 279-284. 10.1016/j.jnutbio.2006.07.003.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=23><li>Hansel DE, Rahman A, Wehner S, Herzog V, Yeo CJ, Maitra A: <strong>Increased expression and processing of the Alzheimer amyloid precursor protein in pancreatic cancer may influence cellular proliferation.</strong> <em>Cancer Res</em>. 2003, <strong>63</strong> (21): 7032-7037.
CAS</li></ol><p>PubMed</p><p>Google Scholar</p><ol start=24><li>Herzog V, Kirfel G, Siemes C, Schmitz A: <strong>Biological roles of APP in the epidermis.</strong> <em>Eur J Cell Biol</em>. 2004, <strong>83</strong> (11–12): 613-624.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=25><li>Kuo YM, Kokjohn TA, Watson MD, Woods AS, Cotter RJ, Sue LI, Kalback WM, Emmerling MR, Beach TG, Roher AE: <strong>Elevated abeta42 in skeletal muscle of Alzheimer disease patients suggests peripheral alterations of AbetaPP metabolism.</strong> <em>Am J Pathol</em>. 2000, <strong>156</strong> (3): 797-805. 10.1016/S0002-9440(10)64947-4.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=26><li>Kang J, Lemaire HG, Unterbeck A, Salbaum JM, Masters CL, Grzeschik KH, Multhaup G, Beyreuther K, Muller-Hill B: <strong>The precursor of Alzheimer&rsquo;s disease amyloid A4 protein resembles a cell-surface receptor.</strong> <em>Nature</em>. 1987, <strong>325</strong> (6106): 733-736. 10.1038/325733a0.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=27><li>Hornsten A, Lieberthal J, Fadia S, Malins R, Ha L, Xu X, Daigle I, Markowitz M, O&rsquo;Connor G, Plasterk R, et al: <strong>APL-1, a Caenorhabditis elegans protein related to the human beta-amyloid precursor protein, is essential for viability.</strong> <em>Proc Natl Acad Sci U S A</em>. 2007, <strong>104</strong> (6): 1971-1976. 10.1073/pnas.0603997104.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=28><li>von Koch CS, Zheng H, Chen H, Trumbauer M, Thinakaran G, van der Ploeg LH, Price DL, Sisodia SS: <strong>Generation of APLP2 KO mice and early postnatal lethality in APLP2/APP double KO mice.</strong> <em>Neurobiol Aging</em>. 1997, <strong>18</strong> (6): 661-669. 10.1016/S0197-4580(97)00151-6.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=29><li>Heber S, Herms J, Gajic V, Hainfellner J, Aguzzi A, Rulicke T, von Kretzschmar H, von Koch C, Sisodia S, Tremml P, et al: <strong>Mice with combined gene knock-outs reveal essential and partially redundant functions of amyloid precursor protein family members.</strong> <em>J Neurosci</em>. 2000, <strong>20</strong> (21): 7951-7963.
CAS</li></ol><p>PubMed</p><p>Google Scholar</p><ol start=30><li>Li H, Wang Z, Wang B, Guo Q, Dolios G, Tabuchi K, Hammer RE, Sudhof TC, Wang R, Zheng H: <strong>Genetic dissection of the amyloid precursor protein in developmental function and amyloid pathogenesis.</strong> <em>J Biol Chem</em>. 2010, <strong>285</strong> (40): 30598-30605. 10.1074/jbc.M110.137729.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=31><li>Poeck B, Strauss R, Kretzschmar D: <strong>Analysis of amyloid precursor protein function in Drosophila melanogaster.</strong> <em>Exp Brain Res</em>. 2012, <strong>217</strong> (3–4): 413-421.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=32><li>Song P, Pimplikar SW: <strong>Knockdown of amyloid precursor protein in zebrafish causes defects in motor axon outgrowth.</strong> <em>PLoS One</em>. 2012, <strong>7</strong> (4): e34209-10.1371/journal.pone.0034209.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=33><li>Joshi P, Liang JO, DiMonte K, Sullivan J, Pimplikar SW: <strong>Amyloid precursor protein is required for convergent-extension movements during Zebrafish development.</strong> <em>Dev Biol</em>. 2009, <strong>335</strong> (1): 1-11. 10.1016/j.ydbio.2009.07.041.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=34><li>Carmine-Simmen K, Proctor T, Tschape J, Poeck B, Triphan T, Strauss R, Kretzschmar D: <strong>Neurotoxic effects induced by the Drosophila amyloid-beta peptide suggest a conserved toxic function.</strong> <em>Neurobiol Dis</em>. 2009, <strong>33</strong> (2): 274-281. 10.1016/j.nbd.2008.10.014.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=35><li>Miklos I, Zadori Z: <strong>Positive evolutionary selection of an HD motif on Alzheimer precursor protein orthologues suggests a functional role.</strong> <em>PLoS Comput Biol</em>. 2012, <strong>8</strong> (2): e1002356-10.1371/journal.pcbi.1002356.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=36><li>Coulson EJ, Paliga K, Beyreuther K, Masters CL: <strong>What the evolution of the amyloid protein precursor supergene family tells us about its function.</strong> <em>Neurochem Int</em>. 2000, <strong>36</strong> (3): 175-184. 10.1016/S0197-0186(99)00125-4.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=37><li><p>Maddison WP, Maddison DP: <em>Mesquite: a modular system for evolutionary analysis</em>. 2011, 275
Google Scholar</p></li><li><p>Edgar RC: <strong>MUSCLE: multiple sequence alignment with high accuracy and high throughput.</strong> <em>Nucleic Acids Res</em>. 2004, <strong>32</strong> (5): 1792-1797. 10.1093/nar/gkh340.
PubMed Central</p></li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=39><li>Goloboff P, Farris J, Nixon K: <strong>TNT, a free program for phylogenetic analysis.</strong> <em>Cladistics</em>. 2008, <strong>24</strong>: 774-786. 10.1111/j.1096-0031.2008.00217.x.
Article</li></ol><p>Google Scholar</p><ol start=40><li>Ronquist F, Huelsenbeck JP: MrBayes 3: <strong>Bayesian phylogenetic inference under mixed models.</strong> <em>Bioinformatics</em>. 2003, <strong>19</strong> (12): 1572-1574. 10.1093/bioinformatics/btg180.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=41><li><p>Turjak M, Trontelj P: <strong>A method for measuring support for synapomorphy using character state distributions on phylogenetic trees.</strong> <em>Cladistics</em>. 2012, <strong>1</strong>: 1-12.
Google Scholar</p></li><li><p>Finch CE, Sapolsky RM: <strong>The evolution of Alzheimer disease, the reproductive schedule, and apoE isoforms.</strong> <em>Neurobiol Aging</em>. 1999, <strong>20</strong> (4): 407-428. 10.1016/S0197-4580(99)00053-6.
Article</p></li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=43><li><p>Sarasa M, Gallego C: <strong>Alzheimer-Like Neurodegeneration as a Probable Cause of Cetacean Stranding.</strong> <em>Poster session presented at: 5th FENS Forum: 2006</em>; Vienna, Austria. 2006
Google Scholar</p></li><li><p>Selkoe DJ, Bell DS, Podlisny MB, Price DL, Cork LC: <strong>Conservation of brain amyloid proteins in aged mammals and humans with Alzheimer&rsquo;s disease.</strong> <em>Science</em>. 1987, <strong>235</strong> (4791): 873-877. 10.1126/science.3544219.
Article</p></li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=45><li>Beach TG: <strong>Physiologic origins of age-related beta-amyloid deposition.</strong> <em>Neurodegener Dis</em>. 2008, <strong>5</strong> (3–4): 143-145.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=46><li>Ionov ID, Pushinskaya II: <strong>Amyloid-beta production in aged guinea pigs: atropine-induced enhancement is reversed by naloxone.</strong> <em>Neurosci Lett</em>. 2010, <strong>480</strong> (1): 83-86. 10.1016/j.neulet.2010.06.010.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=47><li>Dayan AD: <strong>Comparative neuropathology of ageing. Studies on the brains of 47 species of vertebrates.</strong> <em>Brain</em>. 1971, <strong>94</strong> (1): 31-42. 10.1093/brain/94.1.31.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=48><li>Flood DG, Lin YG, Lang DM, Trusko SP, Hirsch JD, Savage MJ, Scott RW, Howland DS: <strong>A transgenic rat model of Alzheimer&rsquo;s disease with extracellular Abeta deposition.</strong> <em>Neurobiol Aging</em>. 2009, <strong>30</strong> (7): 1078-1090. 10.1016/j.neurobiolaging.2007.10.006.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=49><li>Maldonado TA, Jones RE, Norris DO: <strong>Timing of neurodegeneration and beta-amyloid (Abeta) peptide deposition in the brain of aging kokanee salmon.</strong> <em>J Neurobiol</em>. 2002, <strong>53</strong> (1): 21-35. 10.1002/neu.10090.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=50><li>Nakayama H, Katayama K, Ikawa A, Miyawaki K, Shinozuka J, Uetsuka K, Nakamura S, Kimura N, Yoshikawa Y, Doi K: <strong>Cerebral amyloid angiopathy in an aged great spotted woodpecker (Picoides major).</strong> <em>Neurobiol Aging</em>. 1999, <strong>20</strong> (1): 53-56. 10.1016/S0197-4580(99)00004-4.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=51><li>Trovato A, Seno F, Tosatto SC: <strong>The PASTA server for protein aggregation prediction.</strong> <em>Protein Eng Des Sel</em>. 2007, <strong>20</strong> (10): 521-523. 10.1093/protein/gzm042.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=52><li>Fernandez-Escamilla AM, Rousseau F, Schymkowitz J, Serrano L: <strong>Prediction of sequence-dependent and mutational effects on the aggregation of peptides and proteins.</strong> <em>Nat Biotechnol</em>. 2004, <strong>22</strong> (10): 1302-1306. 10.1038/nbt1012.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=53><li>Trovato A, Chiti F, Maritan A, Seno F: <strong>Insight into the structure of amyloid fibrils from the analysis of globular proteins.</strong> <em>PLoS Comput Biol</em>. 2006, <strong>2</strong> (12): e170-10.1371/journal.pcbi.0020170.
PubMed Central</li></ol><p>Article</p><p>PubMed</p><p>Google Scholar</p><ol start=54><li>Frousios KK, Iconomidou VA, Karletidi CM, Hamodrakas SJ: <strong>Amyloidogenic determinants are usually not buried.</strong> <em>BMC Struct Biol</em>. 2009, <strong>9</strong>: 44-10.1186/1472-6807-9-44.
PubMed Central</li></ol><p>Article</p><p>PubMed</p><p>Google Scholar</p><ol start=55><li>Putnam NH, Srivastava M, Hellsten U, Dirks B, Chapman J, Salamov A, Terry A, Shapiro H, Lindquist E, Kapitonov VV, et al: <strong>Sea anemone genome reveals ancestral eumetazoan gene repertoire and genomic organization.</strong> <em>Science</em>. 2007, <strong>317</strong> (5834): 86-94. 10.1126/science.1139158.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=56><li>Link CD: <strong>C. elegans models of age-associated neurodegenerative diseases: lessons from transgenic worm models of Alzheimer&rsquo;s disease.</strong> <em>Exp Gerontol</em>. 2006, <strong>41</strong> (10): 1007-1013. 10.1016/j.exger.2006.06.059.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=57><li>Whelan S, Goldman N: <strong>A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach.</strong> <em>Mol Biol Evol</em>. 2001, <strong>18</strong> (5): 691-699. 10.1093/oxfordjournals.molbev.a003851.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=58><li>Gelman A, Rubin D: <strong>Inference from Iterative Simulation using Multiple Sequences.</strong> <em>Stat Sci</em>. 1992, <strong>7</strong>: 457-511. 10.1214/ss/1177011136.
Article</li></ol><p>Google Scholar</p><ol start=59><li>Huson DH, Richter DC, Rausch C, Dezulian T, Franz M, Rupp R: <strong>Dendroscope: An interactive viewer for large phylogenetic trees.</strong> <em>BMC Bioinformatics</em>. 2007, <strong>8</strong>: 460-10.1186/1471-2105-8-460.
PubMed Central</li></ol><p>Article</p><p>PubMed</p><p>Google Scholar</p><ol start=60><li><p>Rambaut A: <em>FigTree</em>. 2009, 131
Google Scholar</p></li><li><p>Hamodrakas SJ: <strong>Protein aggregation and amyloid fibril formation prediction software from primary sequence: towards controlling the formation of bacterial inclusion bodies.</strong> <em>FEBS J</em>. 2011, <strong>278</strong> (14): 2428-2435. 10.1111/j.1742-4658.2011.08164.x.
Article</p></li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=62><li>Hogl S, Kuhn PH, Colombo A, Lichtenthaler SF: <strong>Determination of the proteolytic cleavage sites of the amyloid precursor-like protein 2 by the proteases ADAM10, BACE1 and gamma-secretase.</strong> <em>PLoS One</em>. 2011, <strong>6</strong> (6): e21337-10.1371/journal.pone.0021337.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=63><li>Yanagida K, Okochi M, Tagami S, Nakayama T, Kodama TS, Nishitomi K, Jiang J, Mori K, Tatsumi S, Arai T, et al: <strong>The 28-amino acid form of an APLP1-derived Abeta-like peptide is a surrogate marker for Abeta42 production in the central nervous system.</strong> <em>EMBO Mol Med</em>. 2009, <strong>1</strong> (4): 223-235. 10.1002/emmm.200900026.
PubMed Central</li></ol><p>Article</p><p>CAS</p><p>PubMed</p><p>Google Scholar</p><ol start=64><li>Minogue AM, Stubbs AK, Frigerio CS, Boland B, Fadeeva JV, Tang J, Selkoe DJ, Walsh DM: <strong>Gamma-secretase processing of APLP1 leads to the production of a p3-like peptide that does not aggregate and is not toxic to neurons.</strong> <em>Brain Res</em>. 2009, <strong>1262</strong>: 89-99.
Article</li></ol><p>CAS</p><p>PubMed</p><p>Google Scholar</p><p>Download references</p><h2 id=acknowledgements>Acknowledgements
<a class=anchor href=#acknowledgements>#</a></h2><p>This work was supported in part by a grant to I.N.S. from the National Library of Medicine (R01 LM009725). W.G.T. is supported by an individual fellowship award from the National Institute of Diabetes and Digestive and Kidney Diseases (F30 DK084605).</p><h2 id=author-information>Author information
<a class=anchor href=#author-information>#</a></h2><h3 id=affiliations>Affiliations
<a class=anchor href=#affiliations>#</a></h3><ol><li>Center for Clinical and Translational Science, University of Vermont, Given Courtyard N309, 89 Beaumont Avenue, Burlington, VT, 05405, USA
William G Tharp & Indra Neil Sarkar</li><li>Division of Endocrinology, Department of Medicine, University of Vermont, Given Courtyard N309, 89 Beaumont Avenue, Burlington, VT, 05405, USA
William G Tharp</li><li>Department of Microbiology and Molecular Genetics, University of Vermont, Given Courtyard N309, 89 Beaumont Avenue, Burlington, VT, 05405, USA
Indra Neil Sarkar</li><li>Department of Computer Science, University of Vermont, Given Courtyard N309, 89 Beaumont Avenue, Burlington, VT, 05405, USA
Indra Neil Sarkar</li></ol><h3 id=corresponding-author>Corresponding author
<a class=anchor href=#corresponding-author>#</a></h3><p>Correspondence to Indra Neil Sarkar.</p><h2 id=additional-information>Additional information
<a class=anchor href=#additional-information>#</a></h2><h3 id=competing-interests>Competing interests
<a class=anchor href=#competing-interests>#</a></h3><p>INS and WGT do not have any conflicts of interest to disclose.</p><h3 id=authors-contributions>Authors’ contributions
<a class=anchor href=#authors-contributions>#</a></h3><p>INS and WGT conceived of and designed the study together. INS collected and aligned the sequences. WGT conducted the tree building and aggregation analyses. Both INS and WGT interpreted the results and drafted the manuscript together. Both authors read and approved the final manuscript.</p><h2 id=rights-and-permissions>Rights and permissions
<a class=anchor href=#rights-and-permissions>#</a></h2><p>This article is published under license to BioMed Central Ltd. This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href=http://creativecommons.org/licenses/by/2.0>http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p><h2 id=reprints-and-permissions>Reprints and Permissions
<a class=anchor href=#reprints-and-permissions>#</a></h2><h2 id=about-this-article>About this article
<a class=anchor href=#about-this-article>#</a></h2><h3 id=cite-this-article>Cite this article
<a class=anchor href=#cite-this-article>#</a></h3><p>Tharp, W.G., Sarkar, I.N. Origins of amyloid-β. BMC Genomics 14, 290 (2013). <a href=https://doi.org/10.1186/1471-2164-14-290>https://doi.org/10.1186/1471-2164-14-290</a></p><p>Download citation</p><p>Received
16 November 2012</p><p>Accepted
24 April 2013</p><p>Published
30 April 2013</p><p>DOI
<a href=https://doi.org/10.1186/1471-2164-14-290>https://doi.org/10.1186/1471-2164-14-290</a></p><h3 id=share-this-article>Share this article
<a class=anchor href=#share-this-article>#</a></h3><p>Anyone you share the following link with will be able to read this content:</p><h3 id=get-shareable-link>Get shareable link
<a class=anchor href=#get-shareable-link>#</a></h3><p>Provided by the Springer Nature SharedIt content-sharing initiative</p><h2 id=keywords>Keywords
<a class=anchor href=#keywords>#</a></h2><p>Amyloid
Alzheimer disease
Phylogenetics
In silico
Aggregation
Maximum parsimony
Bayesian inference
Download PDF
Section</p><p>Comparative and evolutionary genomics
Sections</p><h2 id=figures>Figures
<a class=anchor href=#figures>#</a></h2><p>References
Figure 1
figure1
View in article
Full size image
Figure 2
figure2
View in article
Full size image
Figure 3
figure3
View in article
Full size image
Figure 4
figure4
View in article
Full size image
Figure 5
figure5
View in article
Full size image
Figure 6
figure6
View in article
Full size image
Figure 7
figure7
View in article
Full size image
Figure 8
figure8
View in article
Full size image
BMC Genomics
ISSN: 1471-2164
Contact us</p><h2 id=submission-enquiries>Submission enquiries:
<a class=anchor href=#submission-enquiries>#</a></h2><p><a href=mailto:bmcgenomics@biomedcentral.com>bmcgenomics@biomedcentral.com</a></p><h2 id=general-enquiries>General enquiries:
<a class=anchor href=#general-enquiries>#</a></h2><p><a href=mailto:ORSupport@springernature.com>ORSupport@springernature.com</a></p><h2 id=submit-your-next-manuscript-to-biomed-central-and-take-full-advantage-of>Submit your next manuscript to BioMed Central and take full advantage of:
<a class=anchor href=#submit-your-next-manuscript-to-biomed-central-and-take-full-advantage-of>#</a></h2><ul><li>• Convenient online submission</li><li>• Thorough peer review</li><li>• No space constraints or color figure charges</li><li>• Immediate publication on acceptance</li><li>• Inclusion in PubMed, CAS, Scopus and Google Scholar</li><li>• Research which is freely available for redistribution</li></ul><p>Submit your manuscript at <a href=https://www.biomedcentral.com/submit>www.biomedcentral.com/submit</a></p><p>Read more on our blogs
Receive BMC newsletters
Manage article alerts
Language editing for authors
Scientific editing for authors
Policies
Accessibility
Press center
Support and Contact
Leave feedback
Careers
Follow BMC</p><p>BMC Twitter page
BMC Facebook page
BMC Weibo page
By using this website, you agree to our Terms and Conditions, California Privacy Statement, Privacy statement and Cookies policy. Manage cookies/Do not sell my data we use in the preference centre.</p><p>Springer Nature
© 2021 BioMed Central Ltd unless otherwise stated. Part of Springer Nature.</p><div class=footnotes role=doc-endnotes><hr><ol><li id=fn:1><p>Center for Clinical and Translational Science, University of Vermont, Given Courtyard N309, 89 Beaumont Avenue, Burlington, VT 05405, USA&#160;<a href=#fnref:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref1:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:2><p>Division of Endocrinology, Department of Medicine, University of Vermont, Given Courtyard N309, 89 Beaumont Avenue, Burlington, VT 05405, USA.&#160;<a href=#fnref:2 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:3><p>Department of Microbiology and Molecular Genetics, University of Vermont, Given Courtyard N309, 89 Beaumont Avenue, Burlington, VT 05405, USA&#160;<a href=#fnref:3 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:4><p>Department of Computer Science, University of Vermont, Given Courtyard N309, 89 Beaumont Avenue, Burlington, VT 05405, USA.&#160;<a href=#fnref:4 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li></ol></div></article><footer class=book-footer><div class="flex flex-wrap justify-between"><div><a class="flex align-center" href=https://github.com/alex-shpak/hugo-book/edit/main/exampleSite/content/docs/amyloid-%ce%b2.md target=_blank rel=noopener><img src=/svg/edit.svg class=book-icon alt=Edit>
<span>Edit this page</span></a></div></div><script>(function(){function e(e){const t=window.getSelection(),n=document.createRange();n.selectNodeContents(e),t.removeAllRanges(),t.addRange(n)}document.querySelectorAll("pre code").forEach(t=>{t.addEventListener("click",function(){e(t.parentElement),navigator.clipboard&&navigator.clipboard.writeText(t.parentElement.textContent)})})})()</script></footer><div class=book-comments></div><label for=menu-control class="hidden book-menu-overlay"></label></div><aside class=book-toc><div class=book-toc-content><nav id=TableOfContents><ul><li><a href=#abstract>Abstract</a></li><li><a href=#background>Background</a></li><li><a href=#results>Results</a><ul><li><a href=#phylogenetics-of-aβpp-gene-family>Phylogenetics of AβPP gene family</a></li><li><a href=#persistence-of-amyloid-β>Persistence of amyloid-β</a></li><li><a href=#evolutionary-relationships-of-amyloid-β-formation-potential>Evolutionary relationships of amyloid-β formation potential</a></li></ul></li><li><a href=#discussion>Discussion</a></li><li><a href=#methods>Methods</a><ul><li><a href=#dataset-collection-and-alignment>Dataset collection and alignment</a></li><li><a href=#generation-of-phylogenetic-trees>Generation of phylogenetic trees</a></li><li><a href=#synapomorphic-frequencies>Synapomorphic frequencies</a></li><li><a href=#aggregation-modeling>Aggregation modeling</a></li></ul></li><li><a href=#electronic-supplementary-material>Electronic supplementary material</a><ul><li><a href=#additional-file-figure-s1>Additional file: Figure S1.</a></li><li><a href=#table-s1-taxa-species-names-and-sequence-accession-numbers-pdf-1-mb>Table S1. Taxa Species Names and Sequence Accession Numbers. (PDF 1 MB)</a></li></ul></li><li><a href=#abbreviations>Abbreviations</a></li><li><a href=#references>References</a></li><li><a href=#acknowledgements>Acknowledgements</a></li><li><a href=#author-information>Author information</a><ul><li><a href=#affiliations>Affiliations</a></li><li><a href=#corresponding-author>Corresponding author</a></li></ul></li><li><a href=#additional-information>Additional information</a><ul><li><a href=#competing-interests>Competing interests</a></li><li><a href=#authors-contributions>Authors’ contributions</a></li></ul></li><li><a href=#rights-and-permissions>Rights and permissions</a></li><li><a href=#reprints-and-permissions>Reprints and Permissions</a></li><li><a href=#about-this-article>About this article</a><ul><li><a href=#cite-this-article>Cite this article</a></li><li><a href=#share-this-article>Share this article</a></li><li><a href=#get-shareable-link>Get shareable link</a></li></ul></li><li><a href=#keywords>Keywords</a></li><li><a href=#figures>Figures</a></li><li><a href=#submission-enquiries>Submission enquiries:</a></li><li><a href=#general-enquiries>General enquiries:</a></li><li><a href=#submit-your-next-manuscript-to-biomed-central-and-take-full-advantage-of>Submit your next manuscript to BioMed Central and take full advantage of:</a></li></ul></nav></div></aside></main></body></html>