<!doctype html><html lang=en dir=ltr><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="Skip to Article Content Skip to Article Information Working off-campus? Learn about our remote access options By continuing to browse this site, you agree to its use of cookies as described in our Cookie Policy. × Wiley Online Library Login / Register Ecology and Evolution Ecology and Evolution ORIGINAL RESEARCH Open Access “Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds"><meta name=theme-color content="#FFFFFF"><meta name=color-scheme content="light dark"><meta property="og:title" content="“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds"><meta property="og:description" content="Skip to Article Content Skip to Article Information Working off-campus? Learn about our remote access options By continuing to browse this site, you agree to its use of cookies as described in our Cookie Policy. × Wiley Online Library Login / Register Ecology and Evolution Ecology and Evolution ORIGINAL RESEARCH Open Access “Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds"><meta property="og:type" content="article"><meta property="og:url" content="https://csmeyns.github.io/derivatives/docs/in-out-qinghai-tibet-plateau/"><meta property="article:section" content="docs"><title>“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds | Derivatives</title><link rel=manifest href=/derivatives/manifest.json><link rel=icon href=/derivatives/favicon.png type=image/x-icon><link rel=stylesheet href=/derivatives/book.min.1a6fac2df464e68d664cb846d822c678cd622692d6fae24ee18901a08bfc8eb6.css integrity="sha256-Gm+sLfRk5o1mTLhG2CLGeM1iJpLW+uJO4YkBoIv8jrY=" crossorigin=anonymous><script defer src=/derivatives/flexsearch.min.js></script>
<script defer src=/derivatives/en.search.min.3c43ca93690cd696d222c3f1765342e2fbc2edce19f694aea66fb7ee4cb6c088.js integrity="sha256-PEPKk2kM1pbSIsPxdlNC4vvC7c4Z9pSupm+37ky2wIg=" crossorigin=anonymous></script></head><body dir=ltr><input type=checkbox class="hidden toggle" id=menu-control>
<input type=checkbox class="hidden toggle" id=toc-control><main class="container flex"><aside class=book-menu><div class=book-menu-content><nav><h2 class=book-brand><a class="flex align-center" href=/derivatives/><span>Derivatives</span></a></h2><div class=book-search><input type=text id=book-search-input placeholder=Search aria-label=Search maxlength=64 data-hotkeys=s/><div class="book-search-spinner hidden"></div><ul id=book-search-results></ul></div><ul><li><a href=https://csmeyns.github.io/derivatives/docs/readme/>README</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/architecture-human-orc/>Architecture of the Human Origin Recognition Complex*</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/amyloid-%CE%B2/>Origins of amyloid-β</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/arsenic/>Effects of arsenic and heavy metals on metabolic pathways in cells of human origin: Similarities and differences</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/back-to-future/>Back to the future: origins and directions of the “Agile Manifesto” – views of the originators</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/biopolymers/>Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers</a></li><li><input type=checkbox id=section-4e626a39f813bdfb0b0f11621a8d97e6 class=toggle>
<label for=section-4e626a39f813bdfb0b0f11621a8d97e6 class="flex justify-between"><a href=https://csmeyns.github.io/derivatives/docs/cc-by/>CC-BY</a></label><ul><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-2-summary/>Creative Commons License Deed Attribution 2.0 Generic (CC BY 2.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-2/>Creative Commons Legal Code Attribution 2.0</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-3-summary/>Creative Commons License Deed Attribution 3.0 Unported (CC BY 3.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-3/>Creative Commons Attribution 3.0 Unported</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-4-summary/>Creative Commons License Deed Attribution 4.0 International (CC BY 4.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-4-de/>Creative Commons Konzessionsurkunde Namensnennung 4.0 International (CC BY 4.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-4/>Creative Commons Legal Code Attribution 4.0 International</a></li></ul></li><li><a href=https://csmeyns.github.io/derivatives/docs/charon-smooth-plains/>The nature and origin of Charon's smooth plains</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/chinese-word-cell/>Origin of the Chinese word for “cell”: an unusual but wonderful idea of a mathematician</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/city-origins/>City origins</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/counterfeit-versus-original/>Counterfeit versus original patronage: Do emotional brand attachment, brand involvement, and past experience matter?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/creatine/>ON THE ORIGIN OF CREATINE</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/detecting-moisture/>Detecting the origins of moisture over Southeast China: Seasonal variation and heavy Rainfall</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/differentiation-spirits/>Differentiation Between Spirits According to Their Botanical Origin</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/food-taboos/>Food taboos: their origins and purposes</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/ganges/>Ganges: special at its origin</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/given-that-detailed/>Given that the detailed original criteria for deliberate practice have not changed, could the understanding of this complex concept have improved over time? A response to Macnamara and Hambrick (2020)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/hanwoo-cattle/>Hanwoo cattle: origin, domestication, breeding strategies and genomic selection</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/hoarding/>The developmental origins of hoarding disorder in adolescence: a longitudinal clinical interview study following an epidemiological survey</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/how-to-write/>How to write an original article for the *Journal of Orthopaedics and Traumatology*</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/in-out-qinghai-tibet-plateau/ class=active>“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/interview-patrick-o-brown/>An interview with Patrick O Brown on the origins and future of open access</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/manipulated-photos/>Can people identify original and manipulated photos of real-world scenes?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/neighborhood-spam/>Neighborhoods and bands: an analysis of the origins of spam</a></li><li><input type=checkbox id=section-18bd70f4fad61eb2742d114842aa84cf class=toggle>
<label for=section-18bd70f4fad61eb2742d114842aa84cf class="flex justify-between"><a href=https://csmeyns.github.io/derivatives/docs/letters/>Letters</a></label><ul><li><a href=https://csmeyns.github.io/derivatives/docs/letters/non-neanderthal/>Non-Neanderthal Origin of the HLA-DPB1*0401</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/letters/reply-to-ding/>Reply to Ding et al.: Non-Neanderthal Origin of the HLA-DPB1*0401</a></li></ul></li><li><a href=https://csmeyns.github.io/derivatives/docs/origin-of-the-ectopic-beat/>What is the origin of the ectopic beat?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/origins/>Origins</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/patient-safety/>The 5th anniversary of “Patient Safety in Surgery” – from the Journal’s origin to its future vision</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/rendang/>Tracing the origins of rendang and its development</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/soft-robot/>Assessing the Impact of an Original Soft Communicative Robot in a Nursing Home in Japan: Will Softness or Conversations Bring more Smiles to Older People?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/softwaretest-originaldaten/>Softwaretest mit Originaldaten Eine Analyse aus Sicht des Datenschutzes</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/spectacular-submarine/>Origin of spectacular fields of submarine sediment waves around volcanic islands</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/urban-conservation/>Urban Conservation Through Chinese Local Original Architecture: The Preservation and Renovation of Zhongshan Road in Hangzhou</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/vertebrate-gills/>The Origin of Vertebrate Gills</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/volcanic-origin-1741/>Volcanic origin of the 1741 Oshima-Oshima tsunami in the Japan Sea</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/what-is-morita-therapy/>What is Morita Therapy? The Nature, Origins, and Cross-Cultural Application of a Unique Japanese Psychotherapy</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/wildcat-reappearance/>Revealing the origin of wildcat reappearance after presumed long-term absence</a></li></ul></nav><script>(function(){var e=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(){localStorage.setItem("menu.scrollTop",e.scrollTop)}),e.scrollTop=localStorage.getItem("menu.scrollTop")})()</script></div></aside><div class=book-page><header class=book-header><div class="flex align-center justify-between"><label for=menu-control><img src=/derivatives/svg/menu.svg class=book-icon alt=Menu></label>
<strong>“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds</strong>
<label for=toc-control><img src=/derivatives/svg/toc.svg class=book-icon alt="Table of Contents"></label></div><aside class="hidden clearfix"><nav id=TableOfContents><ul><li><a href=#correspondence>Correspondence</a></li><li><a href=#funding-information>Funding information</a></li><li><a href=#abstract>Abstract</a></li><li><a href=#keywords>KEYWORDS</a></li><li><a href=#1-introduction>1 INTRODUCTION</a></li><li><a href=#2-materials-and-methods>2 MATERIALS AND METHODS</a><ul><li><a href=#21-sampling-strategy>2.1 Sampling strategy</a></li><li><a href=#22-dna-extraction-pcr-and-sequencing>2.2 DNA extraction, PCR, and sequencing</a></li><li><a href=#23-data-assembly>2.3 Data assembly</a></li><li><a href=#24-phylogenetic-reconstruction-and-divergence-time-estimation>2.4 Phylogenetic reconstruction and divergence-time estimation</a></li><li><a href=#25-species-distributions>2.5 Species distributions</a></li><li><a href=#26-ancestral-range-reconstruction-arr>2.6 Ancestral-range reconstruction (ARR)</a></li></ul></li><li><a href=#3-results>3 RESULTS</a><ul><li><a href=#31-phylogeny-and-origin-of-passeroidea>3.1 Phylogeny and origin of Passeroidea</a></li><li><a href=#32-biogeographical-history-of-montane-and-alpine-target-groups-of-passeroidea>3.2 Biogeographical history of montane and alpine target groups of Passeroidea</a></li></ul></li><li><a href=#4-discussion>4 DISCUSSION</a><ul><li><a href=#41-phylogeny-and-biogeographic-history-of-passeroidea>4.1 Phylogeny and biogeographic history of Passeroidea</a></li><li><a href=#42-out-of-qtpin-situ-diversification-on-the-qtp-and-along-its-forested-fringes>4.2 “Out of QTP”—in situ diversification on the QTP and along its forested fringes</a></li><li><a href=#43-faunal-exchange-between-the-plateau-region-and-the-qtp-fringes>4.3 Faunal exchange between the plateau region and the QTP fringes</a></li><li><a href=#44-immigration-into-the-qtp-region-from-adjacent-source-regions>4.4 Immigration into the QTP region from adjacent source regions</a></li></ul></li><li><a href=#5-conclusion>5 CONCLUSION</a></li><li><a href=#acknowledgments>ACKNOWLEDGMENTS</a></li><li><a href=#conflict-of-interest>CONFLICT OF INTEREST</a></li><li><a href=#author-contributions>AUTHOR CONTRIBUTIONS</a></li><li><a href=#data-availability-statement>DATA AVAILABILITY STATEMENT</a></li><li><a href=#orcid>ORCID</a></li><li><a href=#references>REFERENCES</a></li><li><a href=#supporting-information>SUPPORTING INFORMATION</a><ul><li><a href=#citing-literature>Citing Literature</a></li><li><a href=#recommended>Recommended</a></li><li><a href=#contact-us>Contact Us</a></li><li><a href=#subscription-agents>Subscription Agents</a></li><li><a href=#the-wiley-network>The Wiley Network</a></li></ul></li></ul></nav></aside></header><article class=markdown><h1><a href=/derivatives/docs/in-out-qinghai-tibet-plateau/>“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds</a></h1><h2></h2><p>Skip to Article Content
Skip to Article Information
Working off-campus? Learn about our remote access options
By continuing to browse this site, you agree to its use of cookies as described in our Cookie Policy.
×
Wiley Online Library
Login / Register
Ecology and Evolution
Ecology and Evolution
ORIGINAL RESEARCH
Open Access
“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds</p><p>Received: 16 January 2020 | Revised: 12 June 2020 | Accepted: 29 June 2020 DOI: 10.1002/ece3.6615
ORIGINAL RESEARCH</p><p>Martin Päckert<sup id=fnref:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup>, Adrien Favre<sup id=fnref:2><a href=#fn:2 class=footnote-ref role=doc-noteref>2</a></sup>, Jan Schnitzler<sup id=fnref:3><a href=#fn:3 class=footnote-ref role=doc-noteref>3</a></sup><sup id=fnref:4><a href=#fn:4 class=footnote-ref role=doc-noteref>4</a></sup>, Jochen Martens<sup id=fnref:5><a href=#fn:5 class=footnote-ref role=doc-noteref>5</a></sup>, Yue-Hua Sun<sup id=fnref:6><a href=#fn:6 class=footnote-ref role=doc-noteref>6</a></sup>, Dieter Thomas Tietze<sup id=fnref:7><a href=#fn:7 class=footnote-ref role=doc-noteref>7</a></sup><sup id=fnref:8><a href=#fn:8 class=footnote-ref role=doc-noteref>8</a></sup>, Frank Hailer<sup id=fnref:9><a href=#fn:9 class=footnote-ref role=doc-noteref>9</a></sup><sup id=fnref:10><a href=#fn:10 class=footnote-ref role=doc-noteref>10</a></sup>, Ingo Michalak<sup id=fnref1:4><a href=#fn:4 class=footnote-ref role=doc-noteref>4</a></sup>, Patrick Strutzenberger<sup id=fnref1:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup><sup id=fnref:11><a href=#fn:11 class=footnote-ref role=doc-noteref>11</a></sup>
First published: 04 August 2020 <a href=https://doi.org/10.1002/ece3.6615Citations>https://doi.org/10.1002/ece3.6615Citations</a>: 7
SECTIONSPDFPDFTOOLS SHARE</p><h2 id=correspondence>Correspondence
<a class=anchor href=#correspondence>#</a></h2><p>Martin Päckert, Senckenberg Natural History Collections, Museum of Zoology, Königsbrücker Landstraße 159, 01109 Dresden, Germany.
Email: <a href=mailto:martin.paeckert@senckenberg.de>martin.paeckert@senckenberg.de</a></p><h2 id=funding-information>Funding information
<a class=anchor href=#funding-information>#</a></h2><p>Deutsche Forschungsgemeinschaft, Grant/ Award Number: PA1818/3-1, MU2934/2-1, FA1117/1-2 and FZT118</p><div class=abstract><h2 id=abstract>Abstract
<a class=anchor href=#abstract>#</a></h2><p>Encompassing some of the major hotspots of biodiversity on Earth, large mountain systems have long held the attention of evolutionary biologists. The region of the Qinghai-Tibet Plateau (QTP) is considered a biogeographic source for multiple colonization events into adjacent areas including the northern Palearctic. The faunal exchange between the QTP and adjacent regions could thus represent a one-way street (“out of” the QTP). However, immigration into the QTP region has so far received only little attention, despite its potential to shape faunal and floral communities of the QTP. In this study, we investigated centers of origin and dispersal routes between the QTP, its forested margins and adjacent regions for five clades of alpine and montane birds of the passerine superfamily Passeroidea. We performed an ancestral area reconstruction using BioGeoBEARS and inferred a time-calibrated backbone phylogeny for 279 taxa of Passeroidea. The oldest endemic species of the QTP was dated to the early Miocene (ca. 20 Ma). Several additional QTP endemics evolved in the mid to late Miocene (12–7 Ma). The inferred centers of origin and diversification for some of our target clades matched the “out of Tibet hypothesis’ or the “out of Himalayas hypothesis” for others they matched the “into Tibet hypothesis.” Three radiations included multiple independent Pleistocene colonization events to regions as distant as the Western Palearctic and the Nearctic. We conclude that faunal exchange between the QTP and adjacent regions was bidirectional through time, and the QTP region has thus harbored both centers of diversification and centers of immigration.</p></div><p>This is an open access article under the terms of the Creative Commons Attribution License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.
© 2020 The Authors. <em>Ecology and Evolution</em> published by John Wiley & Sons Ltd
Ecology and Evolution. 2020;10:9283–9300. <a href=https://www.ecolevol.org>www.ecolevol.org</a> | 9283</p><h2 id=keywords>KEYWORDS
<a class=anchor href=#keywords>#</a></h2><p>ancestral ranges, center of origin, immigration, in situ diversification, Qinghai-Tibet Plateau, Sinohimalayas</p><h2 id=1-introduction>1 INTRODUCTION
<a class=anchor href=#1-introduction>#</a></h2><p>Many of the World&rsquo;s biodiversity hotspots are located in large mountain systems, such as the Andes, the East African Arc or the Himalayas (Marchese, 2015) and the role of mountains in organismic evolutionary diversification is considered to be manifold (Fjeldså, Bowie, & Rahbeck, 2012; Hoorn, Perrigo, & Antonelli, 2018; Muellner-Riehl, 2019; Rahbek, Borregaard, Antonelli, et al., 2019; Rahbek, Borregaard, Colwell, et al., 2019). On the Eurasian continent, the Qinghai-Tibet Plateau (QTP) and its flanking mountain systems constitute the largest and probably most diverse area of montane species richness (Aliabadian, Sluys, Roselaar, & Nijman, 2008; Fjeldså et al., 2012). For example, passerine bird diversity follows a gradient from warm and humid forest ecosystems harboring 358 species in the Eastern Himalayas (Price et al., 2014) and 441 in northern Myanmar (Renner et al., 2015) toward less diverse avian communities in the colder and drier Western Himalayas (Price et al., 2011). Alpine avian communities on the QTP are generally less species rich than those of the Himalayan forest ecosystems (Fjeldså et al., 2012), however, they harbor a couple of wide-range and narrow-range endemics as well as widespread trans-Palearctic species (Figure 1).</p><p>image
Figure 1
Open in figure viewer
PowerPoint</p><p><img src=/img/in-out-qinghai-tibet-plateau_fig1.jpg alt="Passerine bird species of the Qinghai-Tibet Plateau (QTP)"></p><figcaption><p><strong>Figure 1</strong> Passerine bird species of the Qinghai-Tibet Plateau (QTP); QTP endemics, wide distribution range on the plateau: (a) streaked rosefinch, <em>Carpodacus rubicilloides</em>; (b) robin accentor, <em>Prunella rubeculoides</em>; (c) rock sparrow, <em>Petronia petronia</em>, widespread trans-Palearctic distribution; (d) Tibetan bunting, <em>Emberiza koslowi</em>, narrow-range endemic of the QTP; all photos: M.P., Qinghai, China, June 2013</p></figcaption><p>The three major hotspots of diversity at the QTP fringes are the Mountains of Central Asia in the West, the Himalayas in the South, and the Mountains of Southwest China (i.e., the Hengduanshan and its northward extensions) in the East (Figure 2a; Favre et al., 2015). As the largest and highest plateau on Earth, the QTP extends across a surface area of about 2.3 million km2, comprising the largest continuous area of montane grasslands of the World (Olson et al., 2001). The alpine environments of the QTP and its flanking mountain systems have been declared as the “Third Pole” (Yao et al., 2012). As such, the QTP recently became famous as the source area of cold-adapted organisms of the Eurasian fauna (e.g., the Pleistocene mammal fauna: Wang et al., 2015) or the Holarctic alpine flora (Favre et al., 2016). However, the “out of Tibet hypothesis” for the montane and alpine fauna and flora of Eurasia is not a particularly new concept, dating back to the work of the German ornithologist Hugo Weigold in the early 20th century. He was the first to postulate a Tibetan center of diversification (“Entwicklungszentrum”) for Palearctic terrestrial vertebrates, and earliest emergences of Tibetan faunal elements already during the Early Tertiary (Weigold, 1935, 1949, 2005). Weigold&rsquo;s ideas fell into oblivion, until the out of Tibet hypothesis was reanimated in the discussion on the Tibetan origin of Palearctic cold-adapted mammals inferred from fossil evidence (Deng et al., 2011; Tseng, Li, & Wang, 2013; Tseng et al., 2014; Wang, Li, & Takeuchi, 2016; Wang, Tseng, Li, Takeuchi, & Xie, 2014). Some of these studies suggested a “Himalayan origin” (Wang et al., 2014) or “Central Asian origin” (Tseng et al., 2013) of their study organisms, but all these examples were later reviewed in the context of an “out of Tibet” colonization of the adjacent regions in the North (e.g., Qiu, 2014; Wang et al., 2015). Besides this terminological inaccuracy, the emerging view from analysis of the QTP fossil record was therefore a “one-way view” of faunal interchange between the QTP (including its margins; Figure 2a) and adjacent bioregions, excluding any reverse movement into Tibet (Tseng et al., 2014; Wang et al., 2014, 2016).</p><p>image
Figure 2
Open in figure viewer
PowerPoint</p><p><img src=/img/in-out-qinghai-tibet-plateau_fig2.jpg alt="(a) Target region, the Qinghai-Tibet Plateau (area 1) &amp;hellip; (b) phylogeny for 279 species of Passeroidea"></p><figcaption><p><strong>Figure 2</strong> (a) Target region, the Qinghai-Tibet Plateau (area 1) and the biodiversity hotspots along its forested margins (areas 2–5), modified from Favre et al. (2015); (b) phylogeny for 279 species of Passeroidea, five target clades representing five independent radiations involving QTP species marked in bold</p></figcaption><p>Recent phylogenetic studies have suggested or presumed a one-directional “out of Tibet” or a “out of Himalayas” dispersal pathway, too, but these were typically focusing on single species or a single, species-poor clade (e.g., Fuentes-Hurtado, Hof, & Jansson, 2016; Liu et al., 2017; Voelker, Semenov, Fadeev, Blick, & Drovetski, 2015; and other examples in Table S1). However, work on a larger taxonomic group (e.g., speciose alpine taxa like <em>Saxifraga</em>, Ebersbach et al., 2017) suggested that both in situ speciation and immigration played a key role in the evolution of alpine faunal and floral assemblages of the QTP (e.g., Hauenschild et al., 2017) or of the Himalayas (Johansson et al., 2007; compare Table S1).</p><p>In this study, we compare patterns of biogeographic history among five subclades of Passeroidea, a superfamily of passerine birds (Johansson, Fjeldså, & Bowie, 2008; Selvatti, Gonzaga, & de Morales Russo, 2015). These subclades represent five independent passerine radiations (Figure 2b), representing ideal model groups for the study of biogeographic history of QTP faunal assemblages: the studied species are montane and alpine birds distributed across all mountain systems of the Holarctic (Figure 3) and include 29 endemics to the QTP region. For these five passerine clades, biogeographic analyses and ancestral area reconstructions are either missing, incomplete, or provide contradictory results (Drovetski et al., 2013; Liu et al., 2017; Tietze, Päckert, Martens, Lehmann, & Sun, 2013).</p><p>image
Figure 3
Open in figure viewer
PowerPoint</p><p><img src=/img/in-out-qinghai-tibet-plateau_fig3.jpg alt="Spatial distribution of species richness and diversity hotspots for all five target clades of Passeroidea"></p><figcaption><p><strong>Figure 3</strong> Spatial distribution of species richness and diversity hotspots for all five target clades of Passeroidea; diversity heat maps were compiled with QGIS using shape files inferred from BirdLife International and NatureServe (2015) and from the IUCN Red List (2019); colors indicate regional species richness (high = red; low = dark blue)</p></figcaption><p>Our five studied passerine clades encompass characteristic faunal elements of the QTP region and include several lineages considered as flagship species by Weigold (2005) for the high alpine grasslands and rocky hillslopes of the QTP: (a) Old World sparrows (<em>Passeridae</em>) comprising snowfinches (<em>Montifringilla</em> and allies; two genera <em>Onychostruthus</em> and <em>Pyrgilauda</em> are endemic to this region). The area of high Passeridae species richness at the eastern margin of the QTP is certainly due to the high snowfinch diversity in this region (Figure 3a; a second hotspot in East Africa is characterized by a high diversity of <em>Passer</em> species), (b) the mountain finches (genus Leucosticte), with three species in the QTP region and further three in the Nearctic. This genus along with bullfinches and other allies belong to tribe Pyrrhulini (clade 6 in Zuccon, Prŷs-Jones, Rasmussen, & Ericson, 2012). With another group, (c) the rosefinches (Carpodacini: clade 5 in Zuccon et al., 2012), both tribes of finches (Fringillidae) have areas of highest species richness in the Sinohimalayas (Pyrrhulini also north of the QTP; Figure 3b,c). Weigold (2005) hypothesized that all extant rosefinches originated from a Tibetan ancestor but later diversified in peripheral refuges at the southern and the eastern QTP margin when forest habitats vanished from the central plateau region. Furthermore, we investigate (d) Old World buntings (Emberzidae). Although the hotspot of highest bunting species richness is located in the Eastern Palearctic, a few taxa form a secondary hotspot of species richness at the eastern QTP margin (Figure 3d). According to Weigold (2005) the endemic Tibetan bunting, <em>Emberiza koslowi</em> (Figure 1d) represents an “autochtonous faunal element” of the QTP (i.e., he assumed that this species originated in situ on the QTP), whereas the crested bunting (<em>Emberiza lathami</em>) had originated from a tropical ancestor southwest of the QTP. Finally, (e) accentors, in the Palearctic family Prunellidae, are most speciose at the northeastern and southeastern QTP margin (Figure 3e).</p><p>Our five target clades encompassed 132 Holarctic species, including 29 endemics of the QTP and its forested margins (compare “Material and Methods,” “Species distributions”). Assuming an in situ origin of all QTP endemics that was associated with the supposedly recent formation of the plateau region (Liu, Gao, Chen, & Lu, 2002 and references therein), recent orogenetic processes should have promoted the rise of species-level lineages endemic to the QTP from the Middle Miocene on (Shang et al., 2015). However, such a tight coupling of young evolutionary events (species splits, adaptive radiations in the QTP region) with orogenetic post-Miocene events (“young Tibet hypothesis”) has recently been critically challenged by Renner (2016). Indeed, there is evidence of ancient QTP species that evolved prior to the Miocene (Baker, Pereira, & Paton, 2007; Päckert, Sun, et al., 2015). Furthermore, some ancient Tibetan and Himalayan faunal elements already emerged during the Eocene and later diversified in situ on the QTP over a long period of time (Martens, 2015; Mosbrugger, Favre, Muellner-Riehl, Päckert, & Mulch, 2018 and references therein). These findings are in accordance with Weigold’s (2005) theory of an early Tertiary origin of the endemic fauna of Tibet, though he was originally driven by the idea that diversification of the Tibetan fauna was firmly associated with the uplift of the QTP.</p><p>In accordance with Weigold&rsquo;s theories and with more complex biogeographic patterns derived from large phylogenies and comparison across clades, we searched for evidence of ancestral areas of origin for our target groups on the QTP itself and along its margins (Figure 2a), and also in adjacent regions beyond the QTP region. We therefore provide one of very few case studies that compare patterns of biogeographical history across different clades from a speciose monophyletic group of organisms.</p><h2 id=2-materials-and-methods>2 MATERIALS AND METHODS
<a class=anchor href=#2-materials-and-methods>#</a></h2><h3 id=21-sampling-strategy>2.1 Sampling strategy
<a class=anchor href=#21-sampling-strategy>#</a></h3><p>Biogeographic inferences for our five target clades needed to be anchored within a broader biogeographic scenario, including a broad selection of closely related as well as distant clades. We therefore assembled a DNA sequence dataset for Passeroidea, a highly speciose crown clade of Passeriformes that comprises 25 avian families (see Figure S1; Table S2). For all our target clades, our taxon sampling was 100% or near 100% (missing only <em>Montifringilla theresae</em> from Central Asia and <em>Carpodacus sillemi</em> from the QTP). We furthermore included a few subspecific taxa that are currently not accepted as “good species” but nevertheless represent distinct resident lineages (Table S2). These subspecies-level taxa represented vicariants occurring in different regions of a species’ range (in Eurasia our areas B–H) and in some species, distinct subspecific lineages correspond to distinct phenotypes, like in the common bullfinch (<em>Pyrrhula pyrrhula</em>: Töpfer et al., 2010). Divergence times between all these subspecific taxa equal or even exceed the minimum divergence times estimated for a good species pair of the target clade they are nested in. These species pairs are as follows: (a) for snowfinches <em>Pyrgilauda davidiana/P. blanfordi</em> (Delgado, Bettega, Martens, & Päckert, 2019); (b) for <em>Carpodacini Carpodacus rubicilloides</em> (Figure 1a)/<em>C. rubicilla and C. thura/C. dubius</em> (Tietze et al., 2013); (c) for Pyrrhulini <em>Leucosticte tephrocotis/L. australis/L. atrata</em> (Drovetski, Zink, & Mode, 2009); (d) for buntings: <em>Emberiza hortulana/E. caesia and E. citrinella/E. leucocephalos</em> (Päckert, Sun, et al., 2015); (e) Prunellidae: <em>Prunella ocularis/P. fagani</em> (Liu et al., 2017).</p><h3 id=22-dna-extraction-pcr-and-sequencing>2.2 DNA extraction, PCR, and sequencing
<a class=anchor href=#22-dna-extraction-pcr-and-sequencing>#</a></h3><p>To complete taxon sampling and coverage of loci for our target clades, we used samples and DNA extracts available from previous analyses (Päckert, Sun, et al., 2015; Päckert, Martens, Sun, & Strutzenberger, 2016; Tietze et al., 2013; Töpfer et al., 2011; for newly generated sequences see Table S2). We extracted DNA from new samples using Qiagen blood and tissue kits according to the manufacturer&rsquo;s instructions. Our sequence data set included four loci: the mitochondrial cytochrome-<em>b</em> (cyt<em>b</em>) and NADH dehydrogenase subunit 2 (ND2), as well as the nuclear introns ornithine decarboxylase (ODC) intron 7, and myoglobin (myo) intron 2. Overall, we used general and previously published primers (for PCR and sequencing primer combinations and PCR settings see Table S3). However, we utilized taxon-specific primer pairs for PCR and sequencing of ND2 for Passeridae by Belkacem et al. (2016) and of ODC for rosefinch and bunting samples (Table S3). We purified PCR products using ExoSap-IT (GE Healthcare; adding 0.1 ml ExoSap-IT solution in 4 ml H2O to each sample; PCR settings: 37°C for 30 min, 94°C for 15 min) and sequenced the purified PCR products, which was performed with BigDyeTM 3.1 Dye Terminator Cycle Sequencing Kits (Applied Biosystems), according to the manufacturers’ instructions. Sequencing products were purified by salt/ethanol precipitation or by using Sephadex (GE Healthcare), and sequenced in both directions on an ABI 3130xl DNA sequencer.</p><h3 id=23-data-assembly>2.3 Data assembly
<a class=anchor href=#23-data-assembly>#</a></h3><p>Sequences for core target taxa were compiled from previous studies (<em>n</em> = 101 species analyzed in Päckert et al., 2016; further taxa added from Tietze et al., 2013 and Päckert, Sun, et al., 2015). We added missing species and filled gaps in sequence coverage with newly generated sequences. To have all families of Passeroidea represented by at least one species, we included further sequences from GenBank. This is in accordance with Cai et al. (2018) who emphasized that estimates of ancestral range for ingroups (i.e., our target clades) are strongly affected by the ranges of outgroups (i.e., sister clades of our target clades). We added two passerine outgroup species: <em>Bombycilla garrulus</em> (Bombycilloidea as the sister clade of Passerida) and <em>Acanthisitta chloris</em> (Acanthisittidae as the earliest offshoot of the Passeriformes) and three further nonpasserine outgroup taxa (Table S2). Newly generated sequences were joined with sequences obtained from previous studies and sorted using the R package “ape” (Paradis, Claude, & Strimmer, 2004; Paradis & Schliep, 2019). Sequences were assembled using custom scrips in R (R Development Core Team, 2016) and aligned with the stand-alone version of MAFFT 7.273 (Katoh & Standley, 2013) with automatic selection of the appropriate alignment strategy, the scoring matrix set to “200PAM/<em>κ</em> = 2,” and the gap opening penalty set to 1.53 (with a gap extension penalty of 0.123). The obtained sequence alignments were manually checked for errors. The total sequence alignment had a length of 4,813 bp and totaled 281 taxa.</p><p>All newly generated sequences have been deposited at GenBank under the following accession numbers: MT210104–M210119 (cytb), MT210120–MT210147 (ND2), MT277429–MT277443 (myo), and MT336176–MT336215 (ODC).</p><h3 id=24-phylogenetic-reconstruction-and-divergence-time-estimation>2.4 Phylogenetic reconstruction and divergence-time estimation
<a class=anchor href=#24-phylogenetic-reconstruction-and-divergence-time-estimation>#</a></h3><p>Phylogenetic inference and divergence-time estimation were performed with BEAST v1.8.2 (Drummond, Suchard, Xie, & Rambaut, 2012). We partitioned our dataset in accordance with the best-fitting partitioning scheme that resulted from PartitionFinder v1.1.1 (Lanfear, Calcott, Ho, & Guindon, 2012) with all site models unlinked and one clock model for each gene resulting in a total of eight sites and four clock models. The search for the best strategy relied on the “beast” model-set and heuristic search. We linked all tree models to one tree model and a birth–death tree prior was applied. As initial condition for the BEAST run, we supplied a starting tree calculated with RAxML v8.2.0 (Stamatakis, 2014). Search for the best-known likelihood tree was performed with 100 replicates, bootstrap values obtained from a thorough bootstrap run using the autoMRE option (Pattengale, Alipour, Bininda-Emonds, More, & Stamatakis, 2010) were annotated onto the best-known likelihood tree. All analyses were performed using the same partitioning scheme as applied for BEAST analyses with the GTRGAMMA model applied to all partitions. The BEAST run was performed using the BEAGLE v2.3 library with a chain length of 1.1 × 108 generations with trees being sampled every 10,000 generations. The chain was inspected for convergence and sufficient ESS values (>200) with Tracer v1.6 (Rambaut & Drummond, 2007). Trees were summarized with TreeAnnotator v1.8.2, where median heights were annotated to the maximum clade credibility (MCC) tree.</p><p>We used eight calibration points in order to obtain estimates for node ages (modified from Päckert et al., 2016; see Table S4). Additionally, we applied a normal prior to the root age to avoid the occurrence of implausibly old root ages. Instead of using hard boundaries, we adopted a “soft boundary” strategy where the desired calibration is achieved by setting priors in such a way that 97.5% of the probability density lie within the desired minimum and maximum boundaries (Benton, Donoghue, & Asher, 2009). All calibrated nodes were forced to be monophyletic. Sequence alignments and tree files are deposited at Dryad under <a href=https://doi.org/10.5061/dryad.xksn02vd0>https://doi.org/10.5061/dryad.xksn02vd0</a>.</p><h3 id=25-species-distributions>2.5 Species distributions
<a class=anchor href=#25-species-distributions>#</a></h3><p>As basis of our biogeographic analysis, we delineated eight regions (see maps in Figures 3-7 with the New World as region A). Those areas, located outside the QTP region, were designed to loosely follow the classic biogeographic realms and zoogeographical boundaries (Ficetola, Mazel, & Thuiller, 2017), as done in other studies (e.g., Favre et al., 2016). We divided our target region into the high alpine Tibetan plateau (area E; Figure 2a: region 1) and two major flanking mountain systems: (a) the Central Asian Mountains in the West (area D; Figure 2a: region 2) and (b) the Sinohimalayas comprising the Himalayas (Figure 2a: region 3) and the Hengduanshan (Figure 2a: region 4) in the South and Southeast (area F). The Sinohimalayas represent one of three major global hotspots of avian diversity (e.g., Cai et al., 2018) with a high richness of both ancient and recent species (together with the Andes and the mountains of the African Arc; Fjeldså et al., 2012). Among 29 species endemic to the QTP region, ten species were present in only one area: four on the Tibetan Plateau [Figure 2a: area 1], three in the Himalayas [Figure 2a: area 3], two in the Hengduanshan [Figure 2a: area 4] and one in the Central Asian Mountains [Figure 2a: area 2]. Furthermore, our target groups comprise nineteen endemic species present in two or more areas of the QTP region (Figure 2a), among them four species which are endemic to the Sinohimalayas.</p><p>image
Figure 4
Open in figure viewer
PowerPoint</p><p><img src=/img/in-out-qinghai-tibet-plateau_fig4.jpg alt="“Out of Tibet” dispersal of snowfinches (<em>Montifringilla</em>, <em>Onychostruthus</em>, <em>Pyrgilauda</em>) and rock sparrows (<em>Petronia</em>)"></p><figcaption><p><strong>Figure 4</strong> “Out of Tibet” dispersal of snowfinches (<em>Montifringilla</em>, <em>Onychostruthus</em>, <em>Pyrgilauda</em>) and rock sparrows (<em>Petronia</em>), only the respective clade of the time-calibrated Passeroidea MCC tree is shown; node support from Bayesian inference of phylogeny (BI) and Maximum Likelihood (ML) indicated by symbols explained above the tree (no symbol for support values below 0.95/80); biogeographical reconstruction based on a dispersal–extinction–cladogenesis (DEC) model; letters at nodes show best states, pie charts show per-area probabilities inferred from ARR2 (nine areas); main dispersal events sketched on the map including area codes (right); color codes for most frequent ancestral area combinations below map (right); extant patterns of sympatry on the QTP shown on maps below for (a) small snowfinches, three <em>Pyrgilauda</em> species and one <em>Onychostruthus</em> species (BirdLife International and NatureServe, 2015); (b) large species of <em>Montifringilla</em> (Asian distributions of <em>M. nivalis</em> and <em>M. henrici</em> according to Gebauer et al., 2006)</p></figcaption>image<p>Figure 5
Open in figure viewer
PowerPoint</p><p><img src=/img/in-out-qinghai-tibet-plateau_fig5.jpg alt="“Out of Himalaya” dispersal of rosefinches"></p><figcaption><p><strong>Figure 5</strong> “Out of Himalaya” dispersal of rosefinches (<em>Carpodacus</em>; only the respective clade of the time-calibrated Passeroidea MCC tree is shown); node support from Bayesian inference of phylogeny (BI) and Maximum Likelihood (ML) indicated by symbols explained above the tree (no symbol for support values below 0.95/80); biogeographical reconstruction based on a dispersal–extinction–cladogenesis model (DEC); letters at nodes show best states, pie charts show per-area probabilities inferred from ARR2 (nine areas); main dispersal events sketched on the map including area codes (right); color codes for most frequent ancestral area combinations below map (right)</p></figcaption><p>image
Figure 6
Open in figure viewer
PowerPoint</p><p><img src=/img/in-out-qinghai-tibet-plateau_fig6.jpg alt="“Out of Himalaya” dispersal of mountain finches"></p><figcaption><p><strong>Figure 6</strong> “Out of Himalaya” dispersal of mountain finches (<em>Leucosticte</em>) and allies (only the respective clade of the time-calibrated Passeroidea MCC tree is shown); node support from Bayesian inference of phylogeny (BI) and Maximum Likelihood (ML) indicated by symbols explained above the tree (no symbol for support values below 0.95/80); biogeographical reconstruction based on a dispersal–extinction–cladogenesis model (DEC); letters at nodes show best states, pie charts show per-area probabilities inferred from ARR2 (nine areas); main dispersal events sketched on the map including area codes (right); color codes for most frequent ancestral area combinations below map (right)</p></figcaption><p>image
Figure 7
Open in figure viewer
PowerPoint</p><p><img src=/img/in-out-qinghai-tibet-plateau_fig7.jpg alt="“Into Tibet” dispersal of Old World buntings"></p><figcaption><p><strong>Figure 7</strong> “Into Tibet” dispersal of Old World buntings (<em>Emberiza</em>, four clades I-IV indicated in gray boxes; only the respective clade of the time-calibrated Passeroidea MCC tree is shown); node support from Bayesian inference of phylogeny (BI) and Maximum Likelihood (ML) indicated by symbols explained above the tree (no symbol for support values below 0.95/80); biogeographical reconstruction based on a dispersal–extinction–cladogenesis model (DEC); letters at nodes show best states, pie charts show per-area probabilities inferred from ARR2 (nine areas); main dispersal events sketched on the map including area codes (right); color codes for most frequent ancestral area combinations below map (right)</p></figcaption><p>We used QGIS v2.4 (QGIS Development Team, 2014) to draw shapefiles representing the maximum nine geographic regions in our analysis. We established a workflow for automated extraction of bird distributional ranges, using bird distribution data provided by BirdLife International and NatureServe (2015) in ESRI shapefile format. Geographic ranges encoded in BirdLife International (BLI) shapefiles were filtered to only represent year-round and breeding ranges of natively occurring species according to three types of distribution encoded in the attribute table of a shapefile. A species’ range was therefore reduced to those polygons corresponding to the following categories: type presence: extant; type origin: native; type seasonality: resident and breeding season. Subsequently, a dissolve operation was applied in QGIS to join all separate features for each species resulting in each species being represented by a single polygon feature. The intersection function (QGIS) was then used to obtain a list of all intersections between the geographic area polygons and the bird distribution polygons. The resulting list was converted to a matrix using the “xtabs” function (Rbase) and the “as.matrix.xtabs” command (DescTools).</p><p>In a second step, we made manual corrections to the area matrix for a couple of reasons. First, our data set included two species recognized by Clements et al. (2015) but not by BLI (i.e., their distributions were included in shape files of other species). These were as follows: <em>Montifringilla henrici</em> [listed as subspecies of <em>M. nivalis</em> by BLI; distribution corrected according to Gebauer, Eck, Kaiser, Lei, and Martens (2006) and Martens and Eck (2003) see Figure 4b]; <em>C. formosanus</em> [listed as ssp. of <em>C. vinaceus</em> by BLI]; this species occurs only on Taiwan [included in our area H]; for distinctiveness of <em>C. formosanus</em> according to an integrative taxonomic approach compare Wu et al., 2011). Second, based on modeled species distributions Ramesh, Gopalakrishna, Barve, and Melnick (2017) has raised the concern that for narrow-range endemics, BirdLife International shape files might overestimate the true extent of distribution range. However, their study was limited to the regional avifauna of the Western Ghats and we consider these deviations between projected BLI distributions and modeled distributions (Ramesh et al., 2017) of minor importance for our transcontinental study (even our two smallest areas the Himalayas and the Hengduanshan are 1.7–3.7 times larger than the Western Ghats). Furthermore, other studies suggested a good accordance of potential distribution models (PDMs) and BLI distributions for several Mexican bird species (Ortega-Huerta & Vega-Rivera, 2017). Nevertheless, we took into account that automatic extraction of species ranges from BLI shape files might cause false-positive occurrence of a species in an area with only marginal overlap at the very margins of its breeding range (examples of <em>Carpodacus</em> rosefinches are shown in Figure S2). Because we consider this a potential bias of our automatic extraction of bird ranges from BirdLife shapefiles, we deleted these marginal areas from the area matrix. We cross-checked our area classification with published range maps in the Atlas of Palearctic Breeding Birds (for example for rosefinches, Carpodacini, in Martens & Sun, 2008) as was done in previous studies (e.g., Drovetski et al., 2013) (further atlas volumes consulted: Dathe & Loskot, 1992; Stresemann & Portenko, 1960, 1976, 1977, 1978, 1980, 1981; Stresemann, Portenko, & Mauersberger, 1971; Stresemann, Portenko, Dathe, & Mauersberger, 1974). Our final area matrix is included in our data package deposited at Dryad under <a href=https://doi.org/10.5061/dryad.xksn02vd0>https://doi.org/10.5061/dryad.xksn02vd0</a>.</p><h3 id=26-ancestral-range-reconstruction-arr>2.6 Ancestral-range reconstruction (ARR)
<a class=anchor href=#26-ancestral-range-reconstruction-arr>#</a></h3><p>We used R package BioGeoBEARS (Matzke, 2013) for ancestral-range reconstruction (ARR). The maximum clade credibility tree obtained from BEAST was used as input tree (we kept the waxwing, <em>B. garrulus</em>, as the closest outgroup of Passeroidea and pruned all further outgroups used for fossil calibration from the input tree shown in Figure S1). We fitted the dispersal, extinction, and cladogenesis (DEC) model to the time-calibrated Passeroidea tree (for further details, see Appendix S5). We used a “dispersal multipliers” matrix, allowing dispersal between all areas, favoring adjacent ones (1.0), but penalizing slightly (0.5) dispersal between nonadjacent areas interconnected by a third one (in a land continuum), and penalizing strongly (0.01) long distance dispersal between very distant continents (e.g., Europe and the New World; see Tables S5a,b).</p><p>We performed two alternative runs with BioGeoBEARS: one with the Sinohimalayas classified as a single area F (ARR1; Figure S3) and a second run with the Himalayas and the Hengduanshan as two separate areas (F1, F2; ARR2) with the Mekong-Salween divide as one of the main geographic barriers (compare Liu et al., 2016; Päckert et al., 2012). Separate treatment of these two mountain regions also takes into consideration the different orogenetic histories and the younger age of the Hengduanshan (Favre et al., 2015; and references therein). Dispersal multipliers were adjusted to the latter analyses with eight and nine areas, respectively (Tables S5a,b). Our final dispersal multiplier matrices were included in our data package deposited at Dryad under <a href=https://doi.org/10.5061/dryad.xksn02vd0>https://doi.org/10.5061/dryad.xksn02vd0</a>.</p><h2 id=3-results>3 RESULTS
<a class=anchor href=#3-results>#</a></h2><h3 id=31-phylogeny-and-origin-of-passeroidea>3.1 Phylogeny and origin of Passeroidea
<a class=anchor href=#31-phylogeny-and-origin-of-passeroidea>#</a></h3><p>Phylogenetic analyses with BEAST and RAxML resulted in largely congruent topologies for the major families of Passeroidea (Figure S1; autoMRE bootstrap converged after 200 replicates). Among all Passeroidea, Przevalski&rsquo;s finch, <em>Urocynchramus pylzowi</em>, represented the oldest species-level lineage endemic to the QTP (Figure S3). The split from its sister group (Ploceidae) was dated to a mean age of 20 Ma [16.4–24.7 Ma], which corresponds to the onset of the diversification for all our five target clades (see below). All fourteen endemic species of the QTP region (that occurred in a single area, including four Sinohimalayan endemics) emerged during the late Miocene (e.g., Tibetan bunting, <em>E. koslowi</em>) or early Pleistocene (e.g., Tibetan Snowfinch, <em>M. henrici</em>).</p><p>In ARR1 and ARR2, area uncertainty was high at the basal nodes of the Passeroidea tree, however, best states suggested a Holarctic ancestral range of Passeroidea (Afrotropics and Oriental region were not included in the ancestral range; Figure S3). In contrast, for several suprageneric clades, ancestral ranges were limited to one region only, for example, a Nearctic origin of “Emberizoidea” sensu Barker, Burns, Klicka, Lanyon, and Lovette (2013) (see Figure S1) and Fringillidae (see Figure S3).</p><h3 id=32-biogeographical-history-of-montane-and-alpine-target-groups-of-passeroidea>3.2 Biogeographical history of montane and alpine target groups of Passeroidea
<a class=anchor href=#32-biogeographical-history-of-montane-and-alpine-target-groups-of-passeroidea>#</a></h3><p>We found contrasting areas of diversification and dispersal routes between the QTP and adjacent regions for our five target clades. Generally, there was large congruence between reconstructions based on eight areas (ARR1) and nine areas (ARR2), respectively. Separate classification of the Himalayas and the Hengduanshan as two areas F1 and F2 (ARR2) often resulted in ancestral ranges with highest per-area probabilities in either of the two regions (at nodes with an ancestral range in the Sinohimalayas in ARR1; Figures S3 and S4). In the following, we present and discuss the results from ARR2 based on nine areas and refer to deviations among ARRs only in the one case that showed an effect of the total number of areas on area uncertainty.</p><p>A QTP origin (area E) was recovered for snowfinches and rock sparrows (Figure 4), with a mean root age estimate of 12.4 [9.9–15.3] Ma for the onset of their radiation. For their sister clade (Old World sparrows of genera <em>Passer</em> and <em>Gymnoris</em>) an African area of origin was estimated (best state and highest per-area probabilities). Genus Passer includes several widespread species whose ranges extend into the QTP region (Figures S3 and S4), however, none of these would represent a QTP endemic that had colonized the QTP from elsewhere. Two main clades of snowfinches (<em>Pyrgilauda</em>, small species; <em>Montifringilla</em>, large species) separated at about 7.6 [6.0–9.6] Ma. Area E was the best state and had highest per-area probabilities at most nodes of the snowfinch clade. Only for the ancestral range of small <em>Pyrgilauda</em> species, the Sinohimalayas had the highest per-area probabilities in ARR2 (two areas F1, F2, whereas from ARR1 again area E resulted as best state for the ancestral range of <em>Pyrgilauda</em>). The <em>Montifringilla</em> clade included the only snowfinch species that during the late Pleistocene dispersed as far as to the Western Palearctic (<em>M. nivalis</em>; Figure 4).</p><p>Areas of origin and diversification in the QTP region were also reconstructed for both our target clades of finches (Fringillidae). The onset of rosefinch (Carpodacini) radiation was dated to 11.2 [9.5–13.3] Ma and involved major centers of diversification on the QTP (area E) and in the Sinohimalayas (areas F1 and F2; Figure 5). Their sister clade, the Hawaiian Honeycreepers (Drepanidini), represents a solely Nearctic group. Two larger rosefinch clades diversified in different areas of the QTP region: Clade I comprised 15 species that diversified in the Himalayas (highest or high per-area probabilities for area F1 at five nodes; Sinohimalayas best state; Figure 5, Figure S3). The Himalayan rosefinch radiation included one early dispersal event to Central Asia (area D) and a more recent dispersal event to southern China and Taiwan (area H; Figure 5). For the second major rosefinch clade II, comprising ten species, area E had the highest per-area probabilities at the two successively basal nodes (Figure 5). However, at several nodes of that second clade, the Hengduanshan (area F2) had highest (or high) per-area probabilities or was recovered as best state (Figure 5). Both groups included a terminal Pleistocene dispersal event to the Western Palearctic (area B; Figure 5: clade I, <em>C. rubicilla</em>; clade II: C. <em>synoicus</em>).</p><p>A Himalayan origin was reconstructed for another finch clade, Pyrrhulini, that comprised the mountain finches of genus <em>Leucosticte</em>, bullfinches of genus <em>Pyrrhula</em> and allies from five other genera (Figure 6; highest per-area probabilities for area F1, and best state F1 at several clades). Pyrrhulini are sister to a diverse group of finches (tribe Carduelini) that among others includes several subclades restricted to the New World or to the Afrotropics (Figure S3). The high per-area probability for a New World origin of Carduelini (Figure 6) is certainly due to the basal split leading to the Nearctic house finch clade (genus <em>Haemorhous</em>; Figure S3). One Tibeto-Himalayan endemic, the Tibetan serin (<em>Serinus thibetanus</em>), emerged from the Carduelini clade, sister to a New World clade of siskins (Figure S3: the Eurasian siskin, <em>Spinus spinus</em>, belongs to this clade and has very likely colonized the Palearctic from the New World). Strikingly, this Tibeto-Himalayan endemic does not have any close phylogenetic relationship with other QTP species. The onset of Pyrrhulini diversification was dated to a mean age of 12.3 [10.2–14.4] Ma and their radiation also included a very recent terminal dispersal event of mountain finches (<em>Leucosticte</em>) into the Nearctic (area A1; Figure 6). The entire Himalayan finch group belonged to a larger finch clade, for which a Central Asian ancestral range was inferred (highest per-area probabilities for area D or DF).</p><p>In the buntings (<em>Emberiza</em>) several extant taxa from the QTP were nested within two larger clades that did not originate and diversify in the QTP region. Their sister group is represents a New World clade, the Passerellidae (Figure 7). An Eastern Palearctic center of origin and diversification was unanimously assigned to a clade comprising sixteen species including the Chinese endemic <em>E. siemsseni</em> and further distinct subspecific lineages of East Asian species (Figure 7, clade IV). Breeding ranges of all QTP taxa of clade IV are mainly restricted to the Hengduanshan (area F2 and parts of the adjacent QTP, area E; Figure 7). The ancestral area of clade II was equivocal (many combinations with low per-area probabilities at the two successively basal nodes; Figure 7). The Tibetan bunting (<em>E. koslowi</em>) was one of three early offshoots of clade II (mean split age 9.1 [7.5–10.9] Ma) and the QTP (area E) was recovered as the ancestral range at that node (highest per-area probabilities and best state; Figure 7).</p><p>The onset of accentor (<em>Prunella</em>) radiation was dated to 11.9 [9.4–14.7] Ma. The area origin of accentors (<em>Prunella</em>) was highly equivocal (low per-area probabilities for any combination of regions); however, area E (high QTP) was recovered as best state in both ARRs (Figure 8). For the basal nodes, there was less area uncertainty, when the Sinohimalayas were coded as a single area (a total of eight areas in ARR1 instead of nine in ARR2). According to ARR2, the QTP (area E) was the ancestral range of the accentor crown clade with highest per-area probabilities (Figure 8).</p><p>image
Figure 8
Open in figure viewer
PowerPoint</p><p><img src=/img/in-out-qinghai-tibet-plateau_fig8.jpg alt="“Out of Tibet” dispersal of accentors"></p><figcaption><p><strong>Figure 8</strong> “Out of Tibet” dispersal of accentors (<em>Prunella</em>; only the respective clade of the time-calibrated Passeroidea MCC tree is shown); node support from Bayesian inference of phylogeny (BI) and Maximum Likelihood (ML) indicated by symbols explained above the tree (no symbol for support values below 0.95/80); biogeographical reconstruction based on a dispersal–extinction–cladogenesis model (DEC); letters at nodes show best states, pie charts show per-area probabilities inferred from ARR2 (nine areas; plus those from ARR1 with eight areas and less uncertainty at the three basal nodes); main dispersal events sketched on the map including area codes (right); color codes for most frequent ancestral area combinations below map (right)</p></figcaption><h2 id=4-discussion>4 DISCUSSION
<a class=anchor href=#4-discussion>#</a></h2><h3 id=41-phylogeny-and-biogeographic-history-of-passeroidea>4.1 Phylogeny and biogeographic history of Passeroidea
<a class=anchor href=#41-phylogeny-and-biogeographic-history-of-passeroidea>#</a></h3><p>Our Passeroidea tree is in good accordance with previously published phylogenies, most of them based on a less dense taxon sampling (Barker et al., 2013; Davis & Page, 2014; Johansson et al., 2008; Jønsson & Fjeldsa, 2006). The timing of lineage splits inferred from our fossil dating approach is in accordance with other dated phylogenies (Barker et al., 2013; Claramunt & Cracraft, 2015; Moyle et al., 2016). Our ancestral-range reconstructions suggested a Holarctic origin of Passeroidea, whereas previous studies have suggested either a Palearctic origin (Claramunt & Cracraft, 2015) or an Indomalayan origin (Moyle et al., 2016) of this group. The latter difference is striking, as both studies applied the same global area coding. For our data set, uncertainty of ancestral ranges increased with the number of global areas classified (eight and nine) as illustrated for accentors, Prunellidae (Figure 8). Though previous studies on the biogeographical history of this family relied on even fewer areas (four and six), they predicted a rather broad ancestral range across the entire QTP region (Drovetski et al., 2013; Liu et al., 2017). Apart from different reconstruction methods, deviations between our results and those of the two latter studies might be due to different classifications of global areas on the one hand, and different delimitation of species distribution ranges on the other hand (e.g., inclusion or exclusion of the Himalayan nonbreeding range of <em>Prunella himalayana</em>). Thus, the outcome of ancestral area reconstructions might not only depend on the models applied (Clark et al., 2008; Pirie, Humphreys, Antonelli, Galley, & Linder, 2012) and different reconstruction methods (Johansson, Nylinder, Ohlson, & Tietze, 2018; Tietze et al., 2013), but also the area-coding approach. This has to be kept in mind for the discussion of the following evolutionary trajectories that we identified for our five Eurasian montane and alpine target clades.</p><h3 id=42-out-of-qtpin-situ-diversification-on-the-qtp-and-along-its-forested-fringes>4.2 “Out of QTP”—in situ diversification on the QTP and along its forested fringes
<a class=anchor href=#42-out-of-qtpin-situ-diversification-on-the-qtp-and-along-its-forested-fringes>#</a></h3><p>Our results suggested a late Miocene origin (at about 8 Ma) and in situ diversification on the QTP and along its margins for snowfinches and for rosefinches of clade II. At the same time, ancestors of the major accentor clades dispersed “out of a Tibet” (area E) to a wide ancestral range including Central Asia and the Northern Palearctic (areas DEFG). There, ancestral lineages of accentors appear to have diversified from the late Miocene onward. In these three examples, that early phase of their radiation (8–5 Ma) coincides with a period of global climate cooling toward the end of the Miocene (reviews in Favre et al., 2015; Mosbrugger et al., 2018), with an intensification of Asian winter monsoon (Holbourn et al., 2018) and increasing aridification of the QTP (Miao et al., 2019; but compare Nie et al., 2017 for a warmer and wetter Miocene climate in the QTP region). Along with a decrease of birch and oak forests and a continuous change toward steppe vegetation on the northern QTP from 8.5 Ma on (Chen & Yang, 2016; Hui et al., 2011), new semiopen plateau habitats could have represented an evolutionary opportunity for the ancestors of extant QTP rosefinches, snowfinches, and accentors.</p><p>Adaptive radiations on the QTP appear to have followed general ecogeographic rules. For example, smaller species of snowfinches (<em>Pyrgilauda</em>) diversified at the southern QTP margins, whereas larger species (<em>Montifringilla</em>) did so at higher latitudes, in the northern QTP margins. Similar accordance to Bergmann&rsquo;s rule (Bergmann, 1847) was found along a latitudinal gradient on the QTP in small mammals such as pikas (Lin, Ci, Zhang, & Su, 2008) and zokors (Zhang, Nevo, Tang, Su, & Lin, 2012).</p><p>In the final Holocene phase of alpine passerine radiations, postglacial range expansion resulted in patterns of wide-range sympatry on the QTP, where today extant endemics occupy different ecological niches of the alpine meadows and grasslands. Snowfinch species of the QTP belong to two different ecological guilds with different habitat preferences for vertical or horizontal habitat complexity (Li et al., 2018). Similarly, all sympatric accentor (<em>Prunella</em>) species of the high alpine habitats on the QTP and its margins differ strikingly from each other in habitat preferences and in morphology (Drovetski et al., 2013; Liu et al., 2017). Only a few snowfinch and rosefinch species have colonized alpine habitats of central and western Palearctic mountain systems according to the “out of Tibet” hypothesis (e.g., <em>M. nivalis</em>) during the Pleistocene. Likewise, the (Sino-)Himalayas have been considered a source area of Holarctic montane organisms (Favre et al., 2016; Matuszak, Muellner-Riehl, Sun, & Favre, 2016; Pisano et al., 2016; Wen, Zhang, Nie, Zhong, & Sun, 2014) and “out of Himalayas” colonization in our target groups was suggested for the transcontinental colonization of Nearctic mountain systems from a Himalayan area of origin in mountain finches (<em>Leucosticte</em>; this study). A single rosefinch species emerged from dispersal out of the Himalayas to a subtropical region, that is, <em>C. formosanus</em> on Taiwan. Though none of the five radiations of montane Passeroidea reached into tropical regions (except of one bullfinch species, <em>Pyrrhula leucogenis</em>, on the Philippines), other clades of Passeroidea encompass tropical radiations across Southeast Asia (including the Sinohimalayas) such as those of sunbirds and flowerpeckers (Nectariniidae, <em>Aethopgya</em>: Hosner, Nyári, & Moyle, 2013; Dicaeidae: Nyári, Peterson, Rice, & Moyle, 2009). The Sinohimalayas were also considered the area of origin and in situ diversification of babblers, a highly diverse passerine group that from the early Miocene on colonized the entire Old World including the Tropics where regions of great babbler diversity can be found today (Cai et al., 2020).</p><p>Moreover, biogeographic scenarios reconstructed for Passeroidea did not only include faunal interchange among the QTP and adjacent regions but also within the QTP region itself (e.g., exchange between areas D, E, and F).</p><h3 id=43-faunal-exchange-between-the-plateau-region-and-the-qtp-fringes>4.3 Faunal exchange between the plateau region and the QTP fringes
<a class=anchor href=#43-faunal-exchange-between-the-plateau-region-and-the-qtp-fringes>#</a></h3><p>The discussion about processes fostering diversification within the QTP region has often been limited to phylogeographic patterns, that is, intraspecific genetic divergence as a result of Pleistocene range shifts to glacial refuges along its margins and successive Holocene range expansion (e.g., Lei, Qu, & Song, 2014; Yang, Dong, & Lei, 2009). This particular evolutionary scenario was recently summarized in the “contraction/recolonization” hypothesis by Muellner-Riehl (2019). However, for several of the Tibetan radiations during the Ice Ages also “platform refugia” and “microrefugia” (Muellner-Riehl, 2019) might explain the current patterns of sympatry and parapatry, such as for rosefinches of the <em>Carpodacus pulcherrimus</em> complex (this study, Figure 5; Tietze et al., 2013). Moreover, faunal exchange between the plateau and its peripheral forested mountain regions must have already occurred prior to the Pleistocene. The scenario of in situ diversification in the Himalayas for two finch clades (rosefinches: Figure 5; mountain finches and allies: Figure 6) suggested a major phase of diversification from 10 to 5 Ma that coincides with a period of accelerated elevational niche divergence of Himalayan passerines that reached its peak in the late Pliocene (Price et al., 2014). Along with the establishment of elevational parapatry, the <em>Carpodacus</em> rosefinch radiation involved multiple independent “into Tibet” movements, that is, colonizations of the plateau (area E) from its forested fringes (areas F1 and F2). These were associated with independent colonizations of semiopen and open alpine habitats from ancestral forest habitats (Päckert, Martens, Sun, & Tietze, 2015). In other alpine groups of organisms, there is evidence of the reverse directionality of faunal exchange, that is, from the central plateau to its margins. Indeed, the ancestors of a few Himalayan faunal elements seem to have originated from a Tibetan source area. With increasing aridification and loss of mesophilic forests on the QTP, these Tibetan stem populations dispersed southward into the “Himalayan exile” (area F1) where they later diversified as documented for lazy toads (Hoffmann et al., 2017) and ground beetles (Schmidt, Opgenoorth, Höll, & Bastrop, 2012; further examples in Martens, 2015).</p><h3 id=44-immigration-into-the-qtp-region-from-adjacent-source-regions>4.4 Immigration into the QTP region from adjacent source regions
<a class=anchor href=#44-immigration-into-the-qtp-region-from-adjacent-source-regions>#</a></h3><p>The alpine QTP fauna and flora includes some immigrants from distant regions like the Nearctic (alpine plants: Ebersbach et al., 2017) and the Mediterranean (Clewing, von Oheimb, Vinarski, Wilke, & Albrecht, 2014; Wen et al., 2014), whereas the Himalayan bird communities were apparently shaped by immigration from adjacent subtropical areas in Southeast Asia (Johansson et al., 2007; Päckert et al., 2012).</p><p>Among our target groups, some Old World buntings (Emberiza) have colonized the eastern plateau margin from a center of origin and diversification northeast of the QTP (area G; similar scenario with higher area uncertainty for bullfinches). Weigold (2005) already supposed that those immigrants from the Northeast were already preadapted to cold and dry environments, given their presumed origin in the arid belt of Central Asian steppes and deserts. Further examples of Eastern Palearctic taxa on the QTP are found in rodents (Li & Wang, 2015). Also, depending on the perspective, the “Out of North China” hypothesis by Zhang et al. (2006) for Asian salamanders (Hynobiidae) can be regarded as an “into Tibet” dispersal. In northern China, successive cyclical vegetation shifts between steppe and forest (or grassland and desert) ecosystems during the Pliocene (Wang et al., 2006), might have furthermore promoted north–south faunal interchange at the eastern QTP margin (Päckert et al., 2012). Only a few of these Eastern Palearctic immigrants actually colonized the Himalayas (two bullfinch species for example; this study), thus these East Siberian/Mongolian taxa are characteristic for the montane and alpine ecosystems of the Hengduanshan (our area F2), that is, the eastern QTP margin.</p><p>Several authors hypothesized that, as opposed to “centers of origin,” an area can constitute a “center of accumulation,” that is, a region that obtained taxa through unidirectional immigration from different source areas (Goldberg, Roy, Lande, & Jablonski, 2005; Mora, Chittaro, Sale, Kritzer, & Ludsin, 2003). As an equivalent term, an “immigration center” in the Sinohimalayas was suggested by Liu et al. (2017), however, the authors pointed out that evolutionary history of the regional avifauna was shaped by both in situ speciation and immigration. Also, Martens (2015) emphasized that biogeographic affinities of the Himalayas are manifold and during its long uplift history the Himalayan Arc and its extensions in the East did “not merely absorb” immigrant faunal elements from adjacent regions.</p><h2 id=5-conclusion>5 CONCLUSION
<a class=anchor href=#5-conclusion>#</a></h2><p>Researchers of the early 20th century had already developed clear-sighted hypotheses on centers of origin and diversification in the QTP region as well as on a Tibetan immigrant mammal fauna from cold and arid environments north of the QTP (Weigold, 1949). This is all the more impressive, because these early biogeographic hypotheses were formulated in times prior to the application of cladistic methods or phylogenetic analysis of molecular data. Our analysis suggests that, in accordance with Weigold&rsquo;s predictions, centers of origin and (in situ) diversification of extant QTP passerine bird genera were not restricted the QTP platform, but also occurred along its southern and southeastern margins (in the Himalayas and the Hengduanshan), and northeast of the QTP in the Eastern Palearctic. Faunal interchange among the QTP, its flanking mountains, and the adjacent regions must therefore have been a bidirectional process rather than a one-way street (for birds: Cai et al., 2018; Liu et al., 2016; for alpine plants: Ebersbach et al., 2017).</p><p>Previous biogeographical studies already acknowledged the fact that faunal interchange between adjacent bioregions in other parts of the world is a “two-way traffic” (Cheetham, 1963), such as the Great American Biotic Interchange. Although it has been assumed that faunal interchange across the Isthmus of Panama for birds went primarily from south to north (Weir, Bermingham, & Schluter, 2009), recent studies suggested that transcontinental dispersal and colonization of new habitat occurred in both directions (Pelegrin, Gamboa, Menéndez, & Hernández-Fernández, 2018; Woodburne, 2010). As a result, extant Neotropical faunal assemblages are composed by both native and immigrant taxa (Cione, Gasparini, Soibelzon, Soibelzon, & Tonni, 2015) and this is what Weigold (2005) suggested for the Tibetan avifauna, too. The same pattern of a bidirectional faunal interchange can be expected from future studies that are dedicated either to speciose families with large transcontinental distributions (Ebersbach et al., 2017) or to comparisons of independent radiations across clades of a speciose taxon such as Passeroidea in this study.</p><h2 id=acknowledgments>ACKNOWLEDGMENTS
<a class=anchor href=#acknowledgments>#</a></h2><p>This study received substantial support from the German Research Foundation (Deutsche Forschungsgemeinschaft, DFG; project PA1818/3-1). I.M. was funded by DFG project MU 2934/2-1, A.F was funded by DFG project FA1117/1-2. J.S. was supported by the German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig funded by the German Research Foundation (DFG—FZT 118). J.M. received continuous financial support from Feldbausch-Stiftung and Wagner-Stiftung at Fachbereich Biologie of Mainz University for field research in Asia. We are grateful to all colleagues who provided sample loans for molecular analyses above all to S. Birks at Burke Museum of Natural History and Culture, Seattle, USA; J. Fjeldså at Zoological Museum of the University of Copenhagen, Denmark; H. Grimm at Naturkundemuseum Erfurt, Germany. Wet-lab analyses were performed at the molecular lab of SNG-SNSD and we thank C. Kehlmaier, A. Müller and A. Rauh for assistance, and L. Dong for performing some additional wet-lab work in China. We also thank the subject editor and two anonymous reviewers for constructive criticism on the manuscript. Open access funding enabled and organized by Projekt DEAL.</p><p>[Correction added on March 27, after first online publication: Projekt Deal funding statement has been added.]</p><h2 id=conflict-of-interest>CONFLICT OF INTEREST
<a class=anchor href=#conflict-of-interest>#</a></h2><p>No conflict of interest has been declared by the authors.</p><h2 id=author-contributions>AUTHOR CONTRIBUTIONS
<a class=anchor href=#author-contributions>#</a></h2><p>Martin Päckert: Conceptualization (lead); Data curation (lead); Formal analysis (supporting); Funding acquisition (lead); Investigation (lead); Methodology (supporting); Project administration (lead); Resources (lead); Software (supporting); Supervision (lead); Validation (lead); Visualization (lead); Writing-original draft (lead); Writing-review & editing (lead). Adrien Favre: Conceptualization (supporting); Formal analysis (equal); Methodology (equal); Software (equal); Visualization (supporting); Writing-original draft (supporting); Writing-review & editing (supporting). Jan Schnitzler: Formal analysis (supporting); Methodology (supporting); Software (supporting); Validation (supporting); Writing-review & editing (supporting). Jochen Martens: Conceptualization (supporting); Data curation (supporting); Funding acquisition (supporting); Investigation (supporting); Resources (supporting); Supervision (supporting); Writing-original draft (supporting); Writing-review & editing (supporting). Yue-Hua Sun: Funding acquisition (supporting); Investigation (supporting); Resources (supporting); Writing-review & editing (supporting). Dieter Thomas Tietze: Conceptualization (supporting); Data curation (supporting); Funding acquisition (supporting); Investigation (supporting); Methodology (supporting); Software (supporting); Validation (supporting); Writing-review & editing (supporting). Frank Hailer: Conceptualization (equal); Funding acquisition (supporting); Investigation (supporting); Methodology (supporting); Writing-review & editing (supporting). Ingo Michalak: Formal analysis (supporting); Methodology (equal); Software (supporting); Validation (supporting); Writing-review & editing (supporting). Patrick Strutzenberger: Conceptualization (supporting); Data curation (supporting); Formal analysis (equal); Investigation (supporting); Methodology (equal); Software (equal); Supervision (supporting); Validation (supporting); Writing-original draft (supporting); Writing-review & editing (supporting).</p><h2 id=data-availability-statement>DATA AVAILABILITY STATEMENT
<a class=anchor href=#data-availability-statement>#</a></h2><p>All newly generated sequences have been submitted to GenBank (accession numbers: see Table S3). The tree file and further informa- tion such as data matrices (e.g., area matrices) have been deposited on Dryad under <a href=https://doi.org/10.5061/dryad.xksn02vd0>https://doi.org/10.5061/dryad.xksn02vd0</a>.</p><h2 id=orcid>ORCID
<a class=anchor href=#orcid>#</a></h2><p>Martin Päckert <a href=https://orcid.org/0000-0001-5045-0139>https://orcid.org/0000-0001-5045-0139</a></p><p>Adrien Favre <a href=https://orcid.org/0000-0001-6132-2992>https://orcid.org/0000-0001-6132-2992</a></p><p>Dieter Thomas Tietze <a href=https://orcid.org/0000-0001-6868-227X>https://orcid.org/0000-0001-6868-227X</a></p><p>Open Research
Supporting Information</p><h2 id=references>REFERENCES
<a class=anchor href=#references>#</a></h2><ul><li>Aliabadian, M., Sluys, R., Roselaar, C. S., & Nijman, V. (2008). Species di- versity and endemism: Testing the mid-domain effect on species rich- ness patterns of songbirds in the Palearctic region. <em>Contributions to Zoology</em>, 77, 99–108. <a href=https://doi.org/10.1163/18759866-07702006>https://doi.org/10.1163/18759866-07702006</a></li><li>Baker, A. J., Pereira, S. L., & Paton, T. A. (2007). Phylogenetic relation- ships and divergence times of Charadriiformes genera: Multigene ev- idence for the Cretaceous origin of at least 14 clades of shorebirds. <em>Biology Letters</em>, 3, 205–209. <a href=https://doi.org/10.1098/rsbl.2006.0606>https://doi.org/10.1098/rsbl.2006.0606</a></li><li>Barker, F. K., Burns, K. J., Klicka, J., Lanyon, S. M., & Lovette, I. J. (2013). Going to extremes: Contrasting rates of diversification in a recent radiation of New World passerine birds. <em>Systematic Biology</em>, 62, 298– 320. <a href=https://doi.org/10.1093/sysbio/sys094>https://doi.org/10.1093/sysbio/sys094</a></li><li>Belkacem, A. A., Gast, O., Stuckas, H., Canal, D., LoValvo, M., Giacalone, G., & Päckert, M. (2016). North African hybrid sparrows (Passer domesticus, P. hispaniolensis) back from oblivion – ecological segregation and asymmetric mitochondrial introgression between parental species. <em>Ecology and Evolution</em>, 6, 5190–5206.</li><li>Benton, M. J., Donoghue, P. C., & Asher, R. J. (2009). Calibrating and con- straining molecular clocks. In S. B. Hedges, & S. Kumar (Eds.), <em>The timetree of life</em> (pp. 35–86). Oxford, UK: Oxford University Press.</li><li>Bergmann, C. (1847). Über die Verhältnisse der Wärmeökonomie der Thiere zu ihrer Größe. Göttingen, Germany: Göttinger Studien. BirdLife International and NatureServe (2015). <em>Bird species distribution maps of the world</em>. Version 5.0. BirdLife International, Cambridge, UK and NatureServe, Arlington, USA.</li><li>Cai, T., Fjeldså, J., Wu, Y., Shao, S., Chen, Y., Quan, Q., &mldr; Lei, F. (2018). What makes the Sino-Himalayan mountains the major diversity hotspots for pheasants? <em>Journal of Biogeography</em>, 45, 640–651. <a href=https://doi.org/10.1111/jbi.13156>https://doi.org/10.1111/jbi.13156</a></li><li>Cai, T., Shao, S., Kennedy, J. D., Alström, P., Moyle, R. G., Qu, Y., &mldr; Fjeldså, J. (2020). The role of evolutionary time, diversification rates and dispersal in determining the global diversity of a large radiation of passerine birds. <em>Journal of Biogeography</em>, 47, 1612–1625. https://doi. org/10.1111/jbi.13823</li><li>Cheetham, A. H. (1963). Late Eocene zoogeography of the Eastern Gulf Coast Region. New York, NY: <em>The Ecological Society of America</em>.</li><li>Chen, Y. M., & Yang, X. N. (2016). Miocene woods from the Qaidam Basin on northern Qinghai-Tibet Plateau with implications for paleoen- vironmental change. <em>Journal of Asian Earth Sciences</em>, 116, 198–207. <a href=https://doi.org/10.1016/j.jseaes.2015.11.022>https://doi.org/10.1016/j.jseaes.2015.11.022</a></li><li>Cione, A. L., Gasparini, G. M., Soibelzon, E., Soibelzon, L. H., & Tonni, E. P. (2015). <em>The Great American biotic interchange – A South American perspective</em>. Do[r]drecht, Heidelberg, New York, London: Springer.</li><li>Claramunt, S., & Cracraft, J. (2015). A new time tree reveals Earth histo- ry’s imprint on the evolution of modern birds. <em>Evolutionary Ecology</em>, 1, e1501005. <a href=https://doi.org/10.1126/sciadv.1501005>https://doi.org/10.1126/sciadv.1501005</a></li><li>Clark, J. R., Ree, R. H., Alfaro, M. E., King, M. G., Wagner, W. L., & Roalson, E. H. (2008). A comparative study in ancestral range reconstruction meth- ods: Retracing the uncertain histories of insular lineages. <em>Systematic Biology</em>, 57, 693–707. <a href=https://doi.org/10.1080/10635150802426473>https://doi.org/10.1080/10635150802426473</a></li><li>Clements, J. F., Schulenberg, T. S., Iliff, M. J., Roberson, D., Fredericks, T. A., Sullivan, B. L., & Wood, C. L. (2015) <em>The eBird/Clements checklist of birds of the world: v2015</em>. Retrieved from <a href=http://www.birds.corne>http://www.birds.corne</a> ll.edu/clementschecklist/download/</li><li>Clewing, C., von Oheimb, P. V., Vinarski, M., Wilke, T., & Albrecht, C. (2014). Freshwater mollusc diversity at the roof of the world: Phylogenetic and biogeographical affinities of Tibetan Plateau Valvata. <em>Journal of Molluscan Studies</em>, 80, 452–455. https://doi. org/10.1093/mollus/eyu016</li><li>Dathe, H., & Loskot, W. M. (1992). Atlas der Verbreitung palaearktischer Vögel – 18. Lieferung. Berlin, Germany: Akademie-Verlag.</li><li>Davis, K. E., & Page, R. D. M. (2014). Reweaving the Tapestry: a Supertree of Birds. <em>PLoS Currents</em>, 6. <a href=https://doi.org/10.1371/currents.tol.c1af6>https://doi.org/10.1371/currents.tol.c1af6</a> 8dda7c999ed9f1e4b2d2df7a08e</li><li>Del Mar Delgado, M., Bettega, C., Martens, J., & Päckert, M. (2019). Ecotypic changes of alpine birds to climate change. <em>Scientific Reports</em>, 9, 16082. <a href=https://doi.org/10.1038/s41598-019-52483-0>https://doi.org/10.1038/s41598-019-52483-0</a></li><li>Deng, T., Wang, X., Fortelius, M., Li, Q., Wang, Y., Tseng, Z. J., &mldr; Xie, G. (2011). Out of Tibet: Pliocene woolly rhino suggests high-plateau origin of Ice Age megaherbivores. <em>Science</em>, 333, 1285–1288. https:// doi.org/10.1126/science.1206594</li><li>Drovetski, S. V., Semenov, G., Drovetskaya, S. S., Fadeev, I. V., Red&rsquo;kin, Y. A., & Voelker, G. (2013). Geographic mode of speciation in a moun- tain specialist avian family endemic to the Palearctic. <em>Ecology and Evolution</em>, 3, 1518–1528. <a href=https://doi.org/10.1002/ece3.539>https://doi.org/10.1002/ece3.539</a></li><li>Drovetski, S. V., Zink, R. M., & Mode, N. A. (2009). Patchy distribu- tions belie morphological and genetic homogeneity in rosy-finches. <em>Molecular Phylogenetics and Evolution</em>, 50, 437–445. https://doi. org/10.1016/j.ympev.2008.12.002</li><li>Drummond, A. J., Suchard, M. A., Xie, D., & Rambaut, A. (2012). Bayesian phylogenetics with BEAUti and the BEAST 1.7. <em>Molecular Biology and Evolution</em>, 29, 1969–1973. <a href=https://doi.org/10.1093/molbev/mss075>https://doi.org/10.1093/molbev/mss075</a></li><li>Ebersbach, J., Muellner-Riehl, A. N., Michalak, I., Tkach, N., Hoffmann, M. H., Röser, M., &mldr; Favre, A. (2017). In and out of the Qinghai-Tibet Plateau: Divergence time estimation and historical biogeography of the large arctic-alpine genus Saxifraga L. <em>Journal of Biogeography</em>, 44, 900–910.</li><li>Favre, A., Michalak, I., Chen, C. H., Wang, J. C., Pringle, J. S., Matuszak, S., &mldr; Muellner-Riehl, A. N. (2016). Out-of-Tibet: The spatio-tempo- ral evolution of Gentiana (Gentianaceae). <em>Journal of Biogeography</em>, 43, 1967–1978.</li><li>Favre, A., Päckert, M., Pauls, S. U., Jähnig, S., Uhl, D., Michalak, I., & Muellner-Riehl, A. N. (2015). The role of the uplift of the Qinghai- Tibetan Plateau for the evolution of Tibetan biotas. <em>Biological Reviews</em>, 90, 236–253. <a href=https://doi.org/10.1111/brv.12107>https://doi.org/10.1111/brv.12107</a></li><li>Ficetola, G. F., Mazel, F., & Thuiller, W. (2017). Global determinants of zoogeographical boundaries. <em>Nature Ecology and Evolution</em>, 1, 0089. <a href=https://doi.org/10.1038/s41559-017-0089>https://doi.org/10.1038/s41559-017-0089</a></li><li>Fjeldså, J., Bowie, R. C. K., & Rahbeck, C. (2012). The role of moun- tain ranges in the diversification of birds. <em>Annual Review of Ecology, Evolution, and Systematics</em>, 43, 249–265. <a href=https://doi.org/10.1146/>https://doi.org/10.1146/</a> annurev-ecolsys-102710-145113</li><li>Fuentes-Hurtado, M., Hof, A. R., & Jansson, R. (2016). Paleodistribution modeling suggests glacial refugia and out-of-Tibet range expansion of the Arctic fox. <em>Ecology and Evolution</em>, 6, 170–180.</li><li>Gebauer, A., Eck, S., Kaiser, M., Lei, F. M., & Martens, J. (2006). The Qinghai-Tibet Plateau: Center of evolution for snow sparrows (Montifringilla s. str.) and mountain-steppe sparrows (Onychostruthus, Pyrgilauda). <em>Acta Zoologica Sinica</em>, 52(suppl), 305–309.</li><li>Goldberg, E. E., Roy, K., Lande, R., & Jablonski, D. (2005). Diversity, ende- mism and age distributions in macroevolutionary sources and sinks. <em>American Naturalist</em>, 165, 623–633. <a href=https://doi.org/10.1086/430012>https://doi.org/10.1086/430012</a></li><li>Hauenschild, F., Favre, A., Schnitzler, J., Michalak, I., Freiberg, M., & Muellner-Riehl, A. N. (2017). Spatio-temporal evolution of Allium L. in the Qinghai–Tibet-Plateau region: Immigration and in situ radiation. <em>Plant Diversity</em>, 39, 167–179. <a href=https://doi.org/10.1016/j.pld.2017.05.010>https://doi.org/10.1016/j.pld.2017.05.010</a></li><li>Hoffmann, S., Stöck, M., Zheng, Y., Ficetola, F. G., Li, J. T., Scheidt, U., & Schmidt, J. (2017). Molecular phylogenies indicated a Paleo-Tibetan origin of Himalayan lazy toads (Scutiger). Scientific Reports, 7, 3308. Holbourn, A. E., Kuhnt, W., Clemens, S. C., Kochhann, K. G. D., Jöhnck, J., Lübbers, J., & Andersen, N. (2018). Late Miocene climate cool- ing and intensification of southeast Asian winter monsoon. <em>Nature Communications</em>, 9, 1584. <a href=https://doi.org/10.1038/s41467-018-03950-1>https://doi.org/10.1038/s41467-018-03950-1</a></li><li>Hoorn, C., Perrigo, A., & Antonelli, A. (Eds.) (2018). Mountains, climate, and biodiversity (1st ed.). Hoboken, NJ: Wiley.</li><li>Hosner, P. A., Nyári, A. S., & Moyle, R. G. (2013). Water barriers and intra-island isolation contribute to diversification in the insular Aethopyga sunbirds (Aves: Nectariniidae). <em>Journal of Biogeography</em>, 40, 1094–1106.</li><li>Hui, Z., Li, J., Xu, Q., Song, C., Zhang, J., Wu, F., & Zhao, Z. (2011). Miocene vegetation and climatic changes reconstructed from a sporopollen record of the Tianshui Basin, NE Tibetan Plateau. <em>Paleogeography, Paleoclimatology, Paleoecology</em>, 3–4, 373–382. <a href=https://doi.org/10.1016/j.palaeo.2011.05.043>https://doi.org/10.1016/j.palaeo.2011.05.043</a></li><li>IUCN (2019). <em>The IUCN Red List of Threatened Species</em>. Version 2019–3. Retrieved from <a href=http://www.iucnredlist.org>http://www.iucnredlist.org</a>. Downloaded on 07 January 2020.</li><li>Johansson, U. S., Alström, P., Olsson, U., Ericson, P. G. P., Sundberg, P., & Price, T. D. (2007). Build-up of the Himalayan avifauna through immi- gration: A biogeographical analysis of the Phylloscopus and Seicercus warblers. <em>Evolution</em>, 61, 324–333.</li><li>Johansson, U. S., Fjeldså, J., & Bowie, R. C. K. (2008). Phylogenetic re- lationships within Passerida (Aves: Passeriformes): A review and a new molecular phylogeny based on three nuclear intron markers. <em>Molecular Phylogenetics and Evolution</em>, 48, 858–876. https://doi. org/10.1016/j.ympev.2008.05.029</li><li>Johansson, U. S., Nylinder, S., Ohlson, J. I., & Tietze, D. T. (2018). Reconstruction of the late Miocene biogeographical history of tits and chickadees (Aves: Passeriformes: Paridae): A comparison be- tween discrete area analyses and probabilistic diffusion approach. <em>Journal of Biogeography</em>, 45, 14–25. <a href=https://doi.org/10.1111/jbi.13095>https://doi.org/10.1111/jbi.13095</a></li><li>Jønsson, K. A., & Fjeldsa, J. (2006). A phylogenetic supertree of os- cine passerine birds (Aves: Passeri). <em>Zoologica Scripta</em>, 35, 149–186. <a href=https://doi.org/10.1111/j.1463-6409.2006.00221.x>https://doi.org/10.1111/j.1463-6409.2006.00221.x</a></li><li>Katoh, K., & Standley, D. M. (2013). MAFFT multiple sequence align- ment software version 7: Improvements in performance and us- ability. <em>Molecular Biology and Evolution</em>, 30, 772–780. https://doi. org/10.1093/molbev/mst010</li><li>Lanfear, R., Calcott, B., Ho, S. Y. W., & Guindon, S. (2012). PartitionFinder: Combined selection of partitioning schemes and substitution models for phylogenetic analyses. <em>Molecular Biology and Evolution</em>, 29, 1695– 1701. <a href=https://doi.org/10.1093/molbev/mss020>https://doi.org/10.1093/molbev/mss020</a></li><li>Lei, F., Qu, Y., & Song, G. (2014). Species diversification and phylogeo- graphical patterns of birds in response to the uplift of the Qinghai- Tibet Plateau and Quaternary glaciations. <em>Current Zoology</em>, 60, 149– 161. <a href=https://doi.org/10.1093/czoolo/60.2.149>https://doi.org/10.1093/czoolo/60.2.149</a></li><li>Li, L., Tietze, D. T., Fritz, A., Lü, Z., Bürgi, M., & Storch, I. (2018). Rewilding cultural landscape potentially puts both avian diversity and ende- mism at risk: A Tibetan Plateau case study. <em>Biological Conservation</em>, 224, 75–86. <a href=https://doi.org/10.1016/j.biocon.2018.05.008>https://doi.org/10.1016/j.biocon.2018.05.008</a></li><li>Li, Q., & Wang, X. (2015). Into Tibet: An early Pliocene dispersal of fossil Zokor (Rodentia: Spalacidae) from Mongolian Plateau to the Hinterland of Tibetan Plateau. <em>Plosone</em>, 10, e0144993. https://doi. org/10.1371/journal.pone.0144993</li><li>Lin, G., Ci, H., Zhang, T., & Su, J. (2008). Confirmity to Bergmann’s rule in the plateau pika (Ochotona curzoniae) on the Qinghai-Tibet Plateau. <em>Acta Zoologica Academiae Scientarium Hungaricae</em>, 54, 411–418.</li><li>Liu, B., Alström, P., Olsson, U., Fjeldså, J., Quan, Q., Roselaar, K. C. S., &mldr; Lei, F. (2017). Explosive radiation and spatial expansion across the cold environments of the Old World in an avian family. <em>Ecology and Evolution</em>, 7, 6346–6357. <a href=https://doi.org/10.1002/ece3.3136>https://doi.org/10.1002/ece3.3136</a></li><li>Liu, J. Q., Gao, T. G., Chen, Z. D., & Lu, A. M. (2002). Molecular phylogeny and biogeography of the Qinghai-Tibet Plateau endemic Nannoglottis (Asteraceae). <em>Molecular Phylogenetics and Evolution</em>, 23, 307–325. <a href=https://doi.org/10.1016/S1055-7903%2802%2900039-8>https://doi.org/10.1016/S1055-7903(02)00039-8</a></li><li>Liu, Y., Hu, J., Li, S. H., Duchen, P., Wegmann, D., & Schweizer, M. (2016). Sino-Himalayan mountains act as cradles of diversity and immigra- tion centres in the diversification of parrotbills (Paradoxornithidae). <em>Journal of Biogeography</em>, 43, 1488–1501. <a href=https://doi.org/10.1111/>https://doi.org/10.1111/</a> jbi.12738</li><li>Marchese, C. (2015). Biodiversity hotspots: A shortcut for a more compli- cated concept. <em>Global Ecology and Conservation</em>, 3, 297–309. https:// doi.org/10.1016/j.gecco.2014.12.008</li><li>Martens, J. (2015). Fauna – Himalayan patterns of diversity. In G. Miehe et al (Eds.), <em>Nepal: An introduction to the natural history, ecology and human environment of the Himalayas</em> (pp. 211–249). Edinburgh, UK: Royal Botanic Garden.</li><li>Martens, J., & Eck, S. (Eds.) (2003). <em>Atlas der Verbreitung palaearktischer Vögel</em>, Vol. 20. Erwin-Stresemann-Gesellschaft für paläarktische Avifaunistik e. V., Berlin, Germany.</li><li>Martens, J., & Sun, Y. H. (Eds.) (2008). <em>Atlas der Verbreitung palaearktischer Vögel</em>, Vol. 21. Erwin-Stresemann-Gesellschaft für Paläarktische Avifaunistik e. V, Berlin, Germany.</li><li>Matuszak, S., Muellner-Riehl, A. N., Sun, H., & Favre, A. (2016). Dispersal routes between biodiversity hotspots in Asia: The case of the moun- tain genus Tripterospermum (Gentianinae, Gentianaceae) and its close relatives. <em>Journal of Biogeography</em>, 43, 580–590.</li><li>Matzke, N. J. (2013). <em>BioGeoBEARS: BioGeography with Bayesian (and likelihood) evolutionary analysis in R scripts</em>. R package, version 0.2.1, published July 27, 2013. Retrieved from <a href=http://CRAN.R-project.org/>http://CRAN.R-project.org/</a> package=BioGeoBEARS</li><li>Miao, Y., Wu, F., Warny, S., Fang, X., Lu, H., Fu, B., &mldr; Shi, P. (2019). Miocene fire intensification linked to continuous aridifica- tion on the Tibetan Plateau. <em>Geology</em>, 47, 303–307. https://doi. org/10.1130/G45720.1</li><li>Mora, C., Chittaro, P. M., Sale, P. F., Kritzer, J. P., & Ludsin, S. A. (2003). Patterns and processes in reef fish diversity. <em>Nature</em>, 421, 933–936. <a href=https://doi.org/10.1038/nature01393>https://doi.org/10.1038/nature01393</a></li><li>Mosbrugger, V., Favre, A., Muellner-Riehl, A. N., Päckert, M., & Mulch, A. (2018). Cenozoic evolution of geobiodiversity in the Tibeto- Himalayan Region. In C. Hoorn, A. Perrigo, & A. Antonelli (Eds.), <em>Mountains, climate, and biodiversity</em> (1st ed., pp. 429–448). Oxford, UK: John Wiley & Sons Ltd.</li><li>Moyle, R. G., Oliveros, C. H., Andersen, M. J., Hosner, P. A., Benz, B. W., Manthey, J. D., &mldr; Faircloth, B. C. (2016). Tectonic collision and uplift of Wallacea triggered the global songbird radiation. <em>Nature</em> Communications, 7, 12709. <a href=https://doi.org/10.1038/ncomms12709>https://doi.org/10.1038/ncomms12709</a></li><li>Muellner-Riehl, A. N. (2019). Mountains as evolutionary arenas: Patterns, emerging approaches, paradigm shifts and their implications for plant phylogeographic research in the Tibeto-Himalayan Region. <em>Frontiers in Plant Science</em>, 10, 195. <a href=https://doi.org/10.3389/fpls.2019.00195>https://doi.org/10.3389/fpls.2019.00195</a></li><li>Nie, J., Garzione, C., Su, Q., Liu, Q., Zhang, R., Heslop, D., &mldr; Luo, Z. (2017). Dominant 100,000-year precipitation cyclicity in a late Miocene lake from northeast Tibet. <em>Science Advances</em>, 3, e1600762.</li><li>Nyári, A. S., Peterson, A. T., Rice, N. H., & Moyle, R. G. (2009). Phylogenetic relationships of flowerpeckers (Aves: Dicaeidae): Novel insights into the evolution of a tropical passerine clade. <em>Molecular Phylogenetics and Evolution</em>, 53, 613–619. <a href=https://doi.org/10.1016/j>https://doi.org/10.1016/j</a>. ympev.2009.06.014</li><li>Olson, D. M., Dinerstein, E., Wikramanayake, E. D., Burgess, N. D., Powell, G. V. N., Underwood, E. C., &mldr; Kassem, K. R. (2001). Terrestrial Ecoregions of the World: A New Map of Life on Earth. <em>BioScience</em>,
PÄCKERT ET al.
51, 933–938. <a href=https://doi.org/10.1641/0006-3568%282001%29051[0933:->https://doi.org/10.1641/0006-3568(2001)051[0933:-</a>
TEOTWA]2.0.CO;2</li><li>Ortega-Huerta, M. A., & Vega-Rivera, J. H. (2017). Validating distribution models for twelve endemic bird species of tropical dry forest in Western Mexico. <em>Ecology and Evolution</em>, 7, 7672–7686. https://doi. org/10.1002/ece3.3160</li><li>Päckert, M., Martens, J., Sun, Y. H., Severinghaus, L. L., Nazarenko, A. A., Ting, J., &mldr; Tietze, D. T. (2012). Horizontal and elevational phylogeo- graphic patterns of Himalayan and Southeast Asian forest passerines (Aves: Passeriformes). <em>Journal of Biogeography</em>, 39, 556–573. https:// doi.org/10.1111/j.1365-2699.2011.02606.x</li><li>Päckert, M., Martens, J., Sun, Y.-H., & Strutzenberger, P. (2016). The phy- logenetic relationships of Przevalski’s finch Urocynchramus pylzowi (Aves: Passeriformes, Urocynchramidae) – the most ancient Tibetan endemic passerine known to date. <em>Ibis</em>, 158, 530–540.</li><li>Päckert, M., Martens, J., Sun, Y. H., & Tietze, D. T. (2015). Evolutionary history of passerine birds (Aves: Passeriformes) from the Qinghai- Tibetan Plateau: From a pre-Quarternary perspective to an integra- tive biodiversity assessment. <em>Journal of Ornithology</em>, 156(suppl. 1), 355–365. <a href=https://doi.org/10.1007/s10336-015-1185-6>https://doi.org/10.1007/s10336-015-1185-6</a></li><li>Päckert, M., Sun, Y.-H., Strutzenberger, P., Valchuk, O., Tietze, D. T., & Martens, J. (2015). Phylogenetic relationships of endemic bun- ting species (Aves, Passeriformes, Emberizidae, Emberiza koslowi) from the eastern Qinghai-Tibet Plateau. <em>Vertebrate Zoology</em>, 65, 135–150.</li><li>Paradis, E., Claude, J., & Strimmer, K. (2004). APE: Analyses of phylo- genetics and evolution in R language. <em>Bioinformatics</em>, 20, 289–290. <a href=https://doi.org/10.1093/bioinformatics/btg412>https://doi.org/10.1093/bioinformatics/btg412</a></li><li>Paradis, E., & Schliep, K. (2019). Ape 5.0: An environment for modern phylogenetics and evolutionary analyses in R. <em>Bioinformatics</em>, 35, 526–528. <a href=https://doi.org/10.1093/bioinformatics/bty633>https://doi.org/10.1093/bioinformatics/bty633</a></li><li>Pattengale, N. D., Alipour, M., Bininda-Emonds, O. R., More, B. M., & Stamatakis, A. (2010). How many bootstrap replicates are neces- sary? <em>Journal of Computational Biology</em>, 17, 337–354. https://doi. org/10.1089/cmb.2009.0179</li><li>Pelegrin, J. S., Gamboa, S., Menéndez, I., & Hernández-Fernández, M. (2018). The Great American Biotic Interchange: A paleoecological review considering Neotropical mammals and birds. <em>Ecosistemas</em>, 27, 5–17. (in Spanish with English summary). <a href=https://doi.org/10.7818/>https://doi.org/10.7818/</a> ECOS.1455</li><li>Pirie, M. D., Humphreys, A. M., Antonelli, A., Galley, C., & Linder, H. P. (2012). Model uncertainty in ancestral area reconstruction: A par- simonious solution? <em>Taxon</em>, 61, 652–664. <a href=https://doi.org/10.1002/>https://doi.org/10.1002/</a> tax.613013</li><li>Pisano, J., Condamine, F. L., Lebedev, V., Bannikova, A., Quéré, J. P., Shenbrot, G. I., &mldr; Michaux, J. R. (2016). Out of Himalaya: The impact of past Asian environmental changes on the evolutionary and biogeo- graphical history of Dipodoidea (Rodentia). <em>Journal of Biogeography</em>, 42, 856–870. <a href=https://doi.org/10.1111/jbi.12476>https://doi.org/10.1111/jbi.12476</a></li><li>Price, T. D., Hooper, D. M., Buchanan, C. D., Johansson, U. S., Tietze, D. T., Alström, P., &mldr; Mohan, D. (2014). Niche filling slows the diversi- fication of Himalayan songbirds. <em>Nature</em>, 509, 222–225. https://doi. org/10.1038/nature13272</li><li>Price, T. D., Mohan, D., Tietze, D. T., Hooper, D. M., Orme, C. D. L., & Rasmussen, P. C. (2011). Determinants of northerly range limits along the Himalayan bird diversity gradient. <em>American Naturalist</em>, 178, 97–108. <a href=https://doi.org/10.1086/661926>https://doi.org/10.1086/661926</a></li><li>QGIS Development Team (2014). <em>QGIS Geographic Information System</em>. Open Source Geospatial Foundation Project. Retrieved from http:// qgis.osgeo.org</li><li>Qiu, J. (2014). Origins of Arctic fox traced back to Tibet. <em>Nature News</em>, June, <a href=https://doi.org/10.1038/nature.2014.15398>https://doi.org/10.1038/nature.2014.15398</a></li><li>R Development Core Team (2016). <em>R: A language and environment for sta- tistical computing v.3.3.1</em>. Vienna, Austria: R Foundation for Statistical Computing.</li><li>Rahbek, C., Borregaard, M. K., Antonelli, A., Colwell, R. K., Holt, B. G., Nogues-Bravo, D., &mldr; Fjeldså, J. (2019). Building mountain biodiver- sity: Geological and evolutionary processes. <em>Science</em>, 365, 1114–1119. <a href=https://doi.org/10.1126/science.aax0151>https://doi.org/10.1126/science.aax0151</a></li><li>Rahbek, C., Borregaard, M. K., Colwell, R. K., Dalsgaard, B. O., Holt, B. G., Morueta-Holme, N., &mldr; Fjeldså, J. (2019). Humboldt’s enigma: What causes global patterns of mountain biodiversity? <em>Science</em>, 365, 1108– 1113. <a href=https://doi.org/10.1126/science.aax0149>https://doi.org/10.1126/science.aax0149</a></li><li>Rambaut, A., & Drummond, A. J. (2007). <em>TRACER v1.4</em>. Retrieved from <a href=http://beast.bio.ed.ac.uk/Tracer>http://beast.bio.ed.ac.uk/Tracer</a></li><li>Ramesh, V., Gopalakrishna, T., Barve, S., & Melnick, D. J. (2017). IUCN greatly underestimates threat levels of endemic birds in the Western Ghats. <em>Biological Conservation</em>, 210, 205–221. https://doi. org/10.1016/j.biocon.2017.03.019</li><li>Renner, S. S. (2016). Available data point to a 4 km-high Tibetan pla- teau by 40 Ma, but 100 molecular-clock papers have linked sup- posed recent uplift to young node ages. <em>Journal of Biogeography</em>, 43, 1479–1487.</li><li>Renner, S. C., Rappole, J. H., Milensky, C. M., Aung, M., Shwe, N. M., & Aung, T. (2015). Avifauna of the Southeastern Himalayan Mountains and neighboring Myanmar hill country. <em>Bonn Zoological Bulletin</em> (Suppl), 62, 1–75.</li><li>Schmidt, J., Opgenoorth, L., Höll, S., & Bastrop, R. (2012). Into the Himalayan exile: The phylogeography of the ground beetle Ethira clade supports the Tibetan origin of forest-dwelling Himalayan spe- cies groups. <em>PLoS One</em>, 7, e45482. <a href=https://doi.org/10.1371/journ>https://doi.org/10.1371/journ</a> al.pone.0045482</li><li>Selvatti, A. P., Gonzaga, L. P., & de Morales Russo, C. A. (2015). A Paleogene origin for crown passerines and the diversification of the Oscines in the New World. <em>Molecular Phylogenetics and Evolution</em>, 88, 1–15. <a href=https://doi.org/10.1016/j.ympev.2015.03.018>https://doi.org/10.1016/j.ympev.2015.03.018</a></li><li>Shang, H. Y., Li, Z. H., Dong, M., Adams, R. P., Miehe, G., Opgenorth, L., & Shao, K. S. (2015). Evolutionary origin and demographic history of an ancient conifer (Juniperus microsperma) in the Qinghai-Tibetan Plateau. <em>Scientific Reports</em>, 5, 10216. <a href=https://doi.org/10.1038/srep10216>https://doi.org/10.1038/srep10216</a></li><li>Stamatakis, A. (2014). RAxML Version 8: A tool for phylogenetic analy- sis and post-analysis of large phylogenies. <em>Bioinformatics</em>, 30, 1312– 1313. <a href=https://doi.org/10.1093/bioinformatics/btu033>https://doi.org/10.1093/bioinformatics/btu033</a></li><li>Stresemann, E., & Portenko, L. A. (1960). <em>Atlas der Verbreitung palaeark- tischer Vögel – 1. Lieferung</em>. Berlin, Germany: Akademie-Verlag.</li><li>Stresemann, E., & Portenko, L. A. (1976). <em>Atlas der Verbreitung palaeark- tischer Vögel – 5. Lieferung</em>. Berlin, Germany: Akademie-Verlag.</li><li>Stresemann, E., & Portenko, L. A. (1977). <em>Atlas der Verbreitung palaeark- tischer Vögel – 6. Lieferung</em>. Berlin, Germany: Akademie-Verlag.</li><li>Stresemann, E., & Portenko, L. A. (1978). <em>Atlas der Verbreitung palaeark- tischer Vögel – 7. Lieferung</em>. Berlin, Germany: Akademie-Verlag.</li><li>Stresemann, E., & Portenko, L. A. (1980). <em>Atlas der Verbreitung palaeark- tischer Vögel – 8. Lieferung</em>. Berlin, Germany: Akademie-Verlag.</li><li>Stresemann, E., & Portenko, L. A. (1981). <em>Atlas der Verbreitung palaeark- tischer Vögel – 9. Lieferung</em>. Berlin, Germany: Akademie-Verlag.</li><li>Stresemann, E., Portenko, L. A., Dathe, H., & Mauersberger, G. (1974). <em>Atlas der Verbreitung palaearktischer Vögel – 4. Lieferung</em>. Berlin, Germany: Akademie-Verlag.</li><li>Stresemann, E., Portenko, L. A., & Mauersberger, G. (1971). <em>Atlas der Verbreitung palaearktischer Vögel – 3. Lieferung</em>. Berlin, Germany: Akademie-Verlag.</li><li>Tietze, D. T., Päckert, M., Martens, J., Lehmann, H., & Sun, Y.-H. (2013). Complete phylogeny and historical biogeography of true rosefinches (Aves: Carpodacus). <em>Zoological Journal of the Linnean Society</em>, 169, 215–234.</li><li>Töpfer, T., Haring, E., Birkhead, T. R., Lopes, R. J., Liu Severinghaus, L., Martens, J., & Päckert, M. (2011). A molecular phylogeny of bullfinches Pyrrhula Brisson, 1760 (Aves: Fringillidae). <em>Molecular Phylogenetics and Evolution</em>, 58, 271–282. <a href=https://doi.org/10.1016/j>https://doi.org/10.1016/j</a>. ympev.2010.10.015</li><li>Tseng, Z. J., Li, Q., & Wang, X. (2013). A new cursorial hyena from Tibet, and analysis of biostratigraphy, paleozoography, and dental morphol- ogy of Chasmaporthetes (Mammalia, Carnivora). <em>Journal of Vertebrate Paleontology</em>, 33, 1457–1471.</li><li>Tseng, Z. J., Wang, X., Slater, G. J., Takeuchi, G. T., Li, Q., Liu, J., & Xie, G. (2014). Himalayan fossils of the oldest known pantherine establish ancient origin of big cats. <em>Proceedings of the Royal Society B: Biological Sciences</em>, 281, 20132686. <a href=https://doi.org/10.1098/rspb.2013.2686>https://doi.org/10.1098/rspb.2013.2686</a></li><li>Voelker, G., Semenov, G., Fadeev, I. V., Blick, A., & Drovetski, S. V. (2015). The biogeographic history of Phoenicurus redstarts reveals an allo- patric mode of speciation and an out-of-Himalayas colonization pat- tern. <em>Systematics and Biology</em>, 13, 296–305.</li><li>Wang, L., Lü, H. Y., Wu, N. Q., Li, J., Pei, Y. P., Tong, G. B., & Peng, S. Z. (2006). Palynological evidence for late Miocene-Pliocene vegetation evolution recorded in the red clay sequence of the central Chinese Loess Plateau and implication for palaeoenvironmental change. <em>Palaeogeography, Palaeoclimatology, Palaeoecology</em>, 241, 118–128. <a href=https://doi.org/10.1016/j.palaeo.2006.06.012>https://doi.org/10.1016/j.palaeo.2006.06.012</a></li><li>Wang, X., Li, Q., & Takeuchi, G. G. (2016). Out of Tibet: An early sheep from the Pliocene of Tibet, Protovis himalayensis, genus and species nov. (Bovidae, Caprini), and origin of Ice Age mountain sheep. <em>Journal of Vertebrate Paleontology</em>, 36, e1169190.</li><li>Wang, X., Tseng, Z. J., Li, Q., Takeuchi, G. T., & Xie, G. P. (2014). From ‘third pole’ to north pole: A Himalayan origin for the arctic fox. <em>Proceedings of the Royal Society B</em>, 281, 20140893. <a href=https://doi.org/10.1098/>https://doi.org/10.1098/</a> rspb.2014.0893</li><li>Wang, X., Wang, Y., Li, Q., Tseng, Z. J., Takeuchi, G. T., Deng, T., &mldr; Wang, N. (2015). Cenozoic vertebrate evolution and paleoenvironment in Tibetan Plateau: Progress and prospects. <em>Gondwana Research</em>, 27, 1335–1354. <a href=https://doi.org/10.1016/j.gr.2014.10.014>https://doi.org/10.1016/j.gr.2014.10.014</a></li><li>Weigold, H. (1935). <em>Südost-Tibet als Lebensraum</em>. Jahrbuch Der Geographischen Gesellschaft Hannover, 35, 203–247.</li><li>Weigold, H. (1949). Tibet einst ein Entwicklungszentrum. In E. Mayr, & Schüz (Eds.), <em>Ornithologie als biologische Wissenschaft</em> (pp. 92–107). Heidelberg, Germany: Stresemann-Festschrift.</li><li>Weigold, H. (2005). Die Biogeographie Tibets und seiner Vorländer. <em>Mitteilungen des Vereins Sächsischer Ornithologen</em>, 9, Sonderheft 3, 445 p., Hohenstein-Ernstthal.</li><li>Weir, J. T., Bermingham, E., & Schluter, D. (2009). The Great American Biotic Interchange in birds. <em>Proceedings of the National Academy of Sciences of the USA</em>, 106, 21737–21742. <a href=https://doi.org/10.1073/>https://doi.org/10.1073/</a> pnas.0903811106</li><li>Wen, J., Zhang, J.-Q., Nie, Z.-L., Zhong, Y., & Sun, H. (2014). Evolutionary diversifications of plants on the Qinghai-Tibetan Plateau. <em>Frontiers in Genetics</em>, 5, 4. <a href=https://doi.org/10.3389/fgene.2014.00004>https://doi.org/10.3389/fgene.2014.00004</a></li><li>Woodburne, M. O. (2010). The Great American Biotic interchange: Dispersals, tectonics, climate, sea level and holding pens. <em>Journal of Mammalian Evolution</em>, 17, 245–264. <a href=https://doi.org/10.1007/s1091>https://doi.org/10.1007/s1091</a> 4-010-9144-8</li><li>Wu, H. C., Lin, R. C., Hung, H. Y., Yeh, C. F., Chu, J. H., Yang, X. J., &mldr; Lei, F. M. (2011). Molecular and morphological evidences reveal a cryptic species in the Vinaceous Rosefinch Carpodacus vinaceus (Fringillidae; Aves). <em>Zoologica Scripta</em>, 40, 468–478. https://doi. org/10.1111/j.1463-6409.2011.00487.x</li><li>Yang, S., Dong, H., & Lei, F. (2009). Phylogeography of regional fauna on the Tibetan Plateau: A review. <em>Progress in Natural Science</em>, 19, 789– 799. <a href=https://doi.org/10.1016/j.pnsc.2008.10.006>https://doi.org/10.1016/j.pnsc.2008.10.006</a></li><li>Yao, T., Thompson, L. G., Mosbrugger, V., Zhang, F., Ma, Y., Luo, T., &mldr; Fayziev, R. (2012). Third pole environment (TPE). <em>Environmental Development</em>, 3, 52–64. <a href=https://doi.org/10.1016/j.envdev.2012.04.002>https://doi.org/10.1016/j.envdev.2012.04.002</a></li><li>Zhang, P., Chen, Y. Q., Zhou, H., Liu, Y. F., Wang, X. L., Papenfuss, T. J., &mldr; Qu, L. H. (2006). Phylogeny, evolution, and biogeography of Asiatic salamanders (Hynobiidae). <em>Proceedings of the National Academy of Sciences of the USA</em>, 103, 7360–7365. <a href=https://doi.org/10.1073/>https://doi.org/10.1073/</a> pnas.0602325103</li><li>Zhang, T., Nevo, E., Tang, L., Su, J., & Lin, G. (2012). Plateau zokors on the Tibetan Plateau follow Bergmann’s rule latitudinally, but not altitudi- nally. <em>Mammalian Biology</em>, 77, 108–112.</li><li>Zuccon, D., Prŷs-Jones, R., Rasmussen, P. C., & Ericson, P. (2012). The phylogenetic relationships and generic limits of finches (Fringillidae). <em>Molecular Phylogenetics and Evolution</em>, 62, 581–596. https://doi. org/10.1016/j.ympev.2011.10.002</li></ul><h2 id=supporting-information>SUPPORTING INFORMATION
<a class=anchor href=#supporting-information>#</a></h2><p>Additional supporting information may be found online in the Supporting Information section.</p><p>How to cite this article: Päckert M, Favre A, Schnitzler J, et al. “Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds. Ecol Evol. 2020;10:9283–9300. <a href=https://doi.org/10.1002/ece3.6615>https://doi.org/10.1002/ece3.6615</a></p><h3 id=citing-literature>Citing Literature
<a class=anchor href=#citing-literature>#</a></h3><p>Volume10, Issue17
September 2020
Pages 9283-9300
Citation Statements
beta
Supporting
0
Mentioning
25
Contrasting
2
Explore this article&rsquo;s citation statements on scite.ai
powered by<br>Scite
Figures
References
Related
Information</p><h3 id=recommended>Recommended
<a class=anchor href=#recommended>#</a></h3><p>Phylogeography of the Qinghai–Tibet Plateau endemic alpine herb Pomatosace filicula (Primulaceae)
Gai-Ni WANG, Xin-Yu HE, Georg MIEHE, Kang-Shan MAO
Journal of Systematics and Evolution
In and out of the Qinghai‐Tibet Plateau: divergence time estimation and historical biogeography of the large arctic‐alpine genus Saxifraga L.</p><p>J. Ebersbach, A. N. Muellner-Riehl, I. Michalak, N. Tkach, M. H. Hoffmann, M. Röser, H. Sun, A. Favre
Journal of Biogeography
The uplift of the Qinghai–Tibet Plateau and glacial oscillations triggered the diversification of Tetraogallus (Galliformes, Phasianidae)</p><p>Li Ding, Jicheng Liao, Naifa Liu
Ecology and Evolution
Biogeography and diversification of Old World buntings (Aves: Emberizidae): radiation in open habitats</p><p>Tianlong Cai, Guiyou Wu, Lu Sun, Yu Zhang, Zhaojie Peng, Yanqing Guo, Xinyue Liu, Tao Pan, Jiang Chang, Zhonglou Sun, Baowei Zhang
Journal of Avian Biology
The role of the uplift of the Qinghai‐Tibetan Plateau for the evolution of Tibetan biotas</p><p>Adrien Favre, Martin Päckert, Steffen U. Pauls, Sonja C. Jähnig, Dieter Uhl, Ingo Michalak, Alexandra N. Muellner-Riehl
Biological Reviews</p><p>Download PDF
Back
Additional links
About Wiley Online Library</p><p>Privacy Policy
Terms of Use
Cookies
Accessibility
Publishing Policies
Help & Support</p><h3 id=contact-us>Contact Us
<a class=anchor href=#contact-us>#</a></h3><p>Training and Support
DMCA & Reporting Piracy
Opportunities</p><h3 id=subscription-agents>Subscription Agents
<a class=anchor href=#subscription-agents>#</a></h3><p>Advertisers & Corporate Partners
Connect with Wiley</p><h3 id=the-wiley-network>The Wiley Network
<a class=anchor href=#the-wiley-network>#</a></h3><p>Wiley Press Room
Copyright © 1999-2021 John Wiley & Sons, Inc. All rights reserved
Wiley Home Page</p><div class=footnotes role=doc-endnotes><hr><ol><li id=fn:1><p>Senckenberg Natural History Collections, Museum of Zoology, Dresden, Germany&#160;<a href=#fnref:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref1:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:2><p>Entomology III, Senckenberg Research Institute and Natural History Museum Frankfurt, Frankfurt am Main, Germany&#160;<a href=#fnref:2 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:3><p>German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, Leipzig, Germany&#160;<a href=#fnref:3 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:4><p>Department of Molecular Evolution and Plant Systematics & Herbarium (LZ), Institute of Biology, Leipzig University, Leipzig, Germany&#160;<a href=#fnref:4 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a>&#160;<a href=#fnref1:4 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:5><p>Institute of Organismic and Molecular Evolution, Johannes Gutenberg-Universität, Mainz, Germany&#160;<a href=#fnref:5 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:6><p>Key Laboratory of Animal Ecology and Conservation, Institute of Zoology, Chinese Academy of Sciences, Beijing, China&#160;<a href=#fnref:6 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:7><p>Natural History Museum Basel, Basel, Switzerland&#160;<a href=#fnref:7 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:8><p>Centrum für Naturkunde, Universität Hamburg, Hamburg, Germany&#160;<a href=#fnref:8 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:9><p>School of Biosciences, Cardiff University, Cardiff, UK&#160;<a href=#fnref:9 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:10><p>Senckenberg Biodiversity and Climate Research Centre, Frankfurt am Main, Germany&#160;<a href=#fnref:10 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:11><p>Department of Botany and Biodiversity Research, Universität Wien, Wien, Austria&#160;<a href=#fnref:11 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li></ol></div></article><footer class=book-footer><div class="flex flex-wrap justify-between"></div><script>(function(){function e(e){const t=window.getSelection(),n=document.createRange();n.selectNodeContents(e),t.removeAllRanges(),t.addRange(n)}document.querySelectorAll("pre code").forEach(t=>{t.addEventListener("click",function(){e(t.parentElement),navigator.clipboard&&navigator.clipboard.writeText(t.parentElement.textContent)})})})()</script></footer><div class=book-comments></div><label for=menu-control class="hidden book-menu-overlay"></label></div><aside class=book-toc><div class=book-toc-content><nav id=TableOfContents><ul><li><a href=#correspondence>Correspondence</a></li><li><a href=#funding-information>Funding information</a></li><li><a href=#abstract>Abstract</a></li><li><a href=#keywords>KEYWORDS</a></li><li><a href=#1-introduction>1 INTRODUCTION</a></li><li><a href=#2-materials-and-methods>2 MATERIALS AND METHODS</a><ul><li><a href=#21-sampling-strategy>2.1 Sampling strategy</a></li><li><a href=#22-dna-extraction-pcr-and-sequencing>2.2 DNA extraction, PCR, and sequencing</a></li><li><a href=#23-data-assembly>2.3 Data assembly</a></li><li><a href=#24-phylogenetic-reconstruction-and-divergence-time-estimation>2.4 Phylogenetic reconstruction and divergence-time estimation</a></li><li><a href=#25-species-distributions>2.5 Species distributions</a></li><li><a href=#26-ancestral-range-reconstruction-arr>2.6 Ancestral-range reconstruction (ARR)</a></li></ul></li><li><a href=#3-results>3 RESULTS</a><ul><li><a href=#31-phylogeny-and-origin-of-passeroidea>3.1 Phylogeny and origin of Passeroidea</a></li><li><a href=#32-biogeographical-history-of-montane-and-alpine-target-groups-of-passeroidea>3.2 Biogeographical history of montane and alpine target groups of Passeroidea</a></li></ul></li><li><a href=#4-discussion>4 DISCUSSION</a><ul><li><a href=#41-phylogeny-and-biogeographic-history-of-passeroidea>4.1 Phylogeny and biogeographic history of Passeroidea</a></li><li><a href=#42-out-of-qtpin-situ-diversification-on-the-qtp-and-along-its-forested-fringes>4.2 “Out of QTP”—in situ diversification on the QTP and along its forested fringes</a></li><li><a href=#43-faunal-exchange-between-the-plateau-region-and-the-qtp-fringes>4.3 Faunal exchange between the plateau region and the QTP fringes</a></li><li><a href=#44-immigration-into-the-qtp-region-from-adjacent-source-regions>4.4 Immigration into the QTP region from adjacent source regions</a></li></ul></li><li><a href=#5-conclusion>5 CONCLUSION</a></li><li><a href=#acknowledgments>ACKNOWLEDGMENTS</a></li><li><a href=#conflict-of-interest>CONFLICT OF INTEREST</a></li><li><a href=#author-contributions>AUTHOR CONTRIBUTIONS</a></li><li><a href=#data-availability-statement>DATA AVAILABILITY STATEMENT</a></li><li><a href=#orcid>ORCID</a></li><li><a href=#references>REFERENCES</a></li><li><a href=#supporting-information>SUPPORTING INFORMATION</a><ul><li><a href=#citing-literature>Citing Literature</a></li><li><a href=#recommended>Recommended</a></li><li><a href=#contact-us>Contact Us</a></li><li><a href=#subscription-agents>Subscription Agents</a></li><li><a href=#the-wiley-network>The Wiley Network</a></li></ul></li></ul></nav></div></aside></main></body></html>