<!doctype html><html lang=en dir=ltr><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta name=description content="From the Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02115
Received for publication, April 6, 2001 Published, JBC Papers in Press, June 6, 2001, DOI 10.1074/jbc.M103078200
THE JOURNAL OF BIOLOGICAL CHEMISTRY Vol. 276, No. 31, Issue of August 3, pp. 29067–29071, 2001 © 2001 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in U.S.A.
Skip to main contentSkip to article Elsevier logo Journals & Books Register Sign in View PDF Download full issue Outline MATERIALS AND METHODS RESULTS DISCUSSION Acknowledgments REFERENCES Figures (8) Figure 1."><meta name=theme-color content="#FFFFFF"><meta name=color-scheme content="light dark"><meta property="og:title" content="Architecture of the Human Origin Recognition Complex*"><meta property="og:description" content="From the Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02115
Received for publication, April 6, 2001 Published, JBC Papers in Press, June 6, 2001, DOI 10.1074/jbc.M103078200
THE JOURNAL OF BIOLOGICAL CHEMISTRY Vol. 276, No. 31, Issue of August 3, pp. 29067–29071, 2001 © 2001 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in U.S.A.
Skip to main contentSkip to article Elsevier logo Journals & Books Register Sign in View PDF Download full issue Outline MATERIALS AND METHODS RESULTS DISCUSSION Acknowledgments REFERENCES Figures (8) Figure 1."><meta property="og:type" content="article"><meta property="og:url" content="https://csmeyns.github.io/derivatives/docs/architecture-human-orc/"><meta property="article:section" content="docs"><title>Architecture of the Human Origin Recognition Complex* | Derivatives</title><link rel=manifest href=/derivatives/manifest.json><link rel=icon href=/derivatives/favicon.png type=image/x-icon><link rel=stylesheet href=/derivatives/book.min.1a6fac2df464e68d664cb846d822c678cd622692d6fae24ee18901a08bfc8eb6.css integrity="sha256-Gm+sLfRk5o1mTLhG2CLGeM1iJpLW+uJO4YkBoIv8jrY=" crossorigin=anonymous><script defer src=/derivatives/flexsearch.min.js></script>
<script defer src=/derivatives/en.search.min.e158fcdf95a494f0773e12e8c8c913be63f3f9aac72b50e12ace0ee79181c1eb.js integrity="sha256-4Vj835WklPB3PhLoyMkTvmPz+arHK1DhKs4O55GBwes=" crossorigin=anonymous></script></head><body dir=ltr><input type=checkbox class="hidden toggle" id=menu-control>
<input type=checkbox class="hidden toggle" id=toc-control><main class="container flex"><aside class=book-menu><div class=book-menu-content><nav><h2 class=book-brand><a class="flex align-center" href=/derivatives/><span>Derivatives</span></a></h2><div class=book-search><input type=text id=book-search-input placeholder=Search aria-label=Search maxlength=64 data-hotkeys=s/><div class="book-search-spinner hidden"></div><ul id=book-search-results></ul></div><ul><li><a href=https://csmeyns.github.io/derivatives/docs/readme/>README</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/architecture-human-orc/ class=active>Architecture of the Human Origin Recognition Complex*</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/amyloid-%CE%B2/>Origins of amyloid-β</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/arsenic/>Effects of arsenic and heavy metals on metabolic pathways in cells of human origin: Similarities and differences</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/back-to-future/>Back to the future: origins and directions of the “Agile Manifesto” – views of the originators</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/biopolymers/>Folding, Assembly, and Persistence: The Essential Nature and Origins of Biopolymers</a></li><li><input type=checkbox id=section-4e626a39f813bdfb0b0f11621a8d97e6 class=toggle>
<label for=section-4e626a39f813bdfb0b0f11621a8d97e6 class="flex justify-between"><a href=https://csmeyns.github.io/derivatives/docs/cc-by/>CC-BY</a></label><ul><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-2-summary/>Creative Commons License Deed Attribution 2.0 Generic (CC BY 2.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-2/>Creative Commons Legal Code Attribution 2.0</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-3-summary/>Creative Commons License Deed Attribution 3.0 Unported (CC BY 3.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-3/>Creative Commons Attribution 3.0 Unported</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-4-summary/>Creative Commons License Deed Attribution 4.0 International (CC BY 4.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-4-de/>Creative Commons Konzessionsurkunde Namensnennung 4.0 International (CC BY 4.0)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/cc-by/cc-by-4/>Creative Commons Legal Code Attribution 4.0 International</a></li></ul></li><li><a href=https://csmeyns.github.io/derivatives/docs/charon-smooth-plains/>The nature and origin of Charon's smooth plains</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/chinese-word-cell/>Origin of the Chinese word for “cell”: an unusual but wonderful idea of a mathematician</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/city-origins/>City origins</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/counterfeit-versus-original/>Counterfeit versus original patronage: Do emotional brand attachment, brand involvement, and past experience matter?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/creatine/>ON THE ORIGIN OF CREATINE</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/detecting-moisture/>Detecting the origins of moisture over Southeast China: Seasonal variation and heavy Rainfall</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/differentiation-spirits/>Differentiation Between Spirits According to Their Botanical Origin</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/food-taboos/>Food taboos: their origins and purposes</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/ganges/>Ganges: special at its origin</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/given-that-detailed/>Given that the detailed original criteria for deliberate practice have not changed, could the understanding of this complex concept have improved over time? A response to Macnamara and Hambrick (2020)</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/hanwoo-cattle/>Hanwoo cattle: origin, domestication, breeding strategies and genomic selection</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/hoarding/>The developmental origins of hoarding disorder in adolescence: a longitudinal clinical interview study following an epidemiological survey</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/how-to-write/>How to write an original article for the *Journal of Orthopaedics and Traumatology*</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/in-out-qinghai-tibet-plateau/>“Into and Out of” the Qinghai-Tibet Plateau and the Himalayas: Centers of origin and diversification across five clades of Eurasian montane and alpine passerine birds</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/interview-patrick-o-brown/>An interview with Patrick O Brown on the origins and future of open access</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/manipulated-photos/>Can people identify original and manipulated photos of real-world scenes?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/neighborhood-spam/>Neighborhoods and bands: an analysis of the origins of spam</a></li><li><input type=checkbox id=section-18bd70f4fad61eb2742d114842aa84cf class=toggle>
<label for=section-18bd70f4fad61eb2742d114842aa84cf class="flex justify-between"><a href=https://csmeyns.github.io/derivatives/docs/letters/>Letters</a></label><ul><li><a href=https://csmeyns.github.io/derivatives/docs/letters/non-neanderthal/>Non-Neanderthal Origin of the HLA-DPB1*0401</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/letters/reply-to-ding/>Reply to Ding et al.: Non-Neanderthal Origin of the HLA-DPB1*0401</a></li></ul></li><li><a href=https://csmeyns.github.io/derivatives/docs/origin-of-the-ectopic-beat/>What is the origin of the ectopic beat?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/origins/>Origins</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/patient-safety/>The 5th anniversary of “Patient Safety in Surgery” – from the Journal’s origin to its future vision</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/rendang/>Tracing the origins of rendang and its development</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/soft-robot/>Assessing the Impact of an Original Soft Communicative Robot in a Nursing Home in Japan: Will Softness or Conversations Bring more Smiles to Older People?</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/softwaretest-originaldaten/>Softwaretest mit Originaldaten Eine Analyse aus Sicht des Datenschutzes</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/spectacular-submarine/>Origin of spectacular fields of submarine sediment waves around volcanic islands</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/urban-conservation/>Urban Conservation Through Chinese Local Original Architecture: The Preservation and Renovation of Zhongshan Road in Hangzhou</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/vertebrate-gills/>The Origin of Vertebrate Gills</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/volcanic-origin-1741/>Volcanic origin of the 1741 Oshima-Oshima tsunami in the Japan Sea</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/what-is-morita-therapy/>What is Morita Therapy? The Nature, Origins, and Cross-Cultural Application of a Unique Japanese Psychotherapy</a></li><li><a href=https://csmeyns.github.io/derivatives/docs/wildcat-reappearance/>Revealing the origin of wildcat reappearance after presumed long-term absence</a></li></ul></nav><script>(function(){var e=document.querySelector("aside .book-menu-content");addEventListener("beforeunload",function(){localStorage.setItem("menu.scrollTop",e.scrollTop)}),e.scrollTop=localStorage.getItem("menu.scrollTop")})()</script></div></aside><div class=book-page><header class=book-header><div class="flex align-center justify-between"><label for=menu-control><img src=/derivatives/svg/menu.svg class=book-icon alt=Menu></label>
<strong>Architecture of the Human Origin Recognition Complex*</strong>
<label for=toc-control><img src=/derivatives/svg/toc.svg class=book-icon alt="Table of Contents"></label></div><aside class="hidden clearfix"><nav id=TableOfContents><ul><li><a href=#materials-and-methods>MATERIALS AND METHODS</a><ul><li><a href=#plasmid-constructions>Plasmid Constructions</a></li><li><a href=#expression-of-orc-subunits-in-insect-cells-purification-and-gel-filtration>Expression of ORC Subunits in Insect Cells, Purification, and Gel Filtration</a></li><li><a href=#cell-culture-transfection-immunoblotting-immunoprecipitation-and-silver-stain>Cell Culture, Transfection, Immunoblotting, Immunoprecipitation, and Silver Stain</a></li><li><a href=#in-vitro-transcription-and-translation-reactions-and-gst-pull-down-assay>In Vitro Transcription and Translation Reactions and GST Pull-down Assay</a></li><li><a href=#facs-analysis>FACS Analysis</a></li></ul></li><li><a href=#results>RESULTS</a><ul><li><a href=#gstorc5--2--3-and--4-forms-a-complex>GSTORC5, -2, -3, and -4 Forms a Complex</a></li><li><a href=#orc2-and-orc3-physically-interact-with-each-other>ORC2 and ORC3 Physically Interact with Each Other</a></li><li><a href=#orc2-and-orc3-recruit-orc4-and--5>ORC2 and ORC3 Recruit ORC4 and -5</a></li><li><a href=#n-terminal-portion-of-orc3-interacts-with-the-c-terminal-portion-of-orc2>N-terminal Portion of ORC3 Interacts with the C-terminal Portion of ORC2</a></li><li><a href=#n-terminal-200-amino-acids-of-orc3-can-compete-with-the-full-length-orc3>N-terminal 200 Amino Acids of ORC3 Can Compete with the Full-length ORC3</a></li><li><a href=#orc3n-cannot-form-a-complex-that-contains-orc4-5>ORC3N Cannot Form a Complex That Contains ORC4-5</a></li><li><a href=#expression-of-orc3n-in-u2os-cells-causes-cell-cycle-arrest>Expression of ORC3N in U2OS Cells Causes Cell Cycle Arrest</a></li></ul></li><li><a href=#discussion>DISCUSSION</a></li><li><a href=#acknowledgments>Acknowledgments</a></li><li><a href=#references>REFERENCES</a></li><li><a href=#recommended-articles>Recommended articles</a></li><li><a href=#citing-articles-87>Citing articles (87)</a></li><li><a href=#privacy-policy>Privacy policy</a></li></ul></nav></aside></header><article class=markdown><h1><a href=/derivatives/docs/architecture-human-orc/>Architecture of the Human Origin Recognition Complex*</a></h1><h2>Suman Kumar Dhar‡ Laurie Delmolino§ Anindya Dutta¶</h2><p><em>From the Department of Pathology, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02115</em></p><p>Received for publication, April 6, 2001 Published, JBC Papers in Press, June 6, 2001, DOI 10.1074/jbc.M103078200</p><p>THE JOURNAL OF BIOLOGICAL CHEMISTRY Vol. 276, No. 31, Issue of August 3, pp. 29067–29071, 2001
© 2001 by The American Society for Biochemistry and Molecular Biology, Inc.
<em>Printed in U.S.A.</em></p><link href="https://fonts.googleapis.com/css?family=Rubik+Iso" rel=stylesheet type=text/css><div style="font-family:rubik iso;font-size:9pt;font-style:normal;color:normal;font-weight:400">Skip to main contentSkip to article
Elsevier logo
Journals & Books
Register
Sign in
View 
PDF
Download full issue
Outline
MATERIALS AND METHODS
RESULTS
DISCUSSION
Acknowledgments
REFERENCES
Figures (8)
Figure 1. Human ORC2–5 co-purify in a complex
Figure 2. Direct interaction between ORC2 and ORC3 subunits
Figure 3. ORC2-3 subcomplex can load ORC5 but not ORC4
Figure 4. Mapping domains of interactions between ORC2 and ORC3
Figure 5. ORC3N can compete with full-length ORC3
Figure 6. ORC3N does not form a complex with ORC2, -4, and -5
Show all figures
Journal of Biological Chemistry
Volume 276, Issue 31, August 2001, Pages 29067-29071
Journal home page for Journal of Biological Chemistry
DNA: REPLICATION REPAIR AND RECOMBINATION</div><p>Author links
<link href="https://fonts.googleapis.com/css?family=Rubik+Iso" rel=stylesheet type=text/css><div style="font-family:rubik iso;font-size:9pt;font-style:normal;color:normal;font-weight:400">open overlay panel</div>Suman Kumar Dhar<sup id=fnref:1><a href=#fn:1 class=footnote-ref role=doc-noteref>1</a></sup> Laurie Delmolino<sup id=fnref:2><a href=#fn:2 class=footnote-ref role=doc-noteref>2</a></sup> Anindya Dutta<sup id=fnref:3><a href=#fn:3 class=footnote-ref role=doc-noteref>3</a></sup></p><p>Show more
Share
Cite
<a href=https://doi.org/10.1074/jbc.M103078200>https://doi.org/10.1074/jbc.M103078200Get</a> rights and content
Under a Creative Commons licenseopen access</p><div class=abstract><p>All the human homologs of the six subunits of <em>Saccharomyces cerevisiae</em> origin recognition complex have been reported so far. However, not much has been reported on the nature and the characteristics of the human origin recognition complex. In an attempt to purify recombinant human ORC from insect cells infected with baculoviruses expressing HsORC subunits, we found that human ORC2, -3, -4, and -5 form a core complex. HsORC1 and HsORC6 subunits did not enter into this core complex, suggesting that the interaction of these two subunits with the core ORC2–5 complex is extremely labile. We found that the C-terminal region of ORC2 interacts directly with the N-terminal region of ORC3. The C-terminal region of ORC3 was, however, necessary to bring ORC4 and ORC5 into the core complex. A fragment containing the N-terminal 200 residues of ORC3 (ORC3N) competitively inhibited the ORC2-ORC3 interaction. Overexpression of this fragment in U2OS cells blocked the cells in G1, providing the first evidence that a mammalian ORC subunit is important for the G1-S transition in mammalian cells.</p></div><p>Previous article in issueNext article in issue</p><dl><dt>ORC</dt><dd>origin recognition complex</dd><dt>GST</dt><dd>glutathione S-transferase</dd><dt>FACS</dt><dd>fluorescence-activated cell sorter</dd><dt>GFP</dt><dd>green fluorescent protein</dd></dl><p>This is an open access article under the <a href=http://creativecommons.org/licenses/by/4.0/>CC BY</a> license.</p><p>Origin recognition complex (ORC)<sup id=fnref:4><a href=#fn:4 class=footnote-ref role=doc-noteref>4</a></sup> was first described in yeast <em>Saccharomyces cerevisiae</em> (1). All six subunits, essential for cell viability, collectively bind to the ARS (autonomously replicating sequence) consensus sequence in a sequence-specific manner and lead to the chromatin loading of other replication factors like CDC6 and MCM (mini-chromosome maintenance) that are essential for initiation of DNA replication (2, 3). Similar six protein complexes have been discovered in <em>Xenopus laevis</em>(4), <em>Drosophila melanogaster</em> (5) and <em>Schizosaccharomyces pombe</em> (6), although a consensus DNA sequence that serves as an origin of replication and where ORC may bind has not been found in these species. Conservation of similar ORC subunits in mammals suggests that ORC has an equally important role in mammalian cells.</p><p>Although all six human homologs of yeast <em>S. cerevisiae</em> ORC subunits have been reported (7, 8, 9, 10, 11, 12, 13, 14), purification of a six-protein human origin recognition complex remains elusive. Endogenous ORC2, -3, -4, and -5 subunits have been reported to interact with each other in extracts of cancer cell lines (14). ORC1 and ORC6 did not interact with other ORC subunits under these experimental conditions (14). Therefore it is possible that a functional human ORC exists only during a very short period of the cell cycle or in a specific sub-nuclear compartment, making it difficult to extract such a complex from human cell lines. In fact, in a recent study hamster ORC1 was reported to be easily eluted from chromatin during mitosis and early G1phase (15). It became stably bound to chromatin again during mid-G1 phase with the appearance of a functional pre-replication complex at a hamster replication origin. In contrast, ORC2 was stably bound to chromatin throughout the cell cycle. Difficulties in obtaining six protein human ORCs may also be attributed to the fact that we are still missing some of the unidentified important components of the human ORC. Indeed, immunoprecipitation from [<sup>35</sup>S]methionine-labeled HeLa cell lysate of ORC1, -2, -3, -4, and -6 showed many non-ORC proteins interacting specifically with the respective ORC subunits (14, 16).</p><p>A six protein ORC has been purified from <em>Drosophila</em> embryo extracts and possesses some demonstrated biochemical activities (5, 17, 18). All six <em>Drosophila</em> ORC subunits were expressed and subsequently purified to homogeneity from baculovirus-infected insect cells (17). Using an <em>in vitro</em> transcription translation reaction, a similar six-protein ORC has been reported in yeast <em>S. pombe</em> (6). With all the six human ORC subunits in our hand, we attempted to produce recombinant human ORC from the baculovirus expression system in order to dissect the activities and architecture of human origin recognition complex/subcomplex(s).</p><p>Because genetic experiments are difficult to perform in mammalian systems, the human ORC subunits have not been shown to have a role in replication or cell proliferation. Utilizing knowledge learned about the architecture of the human ORC, we created a dominant negative ORC subunit designed to disrupt the formation of endogenous ORC. Overexpression of this dominant negative ORC subunit blocked the cell cycle in G<sub>1</sub>, providing the first evidence of the importance of ORC in cancer cell proliferation.</p><h2 id=materials-and-methods>MATERIALS AND METHODS
<a class=anchor href=#materials-and-methods>#</a></h2><h3 id=plasmid-constructions>Plasmid Constructions
<a class=anchor href=#plasmid-constructions>#</a></h3><p>Cloning of ORC1–6 cDNAs are described elsewhere (7, 9, 10, 11, 14). Coding sequences of ORC1, ORC4, ORC5, ORC6, and ORC3N200 were cloned into pFastBac (Life Technologies, Inc.), and coding sequences of ORC2 and ORC3 were cloned into pFastBac Dual. ORC2, -4, and -5 were also cloned in pFB-GST vectors to express GST fusion proteins. ORC3 and all the related constructs for <em>in vitro</em> transcription and translation reactions were made into T7T3DPAC vector (GenBank™ accession number U13871). Full-length and C-terminal ORC2 fragments were cloned into pGEX-5X-3 (Invitrogen) to produce bacterial fusion proteins. Additional information regarding the constructs will be made available upon request.</p><h3 id=expression-of-orc-subunits-in-insect-cells-purification-and-gel-filtration>Expression of ORC Subunits in Insect Cells, Purification, and Gel Filtration
<a class=anchor href=#expression-of-orc-subunits-in-insect-cells-purification-and-gel-filtration>#</a></h3><p>Baculoviruses were produced from the recombinant pFastBac or pFB-GST plasmids using the Bac-to-Bac expression system (Life Technologies, Inc.). Hi5 or Sf9 cells (Invitrogen) were infected with these baculoviruses according to the manufacturers&rsquo; recommendations. Cells were harvested 48 h post-infection. The cell pellet was washed once in cold phosphate-buffered saline and subsequently resuspended in hypotonic lysis buffer (10 m<span style=font-variant:small-caps>m</span> Tris/Cl, pH 7.9, 10 m<span style=font-variant:small-caps>m</span> KCl, 1.5 m<span style=font-variant:small-caps>m</span> MgCl<sub>2</sub>,1 m<span style=font-variant:small-caps>m</span> phenylmethylsulfonyl fluoride, 2 <em>µ</em>g/ml pepstatin, 2 <em>µ</em>g/ml leupeptin, 5 <em>µ</em>g/ml aprotinin, 1 m<span style=font-variant:small-caps>m</span> dithiothreitol). The cell suspension was homogenized in a Dounce homogenizer using a B-type pestle followed by centrifugation at 3000 rpm for 7 min. The pellet containing the nuclei was lysed in buffer H/0.15 (50 m<span style=font-variant:small-caps>m</span> HEPES/KOH, pH 7.5, 150 m<span style=font-variant:small-caps>m</span>KCl, 0.02% Nonidet P-40, 5 m<span style=font-variant:small-caps>m</span> magnesium acetate, 1 m<span style=font-variant:small-caps>m</span> EDTA, 1 m<span style=font-variant:small-caps>m</span> EGTA, 10% glycerol, 1 m<span style=font-variant:small-caps>m</span> phenylmethylsulfonyl fluoride, 2 <em>µ</em>g/ml pepstatin, 2 <em>µ</em>g/ml leupeptin, 5 <em>µ</em>g/ml aprotinin, 1 m<span style=font-variant:small-caps>m</span> dithiothreitol). The resulting suspension was subjected to ammonium sulfate precipitation (starting with 10% followed by 30% and finally 50%). The pellet after the 50% ammonium sulfate cut was resuspended in buffer H/0.0 (no salt) and then dialyzed overnight against buffer H/0.15. The dialyzed sample was then bound to GST beads (Sigma) and washed three times with buffer H/0.25. Proteins were eluted using reduced glutathione elution buffer (50 m<span style=font-variant:small-caps>m</span> Tris/Cl, pH 8.0, 20 m<span style=font-variant:small-caps>m</span> reduced glutathione, 0.01% Nonidet P-40, 100 m<span style=font-variant:small-caps>m</span> NaCl). Gel filtration of glutathione eluate using a fast protein liquid chromatography Superose 12 (Amersham Pharmacia Biotech) column was performed as described previously (14).</p><h3 id=cell-culture-transfection-immunoblotting-immunoprecipitation-and-silver-stain>Cell Culture, Transfection, Immunoblotting, Immunoprecipitation, and Silver Stain
<a class=anchor href=#cell-culture-transfection-immunoblotting-immunoprecipitation-and-silver-stain>#</a></h3><p>Sf9 and Hi5 cells were maintained according to the manufacturers&rsquo; protocol (Invitrogen). U2OS cells used for FACS analysis were maintained in Dulbecco&rsquo;s modified Eagle&rsquo;s medium with 10% fetal bovine serum (Life Technologies, Inc.). Plasmid DNA used for transfection were purified using Qiagen maxiprep kits. Cells were grown in 100-mm dishes and transfected using LipofectAMINE (Life Technologies, Inc.). Western blotting and immunoprecipitation techniques were carried out using standard protocols. Anti-GST polyclonal antibodies were purchased from Santa Cruz Biotechnologies. Antibodies against HsORC1–6 have been described previously (9, 10, 11,14). The silver stain method has been described elsewhere (19).</p><h3 id=in-vitro-transcription-and-translation-reactions-and-gst-pull-down-assay>In Vitro Transcription and Translation Reactions and GST Pull-down Assay
<a class=anchor href=#in-vitro-transcription-and-translation-reactions-and-gst-pull-down-assay>#</a></h3><p><em>In vitro</em> transcription and translation reactions to produce [<sup>35</sup>S]methionine-labeled full-length and different deletions of ORC3 were performed using the Promega TNT system (Madison, WI). Pull-down assays on glutathione-agarose beads were done as described previously (20).</p><h3 id=facs-analysis>FACS Analysis
<a class=anchor href=#facs-analysis>#</a></h3><p>U2OS cells were transfected with farnesylated GFP (CLONTECH) alone or in combination with FLAGORC2, GFPC1-ORC3N, or GFPC1-ORC3C1. Forty-eight hours after transfection, cells were trypsinized, washed with phosphate-buffered saline, fixed with cold 70% ethanol, and stored until further use. Before analysis, fixed cells were resuspended in phosphate-buffered saline containing 50 µg/ml propidium iodide (Sigma), 10 <em>µ</em>g/ml RNaseA (Sigma), and 0.05% Noniodet P-40 and then incubated for 1 h at 4 °C. Finally cells were washed in phosphate-buffered saline and analyzed by flow cytometry. The data were further analyzed using FLOWJO software to calculate the percentage of cells residing in different cell cycle stages.</p><h2 id=results>RESULTS
<a class=anchor href=#results>#</a></h2><h3 id=gstorc5--2--3-and--4-forms-a-complex>GSTORC5, -2, -3, and -4 Forms a Complex
<a class=anchor href=#gstorc5--2--3-and--4-forms-a-complex>#</a></h3><p>ORC2, -3, -4, and -5 subunits have been shown to interact with each other in human cell extracts (14). In an attempt to purify recombinant six protein human origin recognition complex, we infected Sf9 insect cells with baculoviruses expressing human ORC1–6 subunits. One of the subunits, ORC5, was GST-tagged. After pull down on glutathione beads, we found that GSTORC5, -2, -3, and -4 can be purified as a complex (Fig. 1<em>A</em>). ORC1 did not enter into the complex in a stoichiometric ratio, and the presence of very little ORC1 in Fig. 1<em>A</em> was not reproducible in different preparations. ORC6 did not enter into the complex at all. Both ORC1 and ORC6 were expressed at a high level. In a control experiment, we expressed GST alone with other ORC subunits. Pull down on glutathione beads purified only GST but none of the ORC subunits. Therefore, the results in Fig. 1<em>A</em> are due to the formation of a complex of ORC2, -3, -4, and -5 and not due to precipitation of the proteins on the glutathione beads. GST pull-down experiments using GST tags on different ORC subunits (GSTORC2 and GSTORC4) confirmed the previous result showing ORC2, -3, -4, and -5 form a core complex (data not shown).</p><p><img src=/img/architecture-human-ORC_fig1.jpg alt="Figure 1. Human ORC2–5 co-purify in a complex."></p><figcaption><p>Download : Download full-size image
<strong>Figure 1. Human ORC2–5 co-purify in a complex.</strong> A, Sf9 insect cells were infected with baculoviruses expressing ORC1, -2, -3, -4, -6 and GSTORC5. Proteins were purified as described under “Materials and Methods.” Both purified proteins and crude lysate (<em>Input</em>) were immunoblotted using ORC1–6 antibodies. <em>B</em>, Sf9 insect cells were infected the same way as in <em>A</em> using baculoviruses expressing ORC2–5 and GST as control. The cell lysate was purified on GST beads and immunoblotted. <em>C</em>, Sf9 insect cells were infected with baculoviruses expressing ORC2, -3, -4 and GST-ORC5. Proteins purified on GST beads were fractionated on a Superose 12 gel filtration column. Alternate fractions were immunoblotted using anti-ORC2, -3, -4 and anti-GST antibodies. The positions of the molecular mass markers thyroglobulin (670 kDa), bovine gamma globulin (158 kDa), and chicken ovalbumin (44 kDa) are shown on top. Input <em>lanes</em> were loaded with 5% of the total lysate passed through the column.</p></figcaption><p>To further show that GSTORC5, -2, -3, and -4 subunits are in one complex, we analyzed the elution pattern of these proteins upon gel filtration. Proteins were eluted from the GST beads using reduced glutathione and subsequently passed through a Superose 12 gel filtration column. Upon Western blotting of different fractions with different anti-ORC antibodies, we found that GSTORC5, -2, -3, and -4 subunits were co-eluted in one fraction (Fig. 1<em>C</em>). The molecular mass of this complex is ∼500 kDa, which is more than the combined molecular mass of the four ORC subunits. This may be because of the multimerization of the GST moieties to give a high molecular mass complex or because the complex has an atypical shape. Silver staining of the purified protein used for the gel filtration experiment indicated that GST-ORC5, ORC2, -3, and -4 were the only proteins present in the preparation in significant amounts (Fig. 1<em>D</em>).</p><h3 id=orc2-and-orc3-physically-interact-with-each-other>ORC2 and ORC3 Physically Interact with Each Other
<a class=anchor href=#orc2-and-orc3-physically-interact-with-each-other>#</a></h3><p>We were interested in seeing which of the four interacting subunits interact directly. Sf9 insect cells were infected with six different combinations of baculoviruses expressing two ORC subunits in each case. One of the two viruses was GST-tagged. Affinity purification on glutathione beads showed that only ORC2 and ORC3 directly interacted with each other (Fig. 2). None of the other dual combinations of baculovirus showed any interaction under our experimental conditions. Therefore, we conclude that ORC2 and ORC3 form a core component of the ORC2, -3, -4, -5 complex.</p><p><img src=/img/architecture-human-ORC_fig2.jpg alt="Figure 2. Direct interaction between ORC2 and ORC3 subunits."></p><figcaption><p>Download : Download full-size image
<strong>Figure 2. Direct interaction between ORC2 and ORC3 subunits.</strong> Sf9 insect cells were infected with six different combinations of baculoviruses expressing two ORC subunits in each case (GSTORC2-3 (<em>GSTO2+03</em>), GSTORC5-3 (<em>GSTO5+03</em>), GSTORC5-4 (<em>GSTO5+04</em>), GSTORC5-2 (<em>GSTO5+02</em>), GSTORC4-2 (<em>GSTO4+02</em>), and GSTORC4-3 (<em>GSTO4+03</em>)). Proteins bound to GST beads were immunoblotted using either anti-GST antibody or respective anti-ORC antibodies. 5% of the total lysate was loaded in the input <em>lanes</em>.</p></figcaption><h3 id=orc2-and-orc3-recruit-orc4-and--5>ORC2 and ORC3 Recruit ORC4 and -5
<a class=anchor href=#orc2-and-orc3-recruit-orc4-and--5>#</a></h3><p>The ORC2-3 complex is expected to recruit ORC4 and ORC5. We were interested in seeing whether ORC2-3 core complex can recruit ORC4 first, followed by ORC5, or vice versa. Sf9 cells were infected by baculoviruses expressing GSTORC4, -2, and -3 subunits or by viruses expressing GSTORC5, -2, and -3. GSTORC4 did not interact with ORC2 and ORC3, whereas GSTORC5 interacted with ORC2 and ORC3 (Fig. 3). Therefore, ORC2-3 core complex is capable of recruiting ORC5, but it cannot recruit ORC4 by itself. The fact that ORC2, -3, -4, and -5 form a complex suggests that ORC2, -3, and -5 complex is necessary to load ORC4. It is also possible that ORC4 and ORC5 can be loaded on ORC2-3 core complex simultaneously independent of each other, but ORC5 is necessary to stabilize the association of ORC4 with the other ORC subunits.</p><p><img src=/img/architecture-human-ORC_fig3.jpg alt="Figure 3. ORC2-3 subcomplex can load ORC5 but not ORC4."></p><figcaption><p>Download : Download full-size image
<strong>Figure 3. ORC2-3 subcomplex can load ORC5 but not ORC4.</strong> Sf9 insect cells were infected either with baculoviruses expressing GSTORC5-2-3 (<em>GSTORC5+2+3</em>) or baculoviruses expressing GSTORC4-2-3 (<em>GSTORC4+2+3</em>). Proteins bound to GST beads were immunoblotted using either anti-GST antibodies or respective anti-ORC antibodies. Input (<em>Inp.</em>) <em>lane</em> contains 5% of proteins input on GST beads.</p></figcaption><h3 id=n-terminal-portion-of-orc3-interacts-with-the-c-terminal-portion-of-orc2>N-terminal Portion of ORC3 Interacts with the C-terminal Portion of ORC2
<a class=anchor href=#n-terminal-portion-of-orc3-interacts-with-the-c-terminal-portion-of-orc2>#</a></h3><p>Upon establishing the fact that ORC2 and ORC3 form a core complex, we mapped the interacting domains of ORC2 and ORC3. N-terminal fragments of ORC3 labeled with [<sup>35</sup>S]methionine were produced by <em>in vitro</em> transcription and translation in rabbit reticulocyte lysate. The proteins were incubated with bacterially expressed and purified GSTORC2 protein. Three polypeptides derived from ORC3 were capable of binding GSTORC2, whereas the control GST protein did not bind any of them (Fig. 4<em>A</em>). The smallest fragment that bound to ORC2 contained 200 amino acids from the N terminus of ORC3 (<em>construct 3</em>, ORC3N). To map the portion of ORC2 involved in the interaction with ORC3, we expressed and purified GSTORC2C, containing the C-terminal 225 amino acids of ORC2. Both full-length ORC3 and ORC3N bound to GSTORC2C (Fig. 4<em>B</em>), whereas control GST alone did not bind to any one of them (data not shown). Therefore, the C-terminal 225 residues of ORC2 interact with the N-terminal 200 residues of ORC3 to form the ORC2-3 complex at the core of human ORC. In the reciprocal deletion, removal of the first 200 amino acids of ORC3 abolished its ability to bind to GSTORC2 (Fig. 4<em>C</em>). Based on these results we conclude that N-terminal 200 residues of ORC3 are necessary and sufficient to interact with ORC2.</p><p><img src=/img/architecture-human-ORC_fig4.jpg alt="Figure 4. Mapping domains of interactions between ORC2 and ORC3."></p><figcaption><p>Download : Download full-size image
<strong>Figure 4. Mapping domains of interactions between ORC2 and ORC3.</strong> <em>A</em>, full-length ORC3 or different N-terminal deletions of ORC3 were produced using <em>in vitro</em> transcription and translation reactions and tested for their ability to bind either GST or GSTORC2 in a pull-down experiment on glutathione-agarose beads coated with GST, GSTORC2 (<em>GSTO2</em>). In each case, the input <em>lanes</em> were loaded with 5% of the amount of the labeled protein incubated with the beads. The labeled proteins were visualized by SDS-polyacrylamide gel electrophoresis followed by fluorography. <em>B</em>, GST pull-down experiment as in <em>A</em> using GSTORC2C (C-terminal portion of ORC2) and <em>in vitro</em> transcribed and translated full-length ORC3 (<em>1</em>) or ORC3N200 (<em>3</em>). C, full-length ORC3 or different C-terminal deletions of ORC3 were produced using <em>in vitro</em> transcription translation reaction. GST pull-down experiments were performed as shown in Fig. 5<em>A</em> using either GST or GSTORC2.</p></figcaption><h3 id=n-terminal-200-amino-acids-of-orc3-can-compete-with-the-full-length-orc3>N-terminal 200 Amino Acids of ORC3 Can Compete with the Full-length ORC3
<a class=anchor href=#n-terminal-200-amino-acids-of-orc3-can-compete-with-the-full-length-orc3>#</a></h3><p>If the N-terminal 200 residues of ORC3 are sufficient for binding ORC2, ORC3N might be able to compete with full-length ORC3 for binding to GSTORC2. [<sup>35</sup>S]Methionine-labeled ORC3 was bound to GSTORC2 beads under conditions where the latter was limiting. These beads were then incubated with increasing amounts of ORC3N. We found that ORC3N protein could compete with the full-length ORC3 protein for association with GSTORC2 (Fig.5).</p><p><img src=/img/architecture-human-ORC_fig5.jpg alt="Figure 5. ORC3N can compete with full-length ORC3."></p><figcaption><p>Download : Download full-size image
<strong>Figure 5. ORC3N can compete with full-length ORC3.</strong> GSTORC2 beads were incubated with <em>in vitro</em> transcribed and translated full-length ORC3. After incubation, beads were thoroughly washed using binding buffer and incubated again with increasing amount of <em>in vitro</em> transcribed and translated ORC3N200. Beads were finally washed, and bound labeled proteins were visualized by SDS-polyacrylamide gel electrophoresis followed by fluorography.</p><h3 id=orc3n-cannot-form-a-complex-that-contains-orc4-5>ORC3N Cannot Form a Complex That Contains ORC4-5
<a class=anchor href=#orc3n-cannot-form-a-complex-that-contains-orc4-5>#</a></h3><p>Next we asked whether ORC3N is capable of mediating the interaction of ORC2 with ORC4 and ORC5. Sf9 insect cells were infected with baculoviruses expressing GSTORC5, ORC2, ORC4, and ORC3N. After affinity purification on glutathione beads and Western blotting, we confirmed the presence of GSTORC5 in the eluate from the beads. Interestingly, in contrast to the result in Fig. 1<em>A</em>, none of the other ORC subunits came down with GSTORC5, although they were all present in the input lane at reasonable quantities (Fig. 6<em>A</em>). We confirmed the physical interaction between ORC2 and ORC3N in the insect cell lysates by co-immunoprecipitation reactions. The lysate was immunoprecipitated using either anti-ORC2 or anti-ORC3 antibodies followed by immunoblotting with both the antibodies. ORC2 was detected in anti-ORC3 immunoprecipitate and vice versa (Fig. 6<em>B</em>). Therefore, ORC3N is capable of interacting with ORC2, but this interaction was not sufficient for further binding of ORC4 and ORC5. The C-terminal portion of ORC3 appears to be crucial for binding of ORC4 and ORC5 subunits to ORC2-3 subcomplex.</p><p><img src=/img/architecture-human-ORC_fig6.jpg alt="Figure 6. ORC3N does not form a complex with ORC2, -4, and -5."></p><figcaption><p>Download : Download full-size image
<strong>Figure 6. ORC3N does not form a complex with ORC2, -4, and -5.</strong> <em>A</em>, Sf9 insect cells were infected with baculoviruses expressing GSTORC5, -2, -4 and ORC3N200. Proteins bound to glutathione-agarose beads were immunoblotted using either anti-GST or anti-ORC antibodies. <em>B</em>, immunoprecipitation using anti-ORC2 and anti-ORC3 antibodies. Insect cell lysate from <em>A</em> was immunoprecipitated (<em>IP</em>) using either anti-ORC2 or anti-ORC3 antibodies followed by immunoblotting (<em>IB</em>) with either anti-ORC3 or anti-ORC2 antibodies. In each case 5% of the lysate (used for immunoprecipitation) was loaded in the input (<em>Inp</em>) <em>lanes</em>. Sera used for immunoprecipitation: <em>I</em>, immune, and <em>PI</em>, preimmune.</p></figcaption><h3 id=expression-of-orc3n-in-u2os-cells-causes-cell-cycle-arrest>Expression of ORC3N in U2OS Cells Causes Cell Cycle Arrest
<a class=anchor href=#expression-of-orc3n-in-u2os-cells-causes-cell-cycle-arrest>#</a></h3><p>Because ORC3N did not form a complex with ORC4 and ORC5 but still could interact with ORC2, we reasoned that ORC3N might show a dominant negative effect on the cell cycle if overexpressed in a human cancer cell line. ORC3N and ORC3C1 were cloned into GFPC1 (CLONTECH) expression vector to produce non-farnesylated GFP fusion proteins. U2OS cells were transfected with plasmids expressing farnesylated GFP alone or in combination with FLAGORC2 or GFPORC3N or GFPORC3C1 (1:3 molar ratio) followed by FACS analysis after 48 h. Cells transfected with GFP alone, FLAG-ORC2, or GFP-ORC3C1 showed normal cell cycle progression, whereas cells transfected with GFP-ORC3N were blocked mostly in G1 (73%) (Fig. 7). This is the first evidence for any cell cycle effect of any human ORC protein. Since ORC3N can still bind ORC2 but not ORC4 and ORC5, it is possible that over-expressed ORC3N interacts with ORC2 but prevents functional ORC formation. Consistent with this, over-expression of full-length ORC3, which interacts with ORC2 but allows functional ORC formation, did not block the cells in G<sub>1</sub> (data not shown).</p><p><img src=/img/architecture-human-ORC_fig7.jpg alt="Figure 7. FACS analysis of U2OS cells transfected with different constructs."></p><figcaption><p>Download : Download full-size image
<strong>Figure 7. FACS analysis of U2OS cells transfected with different constructs.</strong> U2OS cells were transfected either with farnesylated GFP (CLONTECH) or in combination with FLAGORC2, GFPORC3N, or GFPORC3C1. Transfected cells were fixed and stained with propidium iodide and then analyzed by FACS. The percentage of cell population present at different cell cycle stages in each transfection is shown at the bottom of the each <em>panel</em>.</p><h2 id=discussion>DISCUSSION
<a class=anchor href=#discussion>#</a></h2><p>We report here that human ORC2, -3, -4, and -5 form a core complex in baculovirus-infected insect cells. ORC1 and ORC6 did not interact with this core complex under these experimental conditions. This was confirmed by using different tags on different ORC subunits and is consistent with our previously published data showing that endogenous ORC2, -3, -4, and -5 in a HeLa cell extract physically interacted with each other but not with ORC1 and -6(14). Recently ORC1 has been shown to interact with ORC2 by co-immunoprecipitation reaction using high salt nuclear lysate from HeLa cells (21). This study did not include ORC3-6 proteins. Unfortunately, we could not reproduce the co-immunoprecipitation of ORC1 with ORC2 in human cell lines using two different ORC1 antibodies, raised independently. Under the same conditions we still found strong interaction among ORC2–5 subunits. The data from HeLa cell extract and recombinant baculovirus proteins strongly suggest that ORC2–5 form a core complex with ORC1 and ORC6, joining the complex either at very restricted times or locations or in a very labile interaction that is easily disrupted upon cell lysis.</p><p>We further found that ORC2 and ORC3 form a tight complex essential for binding ORC4 and ORC5. Gel filtration of 293T cell extract showed that ORC2 and 3 were the only two subunits that mostly co-eluted, consistent with the direct interaction between ORC2 and -3 reported here (16). Under our experimental conditions, none of the other ORC subunits interact with each other directly. The N-terminal 200 amino acids of ORC3 were enough to interact with C-terminal portion of ORC2 but not sufficient to allow association with ORC4 and ORC5, suggesting that the C-terminal of ORC3 is required for ORC4 and ORC5 loading on ORC2-3 subcomplex.</p><p>Recently, in yeast <em>S. cerevisiae</em>, ORC2 and ORC3 have been shown to interact directly (22). Insect cells were infected with baculoviruses expressing yeast ORCs, and recombinant yeast ORC was purified and tested for its DNA binding ability. ORC6 was found dispensable for DNA binding property. Elimination of ORC3 during baculovirus infection led to formation of ORC sub-complex without the presence of ORC2, suggesting that yeast ORC3 recruits ORC2 to the complex. Likewise, yeast ORC4 and ORC5 were shown to interact with each other. In addition, when yeast ORC was bound to yeast ARS1, ORC1, -2, -4, and -5 subunits were shown to directly contact ARS1 DNA by UV cross-linking (22). The human ORC2–5 sub-complex, however, did not show any sequence-specific DNA binding activity (data not shown).</p><p>Finally we showed that ORC3N has a dominant negative effect on cell cycle progression. U2OS cells expressing the same fragment were blocked in G<sub>1</sub>, whereas the ORC3C1 or ORC2 protein did not prevent the cells from normal cell cycle progression. This can be explained by the fact that ORC3N titrates out ORC2 or an unknown cellular protein in the cell, thereby blocking G<sub>1</sub>-S transition. Given that ORC2 and ORC3N form a very tight complex but the latter cannot support ORC2, -3, -4, -5 complex formation, we believe that ORC2 is the target that is titrated out by ORC3N. We cannot, however, overcome the effect of ORC3N by over-expressing ORC2 or ORC2C.<sup id=fnref:5><a href=#fn:5 class=footnote-ref role=doc-noteref>5</a></sup> ORC2 or ORC2C might not be expressed at high enough levels to titrate out the ORC3N. Alternatively, the ORC3N targets an unknown cellular factor to cause the G<sub>1</sub>-S block.</p><p>Based on these results we propose a molecular architecture of human ORC (Fig. 8). ORC2 and ORC3 interact directly with the C terminus of the former subunit, in close proximity with the N terminus of the latter. This binding favors the loading of ORC4 and -5 subunits via the C-terminal residues of ORC3. Although none of the other ORC subunits interacted with each other, we cannot rule out weak inter-subunit interactions among themselves. Further experiments will address how the ORC2–5 complex recruits ORC1 and ORC6 to form human ORC and study how the complex interacts with DNA.</p><p><img src=/img/architecture-human-ORC_fig8.jpg alt="Figure 8. Model of human ORC2–5 subcomplex. Human ORC subunits are showed as numbers 1–6."></p><figcaption><p>Download : Download full-size image
<strong>Figure 8. Model of human ORC2–5 subcomplex. Human ORC subunits are showed as numbers 1–6.</strong> ORC2–5 subunits are shown to form a complex. N and C in ORC2 and -3 depict N- and C-terminal residues of the individual subunits. The <em>arrow</em> with the <em>bold line</em> shows a strong interaction, whereas <em>arrows</em> with <em>dotted lines</em> indicate weak interactions.</p></figcaption><h2 id=acknowledgments>Acknowledgments
<a class=anchor href=#acknowledgments>#</a></h2><p>We acknowledge David Garcia Quintana, Anjali Satoshkar, and Zhi Hui Hou for making the baculovirus constructs and J. Wohlschlegel for help with the analysis of the FACS data.</p><h2 id=references>REFERENCES
<a class=anchor href=#references>#</a></h2><ol><li>S.P. Bell, B. Stillman <em>Nature</em>, <strong>357</strong> (1992), pp. 128-134
View Record in Scopus</li><li>A. Dutta, S.P. Bell
Annu. Rev. <em>Cell Dev. Biol.</em>, <strong>13</strong> (1997), pp. 293-332
CrossRefView Record in Scopus</li><li>T.J. Kelly, G.W. Brown
<em>Annu. Rev. Biochem.</em>, <strong>69</strong> (2000), pp. 829-880
CrossRefView Record in Scopus</li><li>A. Rowles, J.P. Chong, L. Brown, M. Howell, G.I. Evan, J.J. Blow
<em>Cell</em>, <strong>87</strong> (1996), pp. 287-296
ArticleDownload PDFView Record in Scopus</li><li>M. Gossen, D.T. Pak, S.K. Hansen, J.K. Acharya, M.R. Botchan
<em>Science</em>, <strong>270</strong> (1995), pp. 1674-1677
View Record in Scopus</li><li>K.Y. Moon, D. Kong, J.K. Lee, S. Raychaudhuri, J. Hurwitz
<em>Proc. Natl. Acad. Sci. U. S. A.</em>, <strong>96</strong> (1999), pp. 12367-12372
View Record in Scopus</li><li>K.A. Gavin, M. Hidaka, B. Stillman
<em>Science</em>, <strong>270</strong> (1995), pp. 1667-1671
View Record in Scopus</li><li>K. Takahara, M. Bong, R. Brevard, R.L. Eddy, L.L. Haley, S.J. Sait, T.B. Shows, G.G. Hoffman, D.S. Greenspan
<em>Genomics</em>, <strong>31</strong> (1996), pp. 119-122
ArticleDownload PDFView Record in Scopus</li><li>S. Pinto, D.G. Quintana, P. Smith, R.M. Mihalek, Z.H. Hou, S. Boynton, C.J. Jones, M. Hendricks, K. Velinzon, J.A. Wohlschlegel, R.J. Austin, W.S. Lane, T. Tully, A. Dutta
<em>Neuron</em>, <strong>23</strong> (1999), pp. 45-54
ArticleDownload PDFView Record in Scopus</li><li>D.G. Quintana, Z. Hou, K.C. Thome, M. Hendricks, P. Saha, A. Dutta
<em>J. Biol. Chem.</em>, <strong>272</strong> (1997), pp. 28247-28251
ArticleDownload PDFView Record in Scopus</li><li>D.G. Quintana, K.C. Thome, Z.H. Hou, A.H. Ligon, C.C. Morton, A. Dutta
<em>J. Biol. Chem.</em>, <strong>273</strong> (1998), pp. 27137-27145
ArticleDownload PDFView Record in Scopus</li><li>T. Tugal, X.H. Zou-Yang, K. Gavin, D. Pappin, B. Canas, R. Kobayashi, T. Hunt, B. Stillman
<em>J. Biol. Chem.</em>, <strong>273</strong> (1998), pp. 32421-32429
ArticleDownload PDFView Record in Scopus</li><li>M. Ishiai, F.B. Dean, K. Okumura, M. Abe, K.Y. Moon, A.A. Amin, K. Kagotani, H. Taguchi, Y. Murakami, F. Hanaoka, M. O&rsquo;Donnell, J. Hurwitz, T. Eki
<em>Genomics</em>, <strong>46</strong> (1997), pp. 294-298
ArticleDownload PDFView Record in Scopus</li><li>S.K. Dhar, A. Dutta
<em>J. Biol. Chem.</em>, <strong>275</strong> (2000), pp. 34983-34988
ArticleDownload PDFView Record in Scopus</li><li>D.A. Natale, C.J. Li, W.H. Sun, M.L. DePamphilis
<em>EMBO J.</em>, <strong>19</strong> (2000), pp. 2728-2738
View Record in Scopus</li><li>K.C. Thome, S.K. Dhar, D.G. Quintana, L. Delmolino, A. Shahsafaei, A. Dutta
<em>J. Biol. Chem.</em>, <strong>275</strong> (2000), pp. 35233-35241
ArticleDownload PDFView Record in Scopus</li><li>I. Chesnokov, M. Gossen, D. Remus, M. Botchan
<em>Genes Dev.</em>, <strong>13</strong> (1999), pp. 1289-1296
CrossRefView Record in Scopus</li><li>R.J. Austin, T.L. Orr-Weaver, S.P. Bell
<em>Genes Dev.</em>, <strong>13</strong> (1999), pp. 2639-2649
CrossRefView Record in Scopus</li><li>M.J. Dunn, S.J. Crisp
<em>Methods Mol. Biol.</em>, <strong>32</strong> (1994), pp. 113-118
CrossRefView Record in Scopus</li><li>Y.L. Lin, C. Chen, K.F. Keshav, E. Winchester, A. Dutta
<em>J. Biol. Chem.</em>, <strong>271</strong> (1996), pp. 17190-17198
ArticleDownload PDFView Record in Scopus</li><li>S. Kreitz, M. Ritzi, M. Baack, R. Knippers
<em>J. Biol. Chem.</em>, <strong>276</strong> (2000), pp. 6337-6342</li><li>D.G. Lee, S.P. Bell
<em>Mol. Cell. Biol.</em>, <strong>17</strong> (1997), pp. 7159-7168
View Record in Scopus</li></ol><p>Published, JBC Papers in Press, June 6, 2001, DOI 10.1074/jbc.M103078200</p><p>2
S. K. Dhar and A. Dutta, unpublished information.</p><p>© 2001 ASBMB. Currently published by Elsevier Inc; originally published by American Society for Biochemistry and Molecular Biology.</p><h2 id=recommended-articles>Recommended articles
<a class=anchor href=#recommended-articles>#</a></h2><ul><li>Ribosomal protein P2 localizes to the parasite zoite-surface and is a target for invasion inhibitory antibodies in Toxoplasma gondii and Plasmodium falciparum
<em>Parasitology International</em>, Volume <strong>64</strong>, Issue 1, 2015, pp. 43-49
Purchase PDFView details</li><li>JARID2 and the PRC2 complex regulate the cell cycle in skeletal muscle
<em>Journal of Biological Chemistry</em>, Volume <strong>294</strong>, Issue 51, 2019, pp. 19451-19464
Download PDFView details</li><li>BRCA1-Ku80 Protein Interaction Enhances End-joining Fidelity of Chromosomal Double-strand Breaks in the G1 Phase of the Cell Cycle
<em>Journal of Biological Chemistry</em>, Volume <strong>288</strong>, Issue 13, 2013, pp. 8966-8976
Download PDFView details
12Next</li></ul><h2 id=citing-articles-87>Citing articles (87)
<a class=anchor href=#citing-articles-87>#</a></h2><p>Article Metrics
Citations
Citation Indexes: 87
Captures
Readers: 27
Mentions
References: 6
plumX logoView details
Elsevier logo
About ScienceDirect
Remote access
Shopping cart
Advertise
Contact and support
Terms and conditions</p><h2 id=privacy-policy>Privacy policy
<a class=anchor href=#privacy-policy>#</a></h2><p>We use cookies to help provide and enhance our service and tailor content and ads. By continuing you agree to the use of cookies.</p><p>Copyright © 2021 Elsevier B.V. or its licensors or contributors.</p><p>ScienceDirect ® is a registered trademark of Elsevier B.V.</p><p>ScienceDirect ® is a registered trademark of Elsevier B.V.</p><p>RELX group home page</p><div class=footnotes role=doc-endnotes><hr><ol><li id=fn:1><p>Supported by United States Army Postdoctoral Fellowship DAMD17-00-1-0166.&#160;<a href=#fnref:1 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:2><p>Supported by the American Cancer Society Postdoctoral Fellowship PF-99-328-01-CCG.&#160;<a href=#fnref:2 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:3><p>To whom correspondence should be addressed: Dept. of Pathology, Brigham and Women’s Hospital, Harvard Medical School, 75 Francis St., Boston MA 02115. Tel.: 617-278-0468; Fax: 617-732-7449; E-mail: <a href=adutta@rics.bwh.harvard.edu>adutta@rics.bwh.harvard.edu</a>.&#160;<a href=#fnref:3 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:4><p>The abbreviations used are: ORC, origin recognition complex; GST, glutathione S-transferase; FACS, fluorescence-activated cell sorter; GFP green fluorescent protein.&#160;<a href=#fnref:4 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li><li id=fn:5><p>S. K. Dhar and A. Dutta, unpublished information.&#160;<a href=#fnref:5 class=footnote-backref role=doc-backlink>&#8617;&#xfe0e;</a></p></li></ol></div></article><footer class=book-footer><div class="flex flex-wrap justify-between"></div><script>(function(){function e(e){const t=window.getSelection(),n=document.createRange();n.selectNodeContents(e),t.removeAllRanges(),t.addRange(n)}document.querySelectorAll("pre code").forEach(t=>{t.addEventListener("click",function(){e(t.parentElement),navigator.clipboard&&navigator.clipboard.writeText(t.parentElement.textContent)})})})()</script></footer><div class=book-comments></div><label for=menu-control class="hidden book-menu-overlay"></label></div><aside class=book-toc><div class=book-toc-content><nav id=TableOfContents><ul><li><a href=#materials-and-methods>MATERIALS AND METHODS</a><ul><li><a href=#plasmid-constructions>Plasmid Constructions</a></li><li><a href=#expression-of-orc-subunits-in-insect-cells-purification-and-gel-filtration>Expression of ORC Subunits in Insect Cells, Purification, and Gel Filtration</a></li><li><a href=#cell-culture-transfection-immunoblotting-immunoprecipitation-and-silver-stain>Cell Culture, Transfection, Immunoblotting, Immunoprecipitation, and Silver Stain</a></li><li><a href=#in-vitro-transcription-and-translation-reactions-and-gst-pull-down-assay>In Vitro Transcription and Translation Reactions and GST Pull-down Assay</a></li><li><a href=#facs-analysis>FACS Analysis</a></li></ul></li><li><a href=#results>RESULTS</a><ul><li><a href=#gstorc5--2--3-and--4-forms-a-complex>GSTORC5, -2, -3, and -4 Forms a Complex</a></li><li><a href=#orc2-and-orc3-physically-interact-with-each-other>ORC2 and ORC3 Physically Interact with Each Other</a></li><li><a href=#orc2-and-orc3-recruit-orc4-and--5>ORC2 and ORC3 Recruit ORC4 and -5</a></li><li><a href=#n-terminal-portion-of-orc3-interacts-with-the-c-terminal-portion-of-orc2>N-terminal Portion of ORC3 Interacts with the C-terminal Portion of ORC2</a></li><li><a href=#n-terminal-200-amino-acids-of-orc3-can-compete-with-the-full-length-orc3>N-terminal 200 Amino Acids of ORC3 Can Compete with the Full-length ORC3</a></li><li><a href=#orc3n-cannot-form-a-complex-that-contains-orc4-5>ORC3N Cannot Form a Complex That Contains ORC4-5</a></li><li><a href=#expression-of-orc3n-in-u2os-cells-causes-cell-cycle-arrest>Expression of ORC3N in U2OS Cells Causes Cell Cycle Arrest</a></li></ul></li><li><a href=#discussion>DISCUSSION</a></li><li><a href=#acknowledgments>Acknowledgments</a></li><li><a href=#references>REFERENCES</a></li><li><a href=#recommended-articles>Recommended articles</a></li><li><a href=#citing-articles-87>Citing articles (87)</a></li><li><a href=#privacy-policy>Privacy policy</a></li></ul></nav></div></aside></main></body></html>